<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang xml:lang>
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>-</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    pre > code.sourceCode { white-space: pre; position: relative; }
    pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
    pre > code.sourceCode > span:empty { height: 1.2em; }
    .sourceCode { overflow: visible; }
    code.sourceCode > span { color: inherit; text-decoration: inherit; }
    div.sourceCode { margin: 1em 0; }
    pre.sourceCode { margin: 0; }
    @media screen {
    div.sourceCode { overflow: auto; }
    }
    @media print {
    pre > code.sourceCode { white-space: pre-wrap; }
    pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
    }
    pre.numberSource code
      { counter-reset: source-line 0; }
    pre.numberSource code > span
      { position: relative; left: -4em; counter-increment: source-line; }
    pre.numberSource code > span > a:first-child::before
      { content: counter(source-line);
        position: relative; left: -1em; text-align: right; vertical-align: baseline;
        border: none; display: inline-block;
        -webkit-touch-callout: none; -webkit-user-select: none;
        -khtml-user-select: none; -moz-user-select: none;
        -ms-user-select: none; user-select: none;
        padding: 0 4px; width: 4em;
        background-color: #ffffff;
        color: #a0a0a0;
      }
    pre.numberSource { margin-left: 3em; border-left: 1px solid #a0a0a0;  padding-left: 4px; }
    div.sourceCode
      { color: #1f1c1b; background-color: #ffffff; }
    @media screen {
    pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
    }
    code span { color: #1f1c1b; } /* Normal */
    code span.al { color: #bf0303; background-color: #f7e6e6; font-weight: bold; } /* Alert */
    code span.an { color: #ca60ca; } /* Annotation */
    code span.at { color: #0057ae; } /* Attribute */
    code span.bn { color: #b08000; } /* BaseN */
    code span.bu { color: #644a9b; font-weight: bold; } /* BuiltIn */
    code span.cf { color: #1f1c1b; font-weight: bold; } /* ControlFlow */
    code span.ch { color: #924c9d; } /* Char */
    code span.cn { color: #aa5500; } /* Constant */
    code span.co { color: #898887; } /* Comment */
    code span.cv { color: #0095ff; } /* CommentVar */
    code span.do { color: #607880; } /* Documentation */
    code span.dt { color: #0057ae; } /* DataType */
    code span.dv { color: #b08000; } /* DecVal */
    code span.er { color: #bf0303; text-decoration: underline; } /* Error */
    code span.ex { color: #0095ff; font-weight: bold; } /* Extension */
    code span.fl { color: #b08000; } /* Float */
    code span.fu { color: #644a9b; } /* Function */
    code span.im { color: #ff5500; } /* Import */
    code span.in { color: #b08000; } /* Information */
    code span.kw { color: #1f1c1b; font-weight: bold; } /* Keyword */
    code span.op { color: #1f1c1b; } /* Operator */
    code span.ot { color: #006e28; } /* Other */
    code span.pp { color: #006e28; } /* Preprocessor */
    code span.re { color: #0057ae; background-color: #e0e9f8; } /* RegionMarker */
    code span.sc { color: #3daee9; } /* SpecialChar */
    code span.ss { color: #ff5500; } /* SpecialString */
    code span.st { color: #bf0303; } /* String */
    code span.va { color: #0057ae; } /* Variable */
    code span.vs { color: #bf0303; } /* VerbatimString */
    code span.wa { color: #bf0303; } /* Warning */
    .display.math{display: block; text-align: center; margin: 0.5rem auto;}
  </style>
  <style type="text/css">@font-face {
font-family: octicons-link;
src: url(data:font/woff;charset=utf-8;base64,d09GRgABAAAAAAZwABAAAAAACFQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABEU0lHAAAGaAAAAAgAAAAIAAAAAUdTVUIAAAZcAAAACgAAAAoAAQAAT1MvMgAAAyQAAABJAAAAYFYEU3RjbWFwAAADcAAAAEUAAACAAJThvmN2dCAAAATkAAAABAAAAAQAAAAAZnBnbQAAA7gAAACyAAABCUM+8IhnYXNwAAAGTAAAABAAAAAQABoAI2dseWYAAAFsAAABPAAAAZwcEq9taGVhZAAAAsgAAAA0AAAANgh4a91oaGVhAAADCAAAABoAAAAkCA8DRGhtdHgAAAL8AAAADAAAAAwGAACfbG9jYQAAAsAAAAAIAAAACABiATBtYXhwAAACqAAAABgAAAAgAA8ASm5hbWUAAAToAAABQgAAAlXu73sOcG9zdAAABiwAAAAeAAAAME3QpOBwcmVwAAAEbAAAAHYAAAB/aFGpk3jaTY6xa8JAGMW/O62BDi0tJLYQincXEypYIiGJjSgHniQ6umTsUEyLm5BV6NDBP8Tpts6F0v+k/0an2i+itHDw3v2+9+DBKTzsJNnWJNTgHEy4BgG3EMI9DCEDOGEXzDADU5hBKMIgNPZqoD3SilVaXZCER3/I7AtxEJLtzzuZfI+VVkprxTlXShWKb3TBecG11rwoNlmmn1P2WYcJczl32etSpKnziC7lQyWe1smVPy/Lt7Kc+0vWY/gAgIIEqAN9we0pwKXreiMasxvabDQMM4riO+qxM2ogwDGOZTXxwxDiycQIcoYFBLj5K3EIaSctAq2kTYiw+ymhce7vwM9jSqO8JyVd5RH9gyTt2+J/yUmYlIR0s04n6+7Vm1ozezUeLEaUjhaDSuXHwVRgvLJn1tQ7xiuVv/ocTRF42mNgZGBgYGbwZOBiAAFGJBIMAAizAFoAAABiAGIAznjaY2BkYGAA4in8zwXi+W2+MjCzMIDApSwvXzC97Z4Ig8N/BxYGZgcgl52BCSQKAA3jCV8CAABfAAAAAAQAAEB42mNgZGBg4f3vACQZQABIMjKgAmYAKEgBXgAAeNpjYGY6wTiBgZWBg2kmUxoDA4MPhGZMYzBi1AHygVLYQUCaawqDA4PChxhmh/8ODDEsvAwHgMKMIDnGL0x7gJQCAwMAJd4MFwAAAHjaY2BgYGaA4DAGRgYQkAHyGMF8NgYrIM3JIAGVYYDT+AEjAwuDFpBmA9KMDEwMCh9i/v8H8sH0/4dQc1iAmAkALaUKLgAAAHjaTY9LDsIgEIbtgqHUPpDi3gPoBVyRTmTddOmqTXThEXqrob2gQ1FjwpDvfwCBdmdXC5AVKFu3e5MfNFJ29KTQT48Ob9/lqYwOGZxeUelN2U2R6+cArgtCJpauW7UQBqnFkUsjAY/kOU1cP+DAgvxwn1chZDwUbd6CFimGXwzwF6tPbFIcjEl+vvmM/byA48e6tWrKArm4ZJlCbdsrxksL1AwWn/yBSJKpYbq8AXaaTb8AAHja28jAwOC00ZrBeQNDQOWO//sdBBgYGRiYWYAEELEwMTE4uzo5Zzo5b2BxdnFOcALxNjA6b2ByTswC8jYwg0VlNuoCTWAMqNzMzsoK1rEhNqByEyerg5PMJlYuVueETKcd/89uBpnpvIEVomeHLoMsAAe1Id4AAAAAAAB42oWQT07CQBTGv0JBhagk7HQzKxca2sJCE1hDt4QF+9JOS0nbaaYDCQfwCJ7Au3AHj+LO13FMmm6cl7785vven0kBjHCBhfpYuNa5Ph1c0e2Xu3jEvWG7UdPDLZ4N92nOm+EBXuAbHmIMSRMs+4aUEd4Nd3CHD8NdvOLTsA2GL8M9PODbcL+hD7C1xoaHeLJSEao0FEW14ckxC+TU8TxvsY6X0eLPmRhry2WVioLpkrbp84LLQPGI7c6sOiUzpWIWS5GzlSgUzzLBSikOPFTOXqly7rqx0Z1Q5BAIoZBSFihQYQOOBEdkCOgXTOHA07HAGjGWiIjaPZNW13/+lm6S9FT7rLHFJ6fQbkATOG1j2OFMucKJJsxIVfQORl+9Jyda6Sl1dUYhSCm1dyClfoeDve4qMYdLEbfqHf3O/AdDumsjAAB42mNgYoAAZQYjBmyAGYQZmdhL8zLdDEydARfoAqIAAAABAAMABwAKABMAB///AA8AAQAAAAAAAAAAAAAAAAABAAAAAA==) format('woff');
}
.markdown-body {
-ms-text-size-adjust: 100%;
-webkit-text-size-adjust: 100%;
line-height: 1.5;
color: #24292e;
font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
font-size: 16px;
line-height: 1.5;
word-wrap: break-word;
}
.markdown-body .pl-c {
color: #6a737d;
}
.markdown-body .pl-c1,
.markdown-body .pl-s .pl-v {
color: #005cc5;
}
.markdown-body .pl-e,
.markdown-body .pl-en {
color: #6f42c1;
}
.markdown-body .pl-smi,
.markdown-body .pl-s .pl-s1 {
color: #24292e;
}
.markdown-body .pl-ent {
color: #22863a;
}
.markdown-body .pl-k {
color: #d73a49;
}
.markdown-body .pl-s,
.markdown-body .pl-pds,
.markdown-body .pl-s .pl-pse .pl-s1,
.markdown-body .pl-sr,
.markdown-body .pl-sr .pl-cce,
.markdown-body .pl-sr .pl-sre,
.markdown-body .pl-sr .pl-sra {
color: #032f62;
}
.markdown-body .pl-v,
.markdown-body .pl-smw {
color: #e36209;
}
.markdown-body .pl-bu {
color: #b31d28;
}
.markdown-body .pl-ii {
color: #fafbfc;
background-color: #b31d28;
}
.markdown-body .pl-c2 {
color: #fafbfc;
background-color: #d73a49;
}
.markdown-body .pl-c2::before {
content: "^M";
}
.markdown-body .pl-sr .pl-cce {
font-weight: bold;
color: #22863a;
}
.markdown-body .pl-ml {
color: #735c0f;
}
.markdown-body .pl-mh,
.markdown-body .pl-mh .pl-en,
.markdown-body .pl-ms {
font-weight: bold;
color: #005cc5;
}
.markdown-body .pl-mi {
font-style: italic;
color: #24292e;
}
.markdown-body .pl-mb {
font-weight: bold;
color: #24292e;
}
.markdown-body .pl-md {
color: #b31d28;
background-color: #ffeef0;
}
.markdown-body .pl-mi1 {
color: #22863a;
background-color: #f0fff4;
}
.markdown-body .pl-mc {
color: #e36209;
background-color: #ffebda;
}
.markdown-body .pl-mi2 {
color: #f6f8fa;
background-color: #005cc5;
}
.markdown-body .pl-mdr {
font-weight: bold;
color: #6f42c1;
}
.markdown-body .pl-ba {
color: #586069;
}
.markdown-body .pl-sg {
color: #959da5;
}
.markdown-body .pl-corl {
text-decoration: underline;
color: #032f62;
}
.markdown-body .octicon {
display: inline-block;
vertical-align: text-top;
fill: currentColor;
}
.markdown-body a {
background-color: transparent;
}
.markdown-body a:active,
.markdown-body a:hover {
outline-width: 0;
}
.markdown-body strong {
font-weight: inherit;
}
.markdown-body strong {
font-weight: bolder;
}
.markdown-body h1 {
font-size: 2em;
margin: 0.67em 0;
}
.markdown-body img {
border-style: none;
}
.markdown-body code,
.markdown-body kbd,
.markdown-body pre {
font-family: monospace, monospace;
font-size: 1em;
}
.markdown-body hr {
box-sizing: content-box;
height: 0;
overflow: visible;
}
.markdown-body input {
font: inherit;
margin: 0;
}
.markdown-body input {
overflow: visible;
}
.markdown-body [type="checkbox"] {
box-sizing: border-box;
padding: 0;
}
.markdown-body * {
box-sizing: border-box;
}
.markdown-body input {
font-family: inherit;
font-size: inherit;
line-height: inherit;
}
.markdown-body a {
color: #0366d6;
text-decoration: none;
}
.markdown-body a:hover {
text-decoration: underline;
}
.markdown-body strong {
font-weight: 600;
}
.markdown-body hr {
height: 0;
margin: 15px 0;
overflow: hidden;
background: transparent;
border: 0;
border-bottom: 1px solid #dfe2e5;
}
.markdown-body hr::before {
display: table;
content: "";
}
.markdown-body hr::after {
display: table;
clear: both;
content: "";
}
.markdown-body table {
border-spacing: 0;
border-collapse: collapse;
}
.markdown-body td,
.markdown-body th {
padding: 0;
}
.markdown-body h1,
.markdown-body h2,
.markdown-body h3,
.markdown-body h4,
.markdown-body h5,
.markdown-body h6 {
margin-top: 0;
margin-bottom: 0;
}
.markdown-body h1 {
font-size: 32px;
font-weight: 600;
}
.markdown-body h2 {
font-size: 24px;
font-weight: 600;
}
.markdown-body h3 {
font-size: 20px;
font-weight: 600;
}
.markdown-body h4 {
font-size: 16px;
font-weight: 600;
}
.markdown-body h5 {
font-size: 14px;
font-weight: 600;
}
.markdown-body h6 {
font-size: 12px;
font-weight: 600;
}
.markdown-body p {
margin-top: 0;
margin-bottom: 10px;
}
.markdown-body blockquote {
margin: 0;
}
.markdown-body ul,
.markdown-body ol {
padding-left: 0;
margin-top: 0;
margin-bottom: 0;
}
.markdown-body ol ol,
.markdown-body ul ol {
list-style-type: lower-roman;
}
.markdown-body ul ul ol,
.markdown-body ul ol ol,
.markdown-body ol ul ol,
.markdown-body ol ol ol {
list-style-type: lower-alpha;
}
.markdown-body dd {
margin-left: 0;
}
.markdown-body code {
font-family: "SFMono-Regular", Consolas, "Liberation Mono", Menlo, Courier, monospace;
font-size: 12px;
}
.markdown-body pre {
margin-top: 0;
margin-bottom: 0;
font-family: "SFMono-Regular", Consolas, "Liberation Mono", Menlo, Courier, monospace;
font-size: 12px;
}
.markdown-body .octicon {
vertical-align: text-bottom;
}
.markdown-body .pl-0 {
padding-left: 0 !important;
}
.markdown-body .pl-1 {
padding-left: 4px !important;
}
.markdown-body .pl-2 {
padding-left: 8px !important;
}
.markdown-body .pl-3 {
padding-left: 16px !important;
}
.markdown-body .pl-4 {
padding-left: 24px !important;
}
.markdown-body .pl-5 {
padding-left: 32px !important;
}
.markdown-body .pl-6 {
padding-left: 40px !important;
}
.markdown-body::before {
display: table;
content: "";
}
.markdown-body::after {
display: table;
clear: both;
content: "";
}
.markdown-body>*:first-child {
margin-top: 0 !important;
}
.markdown-body>*:last-child {
margin-bottom: 0 !important;
}
.markdown-body a:not([href]) {
color: inherit;
text-decoration: none;
}
.markdown-body .anchor {
float: left;
padding-right: 4px;
margin-left: -20px;
line-height: 1;
}
.markdown-body .anchor:focus {
outline: none;
}
.markdown-body p,
.markdown-body blockquote,
.markdown-body ul,
.markdown-body ol,
.markdown-body dl,
.markdown-body table,
.markdown-body pre {
margin-top: 0;
margin-bottom: 16px;
}
.markdown-body hr {
height: 0.25em;
padding: 0;
margin: 24px 0;
background-color: #e1e4e8;
border: 0;
}
.markdown-body blockquote {
padding: 0 1em;
color: #6a737d;
border-left: 0.25em solid #dfe2e5;
}
.markdown-body blockquote>:first-child {
margin-top: 0;
}
.markdown-body blockquote>:last-child {
margin-bottom: 0;
}
.markdown-body kbd {
display: inline-block;
padding: 3px 5px;
font-size: 11px;
line-height: 10px;
color: #444d56;
vertical-align: middle;
background-color: #fafbfc;
border: solid 1px #c6cbd1;
border-bottom-color: #959da5;
border-radius: 3px;
box-shadow: inset 0 -1px 0 #959da5;
}
.markdown-body h1,
.markdown-body h2,
.markdown-body h3,
.markdown-body h4,
.markdown-body h5,
.markdown-body h6 {
margin-top: 24px;
margin-bottom: 16px;
font-weight: 600;
line-height: 1.25;
}
.markdown-body h1 .octicon-link,
.markdown-body h2 .octicon-link,
.markdown-body h3 .octicon-link,
.markdown-body h4 .octicon-link,
.markdown-body h5 .octicon-link,
.markdown-body h6 .octicon-link {
color: #1b1f23;
vertical-align: middle;
visibility: hidden;
}
.markdown-body h1:hover .anchor,
.markdown-body h2:hover .anchor,
.markdown-body h3:hover .anchor,
.markdown-body h4:hover .anchor,
.markdown-body h5:hover .anchor,
.markdown-body h6:hover .anchor {
text-decoration: none;
}
.markdown-body h1:hover .anchor .octicon-link,
.markdown-body h2:hover .anchor .octicon-link,
.markdown-body h3:hover .anchor .octicon-link,
.markdown-body h4:hover .anchor .octicon-link,
.markdown-body h5:hover .anchor .octicon-link,
.markdown-body h6:hover .anchor .octicon-link {
visibility: visible;
}
.markdown-body h1 {
padding-bottom: 0.3em;
font-size: 2em;
border-bottom: 1px solid #eaecef;
}
.markdown-body h2 {
padding-bottom: 0.3em;
font-size: 1.5em;
border-bottom: 1px solid #eaecef;
}
.markdown-body h3 {
font-size: 1.25em;
}
.markdown-body h4 {
font-size: 1em;
}
.markdown-body h5 {
font-size: 0.875em;
}
.markdown-body h6 {
font-size: 0.85em;
color: #6a737d;
}
.markdown-body ul,
.markdown-body ol {
padding-left: 2em;
}
.markdown-body ul ul,
.markdown-body ul ol,
.markdown-body ol ol,
.markdown-body ol ul {
margin-top: 0;
margin-bottom: 0;
}
.markdown-body li {
word-wrap: break-all;
}
.markdown-body li>p {
margin-top: 16px;
}
.markdown-body li+li {
margin-top: 0.25em;
}
.markdown-body dl {
padding: 0;
}
.markdown-body dl dt {
padding: 0;
margin-top: 16px;
font-size: 1em;
font-style: italic;
font-weight: 600;
}
.markdown-body dl dd {
padding: 0 16px;
margin-bottom: 16px;
}
.markdown-body table {
display: block;
width: 100%;
overflow: auto;
}
.markdown-body table th {
font-weight: 600;
}
.markdown-body table th,
.markdown-body table td {
padding: 6px 13px;
border: 1px solid #dfe2e5;
}
.markdown-body table tr {
background-color: #fff;
border-top: 1px solid #c6cbd1;
}
.markdown-body table tr:nth-child(2n) {
background-color: #f6f8fa;
}
.markdown-body img {
max-width: 100%;
box-sizing: content-box;
background-color: #fff;
}
.markdown-body img[align=right] {
padding-left: 20px;
}
.markdown-body img[align=left] {
padding-right: 20px;
}
.markdown-body code {
padding: 0.2em 0.4em;
margin: 0;
font-size: 85%;
background-color: rgba(27,31,35,0.05);
border-radius: 3px;
}
.markdown-body pre {
word-wrap: normal;
}
.markdown-body pre>code {
padding: 0;
margin: 0;
font-size: 100%;
word-break: normal;
white-space: pre;
background: transparent;
border: 0;
}
.markdown-body .highlight {
margin-bottom: 16px;
}
.markdown-body .highlight pre {
margin-bottom: 0;
word-break: normal;
}
.markdown-body .highlight pre,
.markdown-body pre {
padding: 16px;
overflow: auto;
font-size: 85%;
line-height: 1.45;
background-color: #f6f8fa;
border-radius: 3px;
}
.markdown-body pre code {
display: inline;
max-width: auto;
padding: 0;
margin: 0;
overflow: visible;
line-height: inherit;
word-wrap: normal;
background-color: transparent;
border: 0;
}
.markdown-body .full-commit .btn-outline:not(:disabled):hover {
color: #005cc5;
border-color: #005cc5;
}
.markdown-body kbd {
display: inline-block;
padding: 3px 5px;
font: 11px "SFMono-Regular", Consolas, "Liberation Mono", Menlo, Courier, monospace;
line-height: 10px;
color: #444d56;
vertical-align: middle;
background-color: #fafbfc;
border: solid 1px #d1d5da;
border-bottom-color: #c6cbd1;
border-radius: 3px;
box-shadow: inset 0 -1px 0 #c6cbd1;
}
.markdown-body :checked+.radio-label {
position: relative;
z-index: 1;
border-color: #0366d6;
}
.markdown-body .task-list-item {
list-style-type: none;
}
.markdown-body .task-list-item+.task-list-item {
margin-top: 3px;
}
.markdown-body .task-list-item input {
margin: 0 0.2em 0.25em -1.6em;
vertical-align: middle;
}
.markdown-body hr {
border-bottom-color: #eee;
}
</style>
  <style type="text/css">.markdown-body {
box-sizing: border-box;
min-width: 200px;
max-width: 980px;
margin: 0 auto;
padding: 45px;
}
@media (max-width: 767px) {
.markdown-body {
padding: 15px;
}
}
del {
color: #b31d28;
background-color: #ffeef0;
}
ins {
color: #22863a;
background-color: #f0fff4;
}
del + ins {
color: #e36209;
background-color: #ffebda;
}
</style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<article class="markdown-body">
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>Johannes Elferich</strong>
<img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGFyaWEtaGlkZGVuPSJ0cnVlIiByb2xlPSJpbWciIHdpZHRoPSIxZW0iIGhlaWdodD0iMWVtIiBwcmVzZXJ2ZUFzcGVjdFJhdGlvPSJ4TWlkWU1pZCBtZWV0IiB2aWV3Qm94PSIwIDAgMzYgMzYiPjxwYXRoIGZpbGw9ImN1cnJlbnRDb2xvciIgZD0iTTMyIDZINGEyIDIgMCAwIDAtMiAydjIwYTIgMiAwIDAgMCAyIDJoMjhhMiAyIDAgMCAwIDItMlY4YTIgMiAwIDAgMC0yLTJabS0xLjU0IDIySDUuNjZsNy03LjI0bC0xLjQ0LTEuMzlMNCAyNi44NFY5LjUybDEyLjQzIDEyLjM3YTIgMiAwIDAgMCAyLjgyIDBMMzIgOS4yMXYxNy41bC03LjM2LTcuMzZsLTEuNDEgMS40MVpNNS4zMSA4aDI1LjA3TDE3Ljg0IDIwLjQ3WiIgY2xhc3M9ImNsci1pLW91dGxpbmUgY2xyLWktb3V0bGluZS1wYXRoLTEiLz48cGF0aCBmaWxsPSJub25lIiBkPSJNMCAwaDM2djM2SDB6Ii8+PC9zdmc+Cg==" class="inline_icon" width="16" height="16" alt="Corresponding" /><img src="data:image/svg+xml;base64,PCEtLSBNb2RpZmllZCBmcm9tIGh0dHBzOi8vZmlnc2hhcmUuY29tL2FydGljbGVzL09SQ0lEX2lEX2ljb25fZ3JhcGhpY3MvNTAwODY5NyB1bmRlciBpdHMgQ0MwIDEuMCBMaWNlbnNlIC0tPg0KPHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgaGVpZ2h0PSIxNiIgdmlld0JveD0iMCAwIDI1NiAyNTYiPg0KCTxwYXRoIGZpbGw9IiM4MDgwODAiIGZpbGwtcnVsZT0iZXZlbm9kZCIgZD0iTTI1NiwxMjhjMCw3MC43LTU3LjMsMTI4LTEyOCwxMjhDNTcuMywyNTYsMCwxOTguNywwLDEyOEMwLDU3LjMsNTcuMywwLDEyOCwwQzE5OC43LDAsMjU2LDU3LjMsMjU2LDEyOHogTTg2LjMsMTg2LjJINzAuOVY3OS4xaDE1LjR2NDguNFYxODYuMnogTTEwOC45LDc5LjFoNDEuNmMzOS42LDAsNTcsMjguMyw1Nyw1My42YzAsMjcuNS0yMS41LDUzLjYtNTYuOCw1My42aC00MS44Vjc5LjF6IE0xMjQuMywxNzIuNGgyNC41IGMzNC45LDAsNDIuOS0yNi41LDQyLjktMzkuN2MwLTIxLjUtMTMuNy0zOS43LTQzLjctMzkuN2gtMjMuN1YxNzIuNHogTTg4LjcsNTYuOGMwLDUuNS00LjUsMTAuMS0xMC4xLDEwLjFjLTUuNiwwLTEwLjEtNC42LTEwLjEtMTAuMWMwLTUuNiw0LjUtMTAuMSwxMC4xLTEwLjEgQzg0LjIsNDYuNyw4OC43LDUxLjMsODguNyw1Ni44eiI+PC9wYXRoPg0KPC9zdmc+DQo=" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9911-706X">0000-0002-9911-706X</a></p>
<p><span class="small">RNA Therapeutics Institute, University of
Massachusetts Chan Medical School, Worcester, MA, USA; Howard Hughes
Medical Institute, USA</span></p></li>
<li><p><strong>Giulia Schiroli</strong>
<img src="data:image/svg+xml;base64,PCEtLSBNb2RpZmllZCBmcm9tIGh0dHBzOi8vZmlnc2hhcmUuY29tL2FydGljbGVzL09SQ0lEX2lEX2ljb25fZ3JhcGhpY3MvNTAwODY5NyB1bmRlciBpdHMgQ0MwIDEuMCBMaWNlbnNlIC0tPg0KPHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgaGVpZ2h0PSIxNiIgdmlld0JveD0iMCAwIDI1NiAyNTYiPg0KCTxwYXRoIGZpbGw9IiM4MDgwODAiIGZpbGwtcnVsZT0iZXZlbm9kZCIgZD0iTTI1NiwxMjhjMCw3MC43LTU3LjMsMTI4LTEyOCwxMjhDNTcuMywyNTYsMCwxOTguNywwLDEyOEMwLDU3LjMsNTcuMywwLDEyOCwwQzE5OC43LDAsMjU2LDU3LjMsMjU2LDEyOHogTTg2LjMsMTg2LjJINzAuOVY3OS4xaDE1LjR2NDguNFYxODYuMnogTTEwOC45LDc5LjFoNDEuNmMzOS42LDAsNTcsMjguMyw1Nyw1My42YzAsMjcuNS0yMS41LDUzLjYtNTYuOCw1My42aC00MS44Vjc5LjF6IE0xMjQuMywxNzIuNGgyNC41IGMzNC45LDAsNDIuOS0yNi41LDQyLjktMzkuN2MwLTIxLjUtMTMuNy0zOS43LTQzLjctMzkuN2gtMjMuN1YxNzIuNHogTTg4LjcsNTYuOGMwLDUuNS00LjUsMTAuMS0xMC4xLDEwLjFjLTUuNiwwLTEwLjEtNC42LTEwLjEtMTAuMWMwLTUuNiw0LjUtMTAuMSwxMC4xLTEwLjEgQzg0LjIsNDYuNyw4OC43LDUxLjMsODguNyw1Ni44eiI+PC9wYXRoPg0KPC9zdmc+DQo=" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9812-8728">0000-0002-9812-8728</a></p>
<p><span class="small">Center for Regenerative Medicine, Massachusetts
General Hospital, Boston, MA, USA; Harvard Stem Cell Institute,
Cambridge, MA, USA; Department of Stem Cell and Regenerative Biology,
Harvard University, Cambridge, MA, USA</span></p></li>
<li><p><strong>David Scadden</strong>
<img src="data:image/svg+xml;base64,PCEtLSBNb2RpZmllZCBmcm9tIGh0dHBzOi8vZmlnc2hhcmUuY29tL2FydGljbGVzL09SQ0lEX2lEX2ljb25fZ3JhcGhpY3MvNTAwODY5NyB1bmRlciBpdHMgQ0MwIDEuMCBMaWNlbnNlIC0tPg0KPHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgaGVpZ2h0PSIxNiIgdmlld0JveD0iMCAwIDI1NiAyNTYiPg0KCTxwYXRoIGZpbGw9IiM4MDgwODAiIGZpbGwtcnVsZT0iZXZlbm9kZCIgZD0iTTI1NiwxMjhjMCw3MC43LTU3LjMsMTI4LTEyOCwxMjhDNTcuMywyNTYsMCwxOTguNywwLDEyOEMwLDU3LjMsNTcuMywwLDEyOCwwQzE5OC43LDAsMjU2LDU3LjMsMjU2LDEyOHogTTg2LjMsMTg2LjJINzAuOVY3OS4xaDE1LjR2NDguNFYxODYuMnogTTEwOC45LDc5LjFoNDEuNmMzOS42LDAsNTcsMjguMyw1Nyw1My42YzAsMjcuNS0yMS41LDUzLjYtNTYuOCw1My42aC00MS44Vjc5LjF6IE0xMjQuMywxNzIuNGgyNC41IGMzNC45LDAsNDIuOS0yNi41LDQyLjktMzkuN2MwLTIxLjUtMTMuNy0zOS43LTQzLjctMzkuN2gtMjMuN1YxNzIuNHogTTg4LjcsNTYuOGMwLDUuNS00LjUsMTAuMS0xMC4xLDEwLjFjLTUuNiwwLTEwLjEtNC42LTEwLjEtMTAuMWMwLTUuNiw0LjUtMTAuMSwxMC4xLTEwLjEgQzg0LjIsNDYuNyw4OC43LDUxLjMsODguNyw1Ni44eiI+PC9wYXRoPg0KPC9zdmc+DQo=" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-9821-7133">0000-0001-9821-7133</a></p>
<p><span class="small">Center for Regenerative Medicine, Massachusetts
General Hospital, Boston, MA, USA; Harvard Stem Cell Institute,
Cambridge, MA, USA; Department of Stem Cell and Regenerative Biology,
Harvard University, Cambridge, MA, USA</span></p></li>
<li><p><strong>Nikolaus Grigorieff</strong>
<img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGFyaWEtaGlkZGVuPSJ0cnVlIiByb2xlPSJpbWciIHdpZHRoPSIxZW0iIGhlaWdodD0iMWVtIiBwcmVzZXJ2ZUFzcGVjdFJhdGlvPSJ4TWlkWU1pZCBtZWV0IiB2aWV3Qm94PSIwIDAgMzYgMzYiPjxwYXRoIGZpbGw9ImN1cnJlbnRDb2xvciIgZD0iTTMyIDZINGEyIDIgMCAwIDAtMiAydjIwYTIgMiAwIDAgMCAyIDJoMjhhMiAyIDAgMCAwIDItMlY4YTIgMiAwIDAgMC0yLTJabS0xLjU0IDIySDUuNjZsNy03LjI0bC0xLjQ0LTEuMzlMNCAyNi44NFY5LjUybDEyLjQzIDEyLjM3YTIgMiAwIDAgMCAyLjgyIDBMMzIgOS4yMXYxNy41bC03LjM2LTcuMzZsLTEuNDEgMS40MVpNNS4zMSA4aDI1LjA3TDE3Ljg0IDIwLjQ3WiIgY2xhc3M9ImNsci1pLW91dGxpbmUgY2xyLWktb3V0bGluZS1wYXRoLTEiLz48cGF0aCBmaWxsPSJub25lIiBkPSJNMCAwaDM2djM2SDB6Ii8+PC9zdmc+Cg==" class="inline_icon" width="16" height="16" alt="Corresponding" /><img src="data:image/svg+xml;base64,PCEtLSBNb2RpZmllZCBmcm9tIGh0dHBzOi8vZmlnc2hhcmUuY29tL2FydGljbGVzL09SQ0lEX2lEX2ljb25fZ3JhcGhpY3MvNTAwODY5NyB1bmRlciBpdHMgQ0MwIDEuMCBMaWNlbnNlIC0tPg0KPHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgaGVpZ2h0PSIxNiIgdmlld0JveD0iMCAwIDI1NiAyNTYiPg0KCTxwYXRoIGZpbGw9IiM4MDgwODAiIGZpbGwtcnVsZT0iZXZlbm9kZCIgZD0iTTI1NiwxMjhjMCw3MC43LTU3LjMsMTI4LTEyOCwxMjhDNTcuMywyNTYsMCwxOTguNywwLDEyOEMwLDU3LjMsNTcuMywwLDEyOCwwQzE5OC43LDAsMjU2LDU3LjMsMjU2LDEyOHogTTg2LjMsMTg2LjJINzAuOVY3OS4xaDE1LjR2NDguNFYxODYuMnogTTEwOC45LDc5LjFoNDEuNmMzOS42LDAsNTcsMjguMyw1Nyw1My42YzAsMjcuNS0yMS41LDUzLjYtNTYuOCw1My42aC00MS44Vjc5LjF6IE0xMjQuMywxNzIuNGgyNC41IGMzNC45LDAsNDIuOS0yNi41LDQyLjktMzkuN2MwLTIxLjUtMTMuNy0zOS43LTQzLjctMzkuN2gtMjMuN1YxNzIuNHogTTg4LjcsNTYuOGMwLDUuNS00LjUsMTAuMS0xMC4xLDEwLjFjLTUuNiwwLTEwLjEtNC42LTEwLjEtMTAuMWMwLTUuNiw0LjUtMTAuMSwxMC4xLTEwLjEgQzg0LjIsNDYuNyw4OC43LDUxLjMsODguNyw1Ni44eiI+PC9wYXRoPg0KPC9zdmc+DQo=" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-1506-909X">0000-0002-1506-909X</a></p>
<p><span class="small">RNA Therapeutics Institute, University of
Massachusetts Chan Medical School, Worcester, MA, USA; Howard Hughes
Medical Institute, USA</span></p></li>
</ul>
<p><img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGFyaWEtaGlkZGVuPSJ0cnVlIiByb2xlPSJpbWciIHdpZHRoPSIxZW0iIGhlaWdodD0iMWVtIiBwcmVzZXJ2ZUFzcGVjdFJhdGlvPSJ4TWlkWU1pZCBtZWV0IiB2aWV3Qm94PSIwIDAgMzYgMzYiPjxwYXRoIGZpbGw9ImN1cnJlbnRDb2xvciIgZD0iTTMyIDZINGEyIDIgMCAwIDAtMiAydjIwYTIgMiAwIDAgMCAyIDJoMjhhMiAyIDAgMCAwIDItMlY4YTIgMiAwIDAgMC0yLTJabS0xLjU0IDIySDUuNjZsNy03LjI0bC0xLjQ0LTEuMzlMNCAyNi44NFY5LjUybDEyLjQzIDEyLjM3YTIgMiAwIDAgMCAyLjgyIDBMMzIgOS4yMXYxNy41bC03LjM2LTcuMzZsLTEuNDEgMS40MVpNNS4zMSA4aDI1LjA3TDE3Ljg0IDIwLjQ3WiIgY2xhc3M9ImNsci1pLW91dGxpbmUgY2xyLWktb3V0bGluZS1wYXRoLTEiLz48cGF0aCBmaWxsPSJub25lIiBkPSJNMCAwaDM2djM2SDB6Ii8+PC9zdmc+Cg==" class="inline_icon" width="16" height="16" alt="Corresponding" />
— correspondence preferred via <a href="https://github.com/jojoelfe/deco_lace_template_matching_manuscript/issues">GitHub
Issues</a>. Otherwise, address correspondence to <a href="mailto:johannes.elferich@umassmed.edu" class="email">johannes.elferich@umassmed.edu</a> and <a href="mailto:niko@grigorieff.org" class="email">niko@grigorieff.org</a>.</p>
<h2 id="abstract">Abstract</h2>
<p>A major goal of biological imaging is localization of biomolecules
inside a cell. Fluorescence microscopy can localize biomolecules inside
whole cells and tissues, but its ability to count biomolecules and
accuracy of the spatial coordinates is limited by the wavelength of
visible light. Cryo-electron microscopy (cryo-EM) provides highly
accurate position and orientation information of biomolecules but is
often confined to small fields of view inside a cell, limiting
biological context. In this study we use a new data-acquisition scheme
called “Defocus-Corrected Large-Area cryo-EM” (DeCo-LACE) to collect
high-resolution images of entire sections (100 –
<del>200</del><ins>250</ins> nm thick lamellae) of neutrophil-like mouse
cells, representing 1-2% of the total cellular volume. We use 2D
template matching (2DTM) to determine localization and orientation of
the large ribosomal subunit in these sections. These data provide “maps”
of ribosomes across entire sections of mammalian cells. This
high-throughput cryo-EM data collection approach together with 2DTM will
advance visual proteomics and provide biological insight that cannot be
obtained by other methods.</p>
<h2 id="introduction">Introduction</h2>
<p>A major goal in understanding cellular processes is the knowledge of
the amounts, location, interactions, and conformations of biomolecules
inside the cell. This knowledge can be obtained by approaches broadly
divided into label- and label-free techniques. In label-dependent
techniques a probe is physically attached to a molecule of interest that
is able to be detected by its strong signal, such as a fluorescent
molecule. In label-free techniques, the physical properties of molecules
themselves are used for detection. An example for this is proteomics
using mass-spectrometry [<span class="citation" data-cites="doi:10.1038/nbt.1592">@doi:10.1038/nbt.1592</span>]. The
advantage of label-free techniques is that they can provide information
over thousands of molecules, while label-dependent techniques offer
highly specific information for a few molecules. in particular, spatial
information is primarily achieved using label-dependent techniques, such
as fluorescence microscopy [<span class="citation" data-cites="doi:10.1038/nmeth817">@doi:10.1038/nmeth817</span>].</p>
<p>Cryo-electron microscopy (cryo-EM) has the potential to directly
visualize the arrangement of atoms that compose biomolecules inside of
cells, thereby allowing label-free detection with high spatial accuracy.
This has been called “visual proteomics” [<span class="citation" data-cites="doi:10.1038/nrm1861">@doi:10.1038/nrm1861</span>]. Since
cryo-EM requires thin samples (&lt;500nm), imaging of biomolecules
inside cells is restricted to small organisms, thin regions of cells, or
samples that have been suitably thinned. Thinning can be achieved either
by mechanical sectioning [<span class="citation" data-cites="doi:10.1111/j.1365-2818.1983.tb04225.x">@doi:10.1111/j.1365-2818.1983.tb04225.x</span>]
or by milling using a focused ion beam (FIB) [<span class="citation" data-cites="doi:10.1016/j.sbi.2013.08.006">@doi:10.1016/j.sbi.2013.08.006</span>].
This complex workflow leads to a low throughput of cryo-EM imaging of
cells and is further limited by the fact that at the required
magnifications, typical fields of view (FOV) are very small compared to
mammalian cells, and the FOV achieved by label-dependent techniques such
as fluorescence light microscopy. The predominant cryo-EM technique for
the localization of biomolecules of defined size and shape inside cells
is cryo-electron tomography [<span class="citation" data-cites="doi:10.1017/S0033583511000102">@doi:10.1017/S0033583511000102</span>].
However, the requirement of a tilt series at every imaged location and
subsequent image alignment, severely limits the throughput for molecular
localization.</p>
<p>An alternative approach is to identify molecules by their structural
“fingerprint” in single projection using “2D template-matching” (2DTM)
[<span class="citation" data-cites="doi:10.7554/eLife.25648">@doi:10.7554/eLife.25648</span>;
@doi:10.1101/2020.04.22.053868; @doi:10.7554/eLife.68946]. In this
method, a 3D model of a biomolecule is used as a template to find 2D
projections that match the molecules visible in the electron
micrographs. This method requires a projection search on a fine angular
grid, and the projections are used to find local cross-correlation peaks
with the micrograph. Since the location of a biomolecule in the
z-direction causes predictable aberrations to the projection image, this
method can be used to calculate complete 3D coordinates and orientations
of a biomolecule in a cellular sample [<span class="citation" data-cites="doi:10.1101/2020.04.22.053868">@doi:10.1101/2020.04.22.053868</span>].</p>
<p>Here we apply 2DTM of the ribosome large subunit (LSU) to a
conditionally immortalized <em>mus musculus</em> (mouse) cell line that
gives rise to functional mature neutrophils [<span class="citation" data-cites="doi:10.1016/j.cell.2016.08.057">@doi:10.1016/j.cell.2016.08.057</span>].
We chose these cells because genetic defects in the ribosome machinery
often leads to hematopoietic disease [<span class="citation" data-cites="doi:10.1093/nar/gkz637">@doi:10.1093/nar/gkz637</span>] and
direct quantification of ribosome location, number and conformational
states in hematopoietic cells could lead to new insight into
hematopoietic disease [<span class="citation" data-cites="doi:10.1186/s12885-018-4178-z">@doi:10.1186/s12885-018-4178-z</span>].
To increase the amount of collected data and to provide unbiased
sampling of the whole lamella, we devised a new data-acquisition scheme,
“Defocus-Corrected Large Area Cryo-Electron microscopy” (DeCo-LACE).
2DTM allows us to test whether aberrations caused by large beam-image
shifts and highly condensed beams deteriorate the high-resolution
signal. We find that these aberrations do not impede LSU detection by
2DTM. The resulting data provide a description of ribosome distribution
in an entire lamella, which represent 1-2% of the cellular volume. We
find a highly heterogeneous density of ribosomes within the cell.
Analysis of the throughput in this method suggests that for the
foreseeable future computation will be the bottleneck for visual
proteomics.</p>
<h2 id="results">Results</h2>
<h3 id="dtm-detects-large-ribosomal-subunits-in-cryo-fib-lamellae-of">2DTM
detects large ribosomal subunits in cryo-FIB lamellae of</h3>
<p>mammalian cells</p>
<p>FIB-milled <em>Saccharomyces cerevisiae</em> (yeast) cells are
sufficiently well preserved to permit localization of 60S ribosomal
subunits with 2DTM [<span class="citation" data-cites="doi:10.1101/2022.04.10.487797">@doi:10.1101/2022.04.10.487797</span>].
Due to the larger size of mammalian cells compared to yeast cells, it
was unclear whether plunge freezing would be adequate to produce
vitreous ice across the whole volume of the cell. To test this we
prepared cryo-lamellae of mouse neutrophil cells. A low magnification
image of a representative lamella clearly shows cellular features
consistent with a neutrophile-like phenotype, mainly a segmented nucleus
and a plethora of membrane-organelles, corresponding to the granules and
secretory vesicles of neutrophils (Fig. [<span class="citation" data-cites="fig:initmatching">@fig:initmatching</span>]A). We then
proceeded to acquire micrographs on this lamella with a defocus of
0.5-1.0 μm, 30 e<sup>-</sup>/Å<span class="math inline"><sup>2</sup></span>/s exposure and 1.76 Å pixel
size. We manually selected multiple locations in the lamella and
acquired micrographs using standard low-dose techniques where focusing
is performed on a sacrificial area. The resulting micrographs showed
smooth bilayered membranes and no signs of crystalline ice (Fig. [<span class="citation" data-cites="fig:initmatching">@fig:initmatching</span>]C,D), indicating
successful vitrification throughout the lamella.</p>
<p>We used an atomic model of the 60S mouse ribosomal subunit (6SWA) for
2DTM [<span class="citation" data-cites="doi:10.1016/j.molcel.2020.11.037">@doi:10.1016/j.molcel.2020.11.037</span>].
In a subset of images, the distribution of cross-correlation scores
significantly exceeded the distribution expected from images devoid of
detectable targets. In the resulting scaled maximum-intensity
projections (MIPs), clear peaks with SNR values up to 10 were apparent
(Fig. [<span class="citation" data-cites="fig:initmatching2">@fig:initmatching2</span>]A). Using a
threshold criterion to select significant targets (see Methods), we
found that in images of cytosolic compartments there were 10-500
ribosomes within one micrograph (Fig. [<span class="citation" data-cites="fig:initmatching">@fig:initmatching</span>]B-E). Notably, we
found no targets in areas corresponding to the nucleus (Fig. [<span class="citation" data-cites="fig:initmatching">@fig:initmatching</span>]B) or
mitochondria (Fig. <a href="#fig:initmatching">1</a>D). In the
cytoplasm, we found a highly variable number of targets, only ~ 50 in
some exposures (Fig. [<span class="citation" data-cites="fig:initmatching">@fig:initmatching</span>]E) and up to 500
in others (Fig. [<span class="citation" data-cites="fig:initmatching">@fig:initmatching</span>]C). However, it
is unclear whether this ten-fold difference in local ribosome
concentration is due to technical variation, such as sample thickness,
or biological variation. To differentiate between the two we reasoned it
was important to not manually choose imaging regions and to collect
larger amounts of data. We therefore set out to collect cryo-EM data for
2DTM from mammalian cell lamellae in a high-throughput unbiased
fashion.</p>
<h3 id="deco-lace-for-2d-imaging-of-whole-lamellae">DeCo-LACE for 2D
imaging of whole lamellae</h3>
<p>In order to obtain high-resolution data from complete lamellae, we
developed a new approach for data collection. This approach uses three
key strategies: (1) every electron that exposes a fresh area of the
sample is collected on the camera (2) image shift is used to precisely
and quickly raster the surface of a lamella and (3) focusing is done
without using a sacrificial area (Fig. [<span class="citation" data-cites="fig:approach">@fig:approach</span>]A).</p>
<p>To ensure that every electron exposing a fresh area of the sample is
captured by the detector, we adjusted the electron beam size to be
entirely contained by the detector area. During canonical low-dose
imaging, the microscope is configured so that the focal plane is
identical to the eucentric plane of the specimen stage. This leaves the
C2 aperture out of focus, resulting in ripples at the edge of the beam
(Fig. [<span class="citation" data-cites="fig:approach">@fig:approach</span>]D). While these ripples
are low-resolution features that likely do not interfere with 2DTM
[<span class="citation" data-cites="doi:10.7554/eLife.25648">@doi:10.7554/eLife.25648</span>],
we also tested data collection under conditions where the C2 aperture is
in focus (“fringe-free”, Fig. [<span class="citation" data-cites="fig:approach">@fig:approach</span>]E) [<span class="citation" data-cites="doi:10.1017/S1431927619005798">@doi:10.1017/S1431927619005798</span>].</p>
<p>We then centered a lamella on the optical axis of the microscope and
used the image shift controls of the microscope to systematically scan
the whole surface of the lamella in a hexagonal pattern (Fig. [<span class="citation" data-cites="fig:approach">@fig:approach</span>]A,C).
Instead of focusing on a sacrificial area, we determined the defocus
from every exposure after it was taken. The defocus was then adjusted
based on the difference between desired and measured defocus (Fig.
[<span class="citation" data-cites="fig:approach">@fig:approach</span>]B). Since we used a
serpentine pattern for data collection, every exposure was close to the
previous exposure, making large changes in the defocus unlikely.
Furthermore, we started our acquisition pattern on the platinum
deposition edge to make sure that the initial exposure where the defocus
was not yet adjusted did not contain any biologically relevant
information.</p>
<p>We used this strategy to collect data on eight lamellae, four using
the eucentric focus condition, hereafter referred to as
Lamella<sub>EUC</sub>, and four using the fringe-free condition,
hereafter referred to as Lamella<sub>FFF</sub>(Fig. [<span class="citation" data-cites="fig:assembly">@fig:assembly</span>] A+D,
Fig. [<span class="citation" data-cites="fig:lamella_images">@fig:lamella_images</span>]A). We were
able to collect data with a highly consistent defocus of 800 nm (Fig.
[<span class="citation" data-cites="fig:approach">@fig:approach</span>]F), both in the eucentric
focus and fringe-free focus condition. To ensure that data were
collected consistently, we mapped defocus values as a function of the
applied image shift (Fig. [<span class="citation" data-cites="fig:lamella_spatial_info">@fig:lamella_spatial_info</span>]A).
This demonstrated that the defocus was consistent across a lamella,
except for rare outliers and in images containing contamination. We also
plotted the measured objective astigmatism of each lamella and found
that it varies with the applied image shift, becoming more astigmatic
mostly due to image shift in the x direction (Fig. [<span class="citation" data-cites="fig:lamella_spatial_info">@fig:lamella_spatial_info</span>]B).
While approaches exist to correct for this during the data collection
[<span class="citation" data-cites="doi:10.1016/j.jsb.2019.09.013">@doi:10.1016/j.jsb.2019.09.013</span>],
we opted to not use these approaches in our initial experiments. We
reasoned that because 2DTM depends on high-resolution information, this
would be an excellent test of how much these aberration affect
imaging.</p>
<p>We assembled the tile micrographs into a montage using the
image-shift values and the SerialEM calibration followed by
cross-correlation based refinement (see Methods). In the resulting
montages, the same cellular features visible in the overview images are
apparent (Fig. [<span class="citation" data-cites="fig:assembly">@fig:assembly</span>]B+E, Fig. [<span class="citation" data-cites="fig:lamella_images">@fig:lamella_images</span>]B), however
due to the high magnification and low defocus many more details, such as
the membrane bilayer separation, can be observed (Fig. [<span class="citation" data-cites="fig:assembly">@fig:assembly</span>]C+F).
For montages collected using the eucentric condition, there are clearly
visible fringes at the edges between the tiles (Fig. [<span class="citation" data-cites="fig:assembly">@fig:assembly</span>]C),
which are absent in the fringe-free focus montages (Fig. [<span class="citation" data-cites="fig:assembly">@fig:assembly</span>]F). In
our analysis below, we show that these fringes do not impede target
detection by 2DTM, making them primarily an aesthetic issue. We also
note that the tiling pattern is visible in the montages (Fig. [<span class="citation" data-cites="fig:assembly">@fig:assembly</span>]B+E),
which we believe is due to the non-linear behavior of the K3 camera
since we can observe these shading artifacts in micrographs of a
condensed beam over vacuum (Fig. [<span class="citation" data-cites="fig:gain">@fig:gain</span>]).</p>
<p>The montages show membrane vesicles and granules with highly variable
sizes and density. We found that a substantial number of granules, which
are characterized by higher density inside the the surrounding cytosol
[<span class="citation" data-cites="doi:10.1084/jem.134.4.907">@doi:10.1084/jem.134.4.907</span>],
seemed to contain a membrane-enclosed inclusion with density similar to
the surrounding cytosol (Fig. [<span class="citation" data-cites="fig:lamella_images">@fig:lamella_images</span>]C) and could
therefore be formed by inward budding of the granule membrane. These
granules were 150-300 nm in diameter and the inclusions were 100-200 nm
in diameter. Based on these dimensions the granules are either azurophil
or specific granules [<span class="citation" data-cites="doi:10.1084/jem.134.4.907">@doi:10.1084/jem.134.4.907</span>].
To our knowledge, these inclusion have not been described in
granulocytes and are further described and discussed below.</p>
<h3 id="dtm-of-deco-lace-data-reveals-large-ribosomal-subunit-distribution">2DTM
of DeCo-LACE data reveals large ribosomal subunit distribution</h3>
<p>in cellular cross-sections</p>
<p>In our initial attempts of using 2DTM on micrographs acquired with
the DeCo-LACE protocol, we did not observe any SNR peaks above threshold
using the large subunit of the mouse ribosome (Fig. [<span class="citation" data-cites="fig:crop_unblur">@fig:crop_unblur</span>]A). We reasoned
that the edges of the beam might interfere with motion-correction of the
movies as they represent strong low-resolution features that do not move
with the sample. When we cropped the movie frames to exclude the beam
edges, the estimated amount of motion increased (Fig. [<span class="citation" data-cites="fig:crop_unblur">@fig:crop_unblur</span>]B), consistent with
successful tracking of sample motion. Furthermore, in the
motion-corrected average we could identify significant SNR peaks (Fig.
[<span class="citation" data-cites="fig:crop_unblur">@fig:crop_unblur</span>]B), confirming the
high sensitivity of 2DTM to the presence of high-resolution signal
preserved in the images by the motion correction. To streamline data
processing, we implemented a function in unblur to consider only a
defined central area of a movie for estimation of sample motion, while
still averaging the complete movie frames (Fig. [<span class="citation" data-cites="fig:crop_unblur">@fig:crop_unblur</span>]C). Using this
approach, we motion-corrected all tiles in the eight lamellae and found
consistently total motion below 1 Å per frame (Fig. [<span class="citation" data-cites="fig:lamella_motion_thickness">@fig:lamella_motion_thickness</span>]
A). In some lamellae we found increased motion in the lamella center,
which indicates areas of variable mechanical stability within FIB-milled
lamellae. In some micrographs we also observed that the beam edges gave
rise to artifacts in the MIP and numerous false-positive detections at
the edge of the illuminated area (Fig. [<span class="citation" data-cites="fig:crop_unblur">@fig:crop_unblur</span>]D). A similar
phenomenon was observed on isolated “hot” pixels in unilluminated areas.
To overcome this issue we implemented a function in unblur to replace
dark areas in the micrograph with Gaussian noise (see Methods), with
mean and standard deviation matching the illuminated portion of the
micrograph (Fig. [<span class="citation" data-cites="fig:crop_unblur">@fig:crop_unblur</span>]D+E). Together,
these pre-processing steps enabled us to perform 2DTM on all tiles of
the eight lamellae.</p>
<p>We used the tile positions to calculate the positions of the detected
LSUs in the lamellae (Fig. [<span class="citation" data-cites="fig:matching_euc">@fig:matching_euc</span>]A, Fig. [<span class="citation" data-cites="fig:matching_fff">@fig:matching_fff</span>]A, Movie 1, Movie
2). Overlaying these positions of the lamellae montages reveals LSU
distribution throughout the FIB-milled slices of individual cells.
Consistent with prior observations imaging selected views in yeast
[<span class="citation" data-cites="doi:10.1101/2022.04.10.487797">@doi:10.1101/2022.04.10.487797</span>],
organelles like the nucleus and mitochondria only showed sporadic
targets detected with low SNRs, consistent with the estimated
false-positive rate of one per tile. For each detected target we also
calculated the Z positions from the individual estimated defocus and
defocus offset for each tile. When viewed from the side, the ribosome
positions therefore show the slight tilts of the lamellae relative to
the microscope frame of reference (Fig. [<span class="citation" data-cites="fig:matching_euc">@fig:matching_euc</span>]B, Fig. [<span class="citation" data-cites="fig:matching_fff">@fig:matching_fff</span>]B, Movie 1, Movie
2). Furthermore, the side views indicated that lamellae were thinner at
the leading edge. <ins>To confirm this we estimated the ice thickness in
individual tiles using the Beer-Lambert law [<span class="citation" data-cites="doi:10.1016/j.jsb.2018.06.007">@doi:10.1016/j.jsb.2018.06.007</span>]
using 322 nm as coefficient [<span class="citation" data-cites="doi:10.1016/j.jsb.2018.06.007">@doi:10.1016/j.jsb.2018.06.007</span>]
(Fig. [<span class="citation" data-cites="fig:lamella_motion_thickness">@fig:lamella_motion_thickness</span>]B).
Indeed, we found that the thickness of the lamella varied between 100
and 250 nm, with consistently thinner icer at the leading edge. Even
though we prepared the lamellae with the “overtilt” approach [<span class="citation" data-cites="doi:10.1016/j.jsb.2016.07.010">@doi:10.1016/j.jsb.2016.07.010</span>],
this means that LSU densities across the lamellae can be skewed by a
change in thickness, and better sample preparation methods are needed to
generate more even samples.</ins></p>
<p>Indeed, when plotting the transmitted beam intensities in individual
tiles as a function of beam image-shift, we observed substantially
higher intensities at the leading edge (Fig. [<span class="citation" data-cites="fig:lamella_motion_thickness">@fig:lamella_motion_thickness</span>]B),
which in energy-filtered TEM indicates a thinner sample [<span class="citation" data-cites="doi:10.1016/j.jsb.2018.06.007">@doi:10.1016/j.jsb.2018.06.007</span>].
Even though we prepared the lamellae with the “overtilt” approach [<span class="citation" data-cites="doi:10.1016/j.jsb.2016.07.010">@doi:10.1016/j.jsb.2016.07.010</span>],
this means that LSU densities across the lamellae can be skewed by a
change in thickness, and better sample preparation methods are needed to
generate more even samples.</p>
<p>As described in [<span class="citation" data-cites="doi:10.7554/eLife.25648">@doi:10.7554/eLife.25648</span>]
the 2DTM SNR threshold for detecting a target is chosen to result in one
false positive detection per image searched. We would therefore expect
to find one false positive detection per tile. We reasoned that the
large nuclear area imaged by DeCo-LACE could be used to test whether
this assumption is true. In the 670 tiles containing exclusively nucleus
(as manually annotated from the overview image) we detected 247 targets,
making the false-positive rate more than twofold lower than expected.
Since earlier work shows that 2DTM with the LSU can produce matches to
nuclear ribosome biogenesis intermediates [<span class="citation" data-cites="doi:10.1101/2022.04.10.487797">@doi:10.1101/2022.04.10.487797</span>],
this could even be an overestimate of the false-positive rate. This
suggests that the detection threshold could be even lower, which is an
area of ongoing research.</p>
<p>Close inspection of the LSU positions in the lamellae revealed
several interesting features. LSUs could be seen associating with
membranes, in patterns reminiscent of the rough endoplasmic reticulum
(Fig. [<span class="citation" data-cites="fig:matching_euc">@fig:matching_euc</span>]C, Fig. [<span class="citation" data-cites="fig:matching_fff">@fig:matching_fff</span>]C) or the outer
nuclear membrane (Fig. [<span class="citation" data-cites="fig:matching_euc">@fig:matching_euc</span>]D). We also
observed LSUs forming ring-like structures (Fig. [<span class="citation" data-cites="fig:matching_euc">@fig:matching_euc</span>]E), potentially
indicating circularized mRNAs [<span class="citation" data-cites="pmid:9702200">@pmid:9702200</span>]. While ribosomes were
for the most part excluded from the numerous granules observed in the
cytoplasm, in some cases we observed clusters of LSUs in the inclusions
of double-membraned granules described earlier (Fig. [<span class="citation" data-cites="fig:matching_euc">@fig:matching_euc</span>]F, Fig. [<span class="citation" data-cites="fig:matching_fff">@fig:matching_fff</span>]D,E). It is, in
principle, possible that these targets are situated above or below the
imaged granules, since the granule positions in z cannot be determined
using 2D projections. However, in the case of Fig. [<span class="citation" data-cites="fig:matching_fff">@fig:matching_fff</span>]E, the detected
LSUs span the whole lamella in the z direction (Fig. [<span class="citation" data-cites="fig:matching_fff">@fig:matching_fff</span>]F), while
positions above or below a granule would result in LSUs situated
<del>exlusively</del><ins>exclusively</ins> at the top or bottom of the
lamella. This is consistent with the earlier hypothesis that the
inclusions are of cytoplasmic origin.</p>
<h3 id="does-deco-lace-induce-aberrations-that-affect-2dtm">Does
DeCo-LACE induce aberrations that affect 2DTM?</h3>
<p>Within the eight lamellae we found different numbers of detected
targets, ranging from 1089 to 6433 per lamella (Fig. [<span class="citation" data-cites="fig:matching_stat">@fig:matching_stat</span>]A).
Lamella<sub>EUC</sub> 1 had the most detected targets, but also has the
largest surface area and contained cytoplasm from two cells.
Lamella<sub>FFF</sub> 4 had the fewest detected targets, but this
particular lamella was dominated by a circular section of the nucleus,
with only small pockets of cytoplasm (Fig. [<span class="citation" data-cites="fig:lamella_images">@fig:lamella_images</span>]). In an
attempt to normalize for these differences in area containing cytoplasm,
we compared the number of detected targets per tile in tiles that
contained more than one target, which should exclude tiles with
non-cytosolic content (Fig. [<span class="citation" data-cites="fig:matching_stat">@fig:matching_stat</span>]B). While this
measure had less variability, there were still differences.
Lamella<sub>EUC</sub> 4 had not only the fewest targets, but also the
lowest density, which could be due to this lamella being the thinnest,
or due to it sectioning the cell in an area with a lower concentration
of ribosomes. Lamella<sub>FFF</sub> 3 had a substantially higher number
of ribosomes per tile. Since all of these lamellae were made from a
cell-line under identical conditions, this underscores the necessity to
collect data from large numbers of lamellae to overcome the inherent
variability. When comparing the distribution of scores between lamellae,
we found them to be fairly comparable with median SNRs ranging from 8.7
to 9.7 (Fig. [<span class="citation" data-cites="fig:matching_stat">@fig:matching_stat</span>]C).
Lamella<sub>EUC</sub> 1 had slightly lower scores compared to the rest,
potentially due to its large size and connected mechanical instability
during imaging. Overall, we did not observe differences in the number or
SNR of detected targets between eucentric or fringe-free illumination
conditions that were bigger than the observed inter-lamella
variability.</p>
<p>Since the SNR values of 2DTM are highly sensitive to image quality,
we reasoned we could use them to verify that DeCo-LACE does not
introduce a systematic loss of image quality. We considered non-parallel
illumination introduced by the unusually condensed beam and
uncharacterized aberrations near the beam periphery. When plotting the
SNR values of detected targets in all eight lamellae as a function of
their location in the tiles, we found uniformly high SNR values
throughout the illuminated areas for both eucentric and fringe-free
focus illumination, demonstrating that both illumination schemes are
suitable for DeCo-LACE (Fig. [<span class="citation" data-cites="fig:matching_stat">@fig:matching_stat</span>]D).</p>
<p>We also wondered whether large image shifts would lead to aberrations
due to astigmatism or beam tilt [<span class="citation" data-cites="doi:10.1016/j.jsb.2019.09.013">@doi:10.1016/j.jsb.2019.09.013</span>].
We reasoned that if that was the case the number of detected targets
should be highest in the center of the lamella where the applied beam
image-shift is 0. Instead, we observed that in both eucentric and
fringe-free focus conditions more targets were detected at the “back”
edge of the lamella (Fig. [<span class="citation" data-cites="fig:matching_stat">@fig:matching_stat</span>]E]). This may
be due to the center of the cell being predominantly occupied by the
nucleus, despite its segmentation in neutrophil-like cells. The increase
in matches at the “back” of the lamellae compared to the “front” can
also be explained by the thickness gradient of the lamellae (Fig. [<span class="citation" data-cites="fig:lamella_motion_thickness">@fig:lamella_motion_thickness</span>]B,
Fig. [<span class="citation" data-cites="fig:matching_euc">@fig:matching_euc</span>]B, Fig. [<span class="citation" data-cites="fig:matching_fff">@fig:matching_fff</span>]B). In addition,
aberrations would be expected to cause average 2DTM SNRs to be higher
when beam-image shift values are small. Instead, we found that SNRs
where on average the highest at the “front” edge of the lamellae,
presumably due to the thinner sample. We therefore conclude that factors
other that beam image-shift or beam condensation aberrations are
limiting 2DTM SNRS, predominantly the thickness of the lamellae.</p>
<h3 id="computation-is-the-bottleneck-of-visual-proteomics">Computation
is the bottleneck of visual proteomics</h3>
<p>All lamellae described above were derived from a clonal cell line
under identical condition and thinned with the same parameters. This
means that the substantial variability of detected targets between the
lamellae must be due to technical variability, including area,
thickness, mechanical stability, and location of the section within the
cell. We therefore predict that further studies that want to draw
quantitative and statistically relevant conclusions about the number and
location of molecules under different experimental conditions, will
require collection of orders of magnitude more data than in this study
to gain enough statistical power given this variability. The samples
used were prepared in two 24 h sessions on a FIB/SEM instrument, and
imaging was performed during another two 24h session on the TEM
microscope. Inspections of the timestamps of the raw data files revealed
that the milling time per lamella was ~30 minutes and TEM imaging was
accomplished in ~10 seconds per tile or 90 minutes for a ~ 6x6 μm
lamella. Processing of the data, however, took substantially longer.
Specifically, 2DTM of all tiles took approximately one week per lamella
on 32 Nvidia A6000 GPUs. Computation is therefore a bottleneck in our
current workflow, and further optimizations of the algorithm may be
necessary increase throughput. Alternatively, this bottleneck could be
reduced by increasing the number of processing units.</p>
<h2 id="discussion">Discussion</h2>
<p>In this study we developed an approach to image entire cellular
cross-section using cryo-EM at high enough resolution to allow for 2DTM
detection of the LSU. The two main advantages compared to previous
approaches are high throughput of imaging and biological context for
detected molecules. The requirement to increase throughput in cryo-EM
data collection of cellular samples has been recognized in the recent
literature. Most approaches described so far are tailored towards
tomography. Peck et al. [<span class="citation" data-cites="doi:10.1016/j.jsb.2022.107860">@doi:10.1016/j.jsb.2022.107860</span>]
and Yang et al. [<span class="citation" data-cites="doi:10.1101/2021.12.31.474669">@doi:10.1101/2021.12.31.474669</span>]
developed approaches to increase the FOV of tomogram data-collection by
using a montaging technique. Peck et al. used a similar “condensed-beam”
approach as described here. However, the montages are substantially
smaller in scope, covering carbon film holes of 2 µm diameter. Bouvette
et al. [<span class="citation" data-cites="doi:10.1038/s41467-021-22251-8">@doi:10.1038/s41467-021-22251-8</span>]
and Eisenstein et al. [<span class="citation" data-cites="doi:10.1101/2022.04.07.487557">@doi:10.1101/2022.04.07.487557</span>]
are using beam image-shift to collect tilt-series in multiple locations
in parallel to increase throughput. However, none of these approaches
provide the full coverage of a cellular cross-section that can be
achieved using DeCo-LACE.</p>
<p>We observed granules containing a vesicle of putative cytosolic
origin. We speculate that upon degranulation, the process in which
granules fuse with the plasma membrane, these vesicles would be released
into the extracelullar space. The main types of extracellular vesicles
of this size are exosomes, up to 100 nm large vesicles derived from
fusion of multivesicular bodies with the plasma membrane, and
microvesicles, which are derived from direct budding of the plasma
membrane [<span class="citation" data-cites="doi:10.1038/nrm.2017.125">@doi:10.1038/nrm.2017.125</span>].
We suggest that granulocytes could release a third type of extracellular
vesicle, granule-derived vesicles (GDV), into the extracellular space.
2DTM showed that a subset of GDVs can contain ribosomes (Fig. [<span class="citation" data-cites="fig:matching_euc">@fig:matching_euc</span>]F, Fig. [<span class="citation" data-cites="fig:matching_fff">@fig:matching_fff</span>]D,E). This could
indicate that these vesicles are transporting translation-capable mRNAs,
as has been described for exosomes [<span class="citation" data-cites="doi:10.1038/ncb1596">@doi:10.1038/ncb1596</span>]. Further
studies will be necessary to confirm the existence of GDVs in
granulocytes isolated from mammals and to understand their functional
significance.</p>
<p>As mentioned in the results, we found a consistent shading artifact
pattern in our montages, that we believe is the result of non-linear
behavior of the K3 camera. Indeed, when we average images with a
condensed beam taken over vacuum we found in both focus conditions a
consistent background pattern with a brighter region on the periphery of
the illuminated area (Fig [<span class="citation" data-cites="fig:gain">@fig:gain</span>]). This might be caused by
dynamic adjustment of the internal camera counting threshold which
expects columns of the sensor to be evenly illuminated as is the case
for SPA applications. Since the signal of this pattern has mainly
low-resolution components it is unlikely to affect 2DTM. However, it
highlights that the non-linear behavior of the camera has to be taken
into account when imaging samples with strongly varying density and
unusual illumination schemes.</p>
<p>We found that even though we used beam image-shift extensively (up to
7 um), we did not see substantially reduced 2DTM SNR values in tiles
acquired at high beam image-shift compared to tiles acquired with low or
no beam image-shift. This is in contrast to reports in single-particle
analysis (SPA) [<span class="citation" data-cites="doi:10.1107/S2052252520013482">@doi:10.1107/S2052252520013482</span>]
where the induced beam tilt substantially reduced the resolution if it
was not corrected during processing. It is possible that 2DTM is less
sensitive to beam-tilt aberrations, since the template is free of any
aberration and only the image is distorted, while in SPA the beam tilt
will affect both the images and the reconstructed template.</p>
<p>
<ins>
We used 2DTM of the ribosome LSU to verify the image quality of
DeCO-LACE data. However, 2DTM has the potential to detect many other
protein complexes. The model of the ribosomal LSU used in this
manuscript has a molecular weight of ~2MDa, ~1.3 MDA of which is nucleic
acid. The higher scattering density of nucleic acid compared to protein
[<span class="citation" data-cites="doi:10.1016/S0969-2126">@doi:10.1016/S0969-2126</span>(00)00185-4]
might mean that a biomolecular complex of equal size consisting of only
protein would result in lower SNR values. Simulations of 2DTM [<span class="citation" data-cites="doi:10.7554/eLife.25648">@doi:10.7554/eLife.25648</span>]
suggest that it should be possible to detect purely protein molecules as
small as 300 kDa in an crowded environment, but is unclear whether this
is achievable in practice.
</ins>
</p>
<p>Since we observed substantial variation in LSU density within and
between lamellae, visual proteomics studies that use cryo-EM to
establish changes in molecular organization within cells will require
orders of magnitude more data than used in this study. One milestone
would be to image enough data to represent one cellular volume, which
for a small eukaryotic cells requires imaging approximately 100
lamellae. While data collection throughput on the TEM is fundamentally
limited by the exposure time, this amount of data could be collected
within 12 hours by improving the data acquisition scheme to perform all
necessary calculations in parallel with actual exposure of the camera.
Sample preparation using a FIB/SEM is also currently a bottleneck, but
preparation of large lamellae with multiple cellular cross-sections
using methods like WAFFLE [<span class="citation" data-cites="doi:10.1038/s41467-022-29501-3">@doi:10.1038/s41467-022-29501-3</span>]
might allow sufficient throughput. As stated in the results, at least
for 2DTM computation will remain challenging and approximately 17,000
GPU hours would be required for a 100 lamellae dataset.<ins> It might be
possible to increase the performance of 2DTM by improving the
implementation of GPU-based fast fourier transforms. Alternatively,
strategies that initially search images at lower resolution and sparser
rotational sampling, followed by refinement at higher resolution, could
be more efficient, but it is yet unclear whether this is feasible in
cellular samples with a lot of low-resolution background.</ins></p>
<h2 id="materials-and-methods">Materials and Methods</h2>
<h3 id="grid-preparation">Grid preparation</h3>
<p>ER-HoxB8 cells were maintained in RPMI medium supplemented with 10%
FBS, penicillin/streptomycin, SCF, and estrogen [<span class="citation" data-cites="doi:10.1016/j.cell.2016.08.057">@doi:10.1016/j.cell.2016.08.057</span>]
at 37 °C and 5% CO2. 120 h prior to grid preparation, cells were washed
twice in PBS and cultured in the same medium except without estrogen.
Differentiation was verified by staining with Hoechst-dye and inspection
of nuclear morphology. Cells were then counted and diluted to <span class="math inline">1 ⋅ 10<sup>6</sup></span> cells/ml. Grids (either
200 mesh copper grids, with a sillicone-oxide and 2 µm holes with a 2 µm
spacing or 200 mesh gold grids with a thin gold film and 2 µm holes in 2
µm spacing) were glow-discharged from both sides using a 15 mA for 45 s.
3.5 µl of cell suspension was added to grids on the thin-film side and
grids were blotted from the back side using a GP2 cryoplunger (Leica,
Wetzlar, Germany) for 8 s and rapidly plunged into liquid ethane at -185
°C.</p>
<h3 id="fib-milling">FIB-milling</h3>
<p>Grids were loaded into an Aquilos 2 FIB/SEM (Thermo Fisher, Waltham,
MA) instrument with a stage cooled to -190 °C. Grids were sputter-coated
with platinum for 15 s at 45 mA and then coated with a layer of
platinum-precursor by opening the GIS-valve for 45 s. An overview of the
grid was created by montaging SEM images and isolated cells at the
center of gridsquares were selected for FIB-milling. Lamellae were
generated automatically using the AutoTEM software (Thermo Fisher), with
the following parameters:</p>
<ul>
<li>Milling angle: 20°</li>
<li>Rough milling: 3.2 µm thickness, 0.5 nA current</li>
<li>Medium milling: 1.8 µm thickness, 0.3 nA current, 1.0° overtilt</li>
<li>Fine milling: 1.0 µm tchickness, 0.1 nA current, 0.5° overtilt</li>
<li>Finer milling: 700 nm thickness, 0.1 nA curent, 0.2° overtilt</li>
<li>Polish 1: 450 nm thickness, 50 pA current</li>
<li>Polish 2: 200 nm thickness, 30 pA current</li>
</ul>
<p>This resulted in 6-10 µm wide lamella with 150-250 nm thickness as
determined by FIB-imaging of the lamella edges.</p>
<h3 id="data-collection">Data collection</h3>
<p>Grids were loaded into a Titan Krios TEM (Thermo Fisher) operated at
300 keV and equipped with a BioQuantum energy filter (Gatan, Pleasanton,
CA) and K3 camera (Gatan). The microscope was aligned using a
cross-grating grid on the stage. Prior to each session, we carefully
performed the “Image/Beam” calibration in nanoprobe. We set the
magnification to a pixel size of 1.76 Å and condensed the beam to ~ 900
nm diameter, resulting in the beam being completely visible on the
camera. To establish fringe-free conditions, the “Fine eucentric”
procedure of SerialEM [<span class="citation" data-cites="doi:10.1016/j.jsb.2005.07.007">@doi:10.1016/j.jsb.2005.07.007</span>]
was used to move a square of the cross-grating grid to the eucentric
position of the microscope. The effective defocus was then set to 2 µm,
using the “autofocus” routine of SerialEM. The objective focus of the
microscope was changed until no fringes were visible. The stage was then
moved in Z until images had an apparent defocus of 2 µm. The difference
in stage Z-position between the eucentric and fringe-free conditions was
used to move other areas into fringe-free condition.</p>
<p>Low magnification montages were used to find lamellae and lamellae
that were sufficiently thin and free of contamination were selected for
automated data collection. Overview images of each lamella were taken at
2250x magnification (38 Å pixel size). The corners of the lamella in the
overview image were manually annotated in SerialEM and translated into
beam image-shift values using SerialEM’s calibration. A hexagonal
pattern of beam image-shift positions was calculated that covered the
area between the four corners in a serpentine way, with a <del><span class="math inline">$\sqrt{3}\cdot425$</span></del><ins><span class="math inline">$\sqrt{3}\cdot 0.95 \cdot r$</span></ins> nm
horizontal spacing and <del><span class="math inline">3/4 ⋅ 850</span></del><ins><span class="math inline">3/2 ⋅ 0.95 ⋅ <em>r</em></span></ins> nm vertical
<del>spacing.</del><ins>spacing, where </ins><del>\(\)</del><ins><span class="math inline"><em>r</em></span> is the radius of the illuminated
area. For the fringe-free focus condition r was 250 nm and for the
eucentric focus r was 230 nm. This resulted in a fractional overlap of
adjacent tiles of 10.3% for the fringe-free focus condition and 12.9 %
for the eucentric focus condition.</ins> Exposures were taken at each
position with a 30 e<sup>-</sup>/Å<span class="math inline"><sup>2</sup></span> total dose. After each exposure,
the defocus was estimated using the ctffind function of SerialEM and the
focus for the next exposure was corrected by the difference between the
estimated focus and the desired defocus of 800 nm. Furthermore, after
each exposure the deviation of the beam from the center of the camera
was measured and corrected using the “CenterBeamFromImage” command of
SerialEM.</p>
<p>After data collection, a 20 s exposure at 2250x magnification of the
lamella at 200 μm defocus was taken for visualization purposes. A Python
script implementing this procedure is available at <a href="https://github.com/jojoelfe/deco_lace_template_matching_manuscript" class="uri">https://github.com/jojoelfe/deco_lace_template_matching_manuscript</a>.</p>
<h3 id="deco-lace-data-processing">DeCo-LACE data processing</h3>
<p>An overview of the data analysis pipeline is shown in Fig. {<span class="citation" data-cites="fig:deco_lace_workflow">@fig:deco_lace_workflow</span>}.</p>
<h4 id="pre-processing">Pre-processing</h4>
<p>Motion-correction, dose weighting and other preprocessing as detailed
below was performed using <em>cis</em>TEM [<span class="citation" data-cites="doi:10.7554/eLife.35383">@doi:10.7554/eLife.35383</span>].
To avoid influence of the beam-edge on motion-correction, only a quarter
of the movie in the center of the camera was considered for calculation
of the estimated motion. After movie frames were aligned and summed, a
mask for the illuminated area was calculated by lowpass filtering the
image with a 100 Å resolution cutoff, thresholding the image at 10% of
the maximal value and then lowpass filtering the mask again with a 100 Å
resolution cutoff to smooth the mask edges. This mask was then used to
select dark areas in the image and fill the pixels with Gaussian noise,
with the same mean and standard deviation as the illuminated area. A
custom version of the unblur program [<span class="citation" data-cites="doi:10.7554/eLife.06980">@doi:10.7554/eLife.06980</span>]
implementing this procedure is available at [link to decolace branch].
During motion correction images were resampled to a pixel size of 1.5 Å.
The contrast-transfer function (CTF) was estimated using ctffind [<span class="citation" data-cites="doi:10.1016/j.jsb.2015.08.008">@doi:10.1016/j.jsb.2015.08.008</span>],
searching between 0.2 and 2 μm defocus.</p>
<h4 id="dtm">2DTM</h4>
<p>The search template was generated from the atomic model of the mouse
LSU (PDB 6SWA, exluding the Epb1 subunit) using the cryo-EM simulator
implemented in <em>cis</em>TEM [<span class="citation" data-cites="doi:10.1107/S2052252521008538">@doi:10.1107/S2052252521008538</span>].
The match_template program [<span class="citation" data-cites="doi:10.7554/eLife.68946">@doi:10.7554/eLife.68946</span>]
was used to search for this template in the movie-aligned,
exposure-filtered and masked images, using a 1.5° angular step in
out-of-plane angles and a 1.0° angular step in-plane. 11 defocus planes
in 20 nm steps centered around the ctffind-determined defocus were
searched. Targets were defined as detected when their matches with the
template produced peaks with a singal-to-noise ratio (SNR) above a
threshold of 7.75, which was chosen based on the
one-false-positive-per-tile criterion [<span class="citation" data-cites="doi:10.7554/eLife.25648">@doi:10.7554/eLife.25648</span>].</p>
<h4 id="montage-assembly">Montage assembly</h4>
<p>The coordinates of each tile <span class="math inline"><em>i</em></span>, <span class="math inline"><strong>c</strong><sub><em>i</em></sub></span> [2D
Vector in pixels] were initialized using beam image-shift of the tile,
<span class="math inline"><strong>b</strong><sub><em>i</em></sub></span>
[2D Vector in μm], and the ISToCamera matrix <span class="math inline"><strong>I</strong><strong>C</strong></span>, as
calibrated by SerialEM:</p>
<p><span class="math display"><strong>c</strong><sub><em>i</em></sub> = <strong>I</strong><strong>C</strong> ⋅ <strong>b</strong><sub><em>i</em></sub></span></p>
<p>A list of tile pairs <span class="math inline"><em>i</em>, <em>j</em></span> that overlap were
assembled by selecting images where <span class="math inline">|<strong>c</strong><sub><em>i</em></sub>−<strong>c</strong><sub><em>j</em></sub>| &lt; <em>D</em><sub><em>B</em><em>e</em><em>a</em><em>m</em></sub></span>.
In order to calculate the precise offset between tiles <span class="math inline"><em>i</em></span> and <span class="math inline"><em>j</em></span>, <span class="math inline"><strong>r</strong><sub><em>i</em>, <em>j</em></sub></span>,
we calculated the cross-correlation between the two tiles, masked to the
overlapping illuminated area using the scikit-image package [<span class="citation" data-cites="doi:10.7717/peerj.453">@doi:10.7717/peerj.453</span>] was
used to calculate refined offsets . The coordinates <span class="math inline"><strong>c</strong><sub><em>i</em></sub></span> were
then refined by a least-square minimization against <span class="math inline"><strong>r</strong><sub><em>i</em>, <em>j</em></sub></span>:</p>
<p><span class="math display">min<sub><strong>c</strong></sub>∑<sub><em>p</em><em>a</em><em>i</em><em>r</em><em>s</em></sub>(<strong>r</strong><sub><em>i</em>, <em>j</em></sub>−(<strong>c</strong><sub><em>i</em></sub>−<strong>c</strong><sub><em>j</em></sub>))<sup>2</sup></span></p>
<p>using the scipy package [<span class="citation" data-cites="doi:10.1038/s41592-019-0686-2">@doi:10.1038/s41592-019-0686-2</span>].
The masked cross-correlation and the least-square minimization was
repeated once more to arrive at the final tile alignment.</p>
<p>The x,y coordinates of target <span class="math inline"><em>n</em></span> detected by 2DTM in the tile <span class="math inline"><em>i</em></span>, <span class="math inline"><strong>m</strong><sub><em>n</em>, <em>i</em></sub><sup>T</sup></span>,
was transformed into the montage frame by adding the coordinate of the
tile.</p>
<p><span class="math display"><strong>m</strong><sub><em>n</em></sub><sup>M</sup> = <strong>m</strong><sub><em>n</em>, <em>i</em></sub><sup>T</sup> + <strong>c</strong><sub><em>i</em></sub></span></p>
<p>The z coordinate of each target was calculated as the sum of the
defocus offset for the target, the estimated defocus of the tile, and
the nominal defocus of the microscope when the tile was acquired.</p>
<p>Images were rendered using UCSF ChimeraX [<span class="citation" data-cites="doi:10.1002/pro.3943">@doi:10.1002/pro.3943</span>] using a
custom extension to render 2DTM results available at <a href="https://github.com/jojoelfe/tempest" class="uri">https://github.com/jojoelfe/tempest</a>. The Python scripts
used for data processing are available under <a href="https://github.com/jojoelfe/deco_lace_template_matching_manuscript" class="uri">https://github.com/jojoelfe/deco_lace_template_matching_manuscript</a>.</p>
<h2 id="acknowledgments">Acknowledgments</h2>
<p>The authors would like to thank Bronwyn Lucas, Carsten Sachse, and
Chen Xu for helpful suggestions and careful reading of the manuscriptas
as well as members of the Grigorieff lab for helpful discussions. Data
was collected at the UMass Chan Medical School Cryo-EM core with help by
Kankang Song, Christna Ouch, and Chen Xu.</p>
<h2 id="data-availability">Data availability</h2>
<p>Cryo-EM movies, motion-corrected images and 2DTM results have been
deposited in EMPIAR under accession code [will be inserted]. The custom
cisTEM version is available under <a href="https://github.com/jojoelfe/cisTEM/tree/2574dbdf6161658fd177660b3a841100a792f61b" class="uri">https://github.com/jojoelfe/cisTEM/tree/2574dbdf6161658fd177660b3a841100a792f61b</a>
until features have been integrated into a cisTEM release. The ChimeraX
extension for rendering is available under <a href="https://github.com/jojoelfe/tempest" class="uri">https://github.com/jojoelfe/tempest</a>. This manuscript was
prepared using the manubot package [<span class="citation" data-cites="doi:10.1371/journal.pcbi.1007128">@doi:10.1371/journal.pcbi.1007128</span>]
. The corresponding repository containing all scripts used for figure
generation can be found under <a href="https://github.com/jojoelfe/deco_lace_template_matching_manuscript" class="uri">https://github.com/jojoelfe/deco_lace_template_matching_manuscript</a>.</p>
<h2 id="conflicts-of-interest">Conflicts of interest</h2>
<p>The Authors declare that there is no conflict of interest.</p>
<h2 id="figures">Figures</h2>
<figure>
<p><img src="tikz:initmatching" id="fig:initmatching" alt="2D template matching of the large subunit of the ribosome in fib-milled neutrophil-like cells (A) Overview image of the lamella. Major cellular regions are labeled, as Nucleus (Nuc), Mitochondria (M), and granular cytoplasm (GrCyt). FOVs where high-magnification images for template matching where acquired are indicated as boxes with the number of detected targets indicated on the bottom right. FOVs displayed in Panels B-E are color-coded. Scalebar corresponds to 1 μm. (B-E) FOVs with projection of detected LSUs shown in cyan. (B) Perinuclear region, the only detected targets are in the cytoplasmic half. (C) Cytoplasmic region with high density of ribosomes (D) Mitochondrium, as expected there are only detected LSUs in the cytoplasmic region (E) Cytoplasm, with low density of ribosomes." /></p>
<figcaption aria-hidden="true">
<p>2D template matching of the large subunit of the ribosome in
fib-milled neutrophil-like cells (A) Overview image of the lamella.
Major cellular regions are labeled, as Nucleus (Nuc), Mitochondria (M),
and granular cytoplasm (GrCyt). FOVs where high-magnification images for
template matching where acquired are indicated as boxes with the number
of detected targets indicated on the bottom right. FOVs displayed in
Panels B-E are color-coded. Scalebar corresponds to 1 μm. (B-E) FOVs
with projection of detected LSUs shown in cyan. (B) Perinuclear region,
the only detected targets are in the cytoplasmic half. (C) Cytoplasmic
region with high density of ribosomes (D) Mitochondrium, as expected
there are only detected LSUs in the cytoplasmic region (E) Cytoplasm,
with low density of ribosomes.</p>
</figcaption>
</figure>
<pre class="tikz-figure"><code>\node (anchor) at (0.5,9.35) {}; 
\node[labelNode] {A};
\node[graphicNode] (panel) {\includegraphics[height=9cm]{content/images/the_lamella.png}};

\node[annotNode] at (3.5,8.35) {\contour{white}{Nuc}};
\node[annotNode] at (1.75,4.0) {\contour{white}{Nuc}};

\node[annotNode] at (3.2,4.7) {\contour{white}{M}};
\node[annotNode] at (2.75,2.25) {\contour{white}{M}};

\node[annotNode] at (1.5,8.0) {\contour{white}{GrCyt}};
\node[annotNode] at (4.5,3.5) {\contour{white}{GrCyt}};

% scalebar - physical height is 12810 nm
\draw [line width=1.55mm,white] ([xshift=-0.05cm, yshift=0.1cm]panel.south east) -- ([xshift=-1.25cm  , yshift=0.1cm]panel.south east);
\draw [line width=0.7mm,black] ([xshift=-0.05cm, yshift=0.1cm]panel.south east) -- ([xshift=-1.25cm , yshift=0.1cm]panel.south east);

\node (rect) at (1.1,7.5) [anchor=center,draw,thick,minimum width=0.85cm,minimum height=0.6cm,color=blue!60] {};
\node[mnNode, color=blue!60] {\contour{white}{2}};

\node (rect) at (3.2,7.3) [anchor=center,draw,thick,minimum width=0.85cm,minimum height=0.6cm,color=blue!60] {};
\node[mnNode, color=blue!60] {\contour{white}{0}};

\node (rect) at (4.5,6.25) [anchor=center,draw,thick,minimum width=0.85cm,minimum height=0.6cm,color=red] {};
\node[mnNode, color=red] {\contour{white}{506}};

\node (rect) at (3.4,5.5) [anchor=center,draw,thick,minimum width=0.85cm,minimum height=0.6cm,color=blue!60] {};
\node[mnNode, color=blue!60] {\contour{white}{16}};

\node (rect) at (2.75,3.5) [anchor=center,draw,thick,minimum width=0.85cm,minimum height=0.6cm,color=green] {};
\node[mnNode, color=green] {\contour{white}{30}};

\node (rect) at (2.65,2.35) [anchor=center,draw,thick,minimum width=0.85cm,minimum height=0.6cm,color=orange] {};
\node[mnNode, color=orange] {\contour{white}{17}};

\node (rect) at (1.25,1.5) [anchor=center,draw,thick,minimum width=0.85cm,minimum height=0.6cm,color=blue!60] {};
\node[mnNode, color=blue!60] {\contour{white}{8}};

\node (rect) at (4.0,2.2) [anchor=center,draw,thick,minimum width=0.85cm,minimum height=0.6cm,color=magenta] {};
\node[mnNode, color=magenta] {\contour{white}{52}};


\node (anchor) at (6.5,9.35) {}; 
\node[labelNode] {B};
\node (img) [graphicNode]  {\includegraphics[height=4cm]{content/images/30_matches_m.png}};
\node (img) [graphicNode]  {\includegraphics[height=4cm]{content/images/30_matches.png}};
\node (n3) [box=green, fit=(img)] {};

\node (anchor) at (12.75,9.35) {}; 
\node[labelNode] {C};
\node (img) [graphicNode]  {\includegraphics[height=4cm]{content/images/506_matches_m.png}};
\node (img) [graphicNode]  {\includegraphics[height=4cm]{content/images/506_matches.png}};
\node (n3) [box=red, fit=(img)] {};

\node (anchor) at (6.5,4.75) {}; 
\node[labelNode] {D};
\node (img) [graphicNode]  {\includegraphics[height=4cm]{content/images/17_matches_m.png}};
\node (img) [graphicNode]  {\includegraphics[height=4cm]{content/images/17_matches.png}};
\node (n3) [box=orange, fit=(img)] {};

\node (anchor) at (12.75,4.75) {}; 
\node[labelNode] {E};
\node (img) [graphicNode]  {\includegraphics[height=4cm]{content/images/52_matches_m.png}};
\node (img) [graphicNode]  {\includegraphics[height=4cm]{content/images/52_matches.png}};
\node (n3) [box=magenta, fit=(img)] {};

%%\iftoggle{draft}{\node [redAnchorNode] {};};</code></pre>
<figure>
<p><img src="tikz:initmatching2" id="fig:initmatching2" data-tag="2 - figure supplement 1" alt="2D template matching of the large subunit of the ribosome in fib-milled neutrophil-like cells (A) Maximum intensity projection (MIP) cross-correlation map of micrograph shown in Figure {@fig:initmatching} (B+C) 3D plot of MIP regions indicated by color boxes in Panel A" /></p>
<figcaption aria-hidden="true">
<p>2D template matching of the large subunit of the ribosome in
fib-milled neutrophil-like cells (A) Maximum intensity projection (MIP)
cross-correlation map of micrograph shown in Figure {<span class="citation" data-cites="fig:initmatching">@fig:initmatching</span>}
(B+C) 3D plot of MIP regions indicated by color boxes in Panel A</p>
</figcaption>
</figure>
<pre class="tikz-figure"><code>\node (anchor) at (0.5,8.35) {}; 

\node at ([shift={(-0.3cm,0.9cm)}]anchor) [anchor=north west] {\includegraphics[width=12cm]{content/code/img/initial_map_match.pdf}};
\node[labelNode] {A};
\iftoggle{draft}{\node [redAnchorNode] {};};
\node (anchor) at (8.5,8.35) {}; 
\node[labelNode] {B};
\node (anchor) at (8.5,4.85) {}; 
\node[labelNode] {C};</code></pre>
<figure>
<p><img src="tikz:approach" id="fig:approach" alt="DeCo-LACE approach (A) Graphic demonstrating the data-collection strategy for DeCo-LACE. The electron beam is condensed to a diameter D_{Beam} that allows captured of the whole illuminated area on the camera. Beam-image shift along X and Y (BIS_X,$BIS_Y) is used to scan the whole lamella (B) Diagram of the collection algorithm (C) Example overview image of a lamella with the designated acquisition positions and the used beam diameter indicated with red circles. Scalebar corresponds to 1 μm. (D+E) Representative micrographs takne with a condensed beam at eucentric focus (D) or fringe-free focus (E). Scalebar corresponds to 100 nm. (F) Boxplot of defocus measured by ctffind of micrographs taken by the DeCo-LACE approach on four lamellae images at eucentric focus and four lamellae imaged with fringe-free focus." /></p>
<figcaption aria-hidden="true">
<p>DeCo-LACE approach (A) Graphic demonstrating the data-collection
strategy for DeCo-LACE. The electron beam is condensed to a diameter
<span class="math inline"><em>D</em><sub><em>B</em><em>e</em><em>a</em><em>m</em></sub></span>
that allows captured of the whole illuminated area on the camera.
Beam-image shift along X and Y (<span class="math inline"><em>B</em><em>I</em><em>S</em><sub><em>X</em></sub></span>,$<span class="math inline"><em>B</em><em>I</em><em>S</em><sub><em>Y</em></sub></span>)
is used to scan the whole lamella (B) Diagram of the collection
algorithm (C) Example overview image of a lamella with the designated
acquisition positions and the used beam diameter indicated with red
circles. Scalebar corresponds to 1 μm. (D+E) Representative micrographs
takne with a condensed beam at eucentric focus (D) or fringe-free focus
(E). Scalebar corresponds to 100 nm. (F) Boxplot of defocus measured by
ctffind of micrographs taken by the DeCo-LACE approach on four lamellae
images at eucentric focus and four lamellae imaged with fringe-free
focus.</p>
</figcaption>
</figure>
<pre class="tikz-figure"><code>\node (anchor) at (0.5,9.35) {}; 
\node[labelNode] {A};
\node[graphicNode] {\includegraphics[width=5cm]{content/graphics/approach/approach.png}};
\draw [densely dotted, red,thick] (3,9.5) -- (3,5.0);
\draw [densely dotted, red,thick] (3,4.2) -- (3,1.4);

\tikzset{dimen/.style={&lt;-&gt;,&gt;=latex,thin,every rectangle node/.style={above right=0.2cm and -0.3cm,fill=white,midway,font=\sffamily}}}

\draw (3.55,5.4) -- ++(0,1.5) coordinate (D1) -- +(0,5pt);
\draw (4.15,5.4) -- ++(0,1.5) coordinate (D2) -- +(0,5pt);
\draw [dimen] (D1) -- (D2) node {$D_{Beam}$};

\draw [-&gt;, very thick] (3,5) -- ++(0.5,-0.2)  coordinate (S1) node[midway,below left] {$BIS_X$} ;
\draw [-&gt;, very thick] (S1) -- (3.85,5.4) node[midway,below right] {$BIS_Y$};



\node (anchor) at (6.0,9.35) {}; 
\node[labelNode] {B};

% Place nodes
\node [block] (init) at (8.0,8.35) {\textbf{START} \\ Collect overview image};
\node [block, below = 0.3cm of init.south] (identify) {Setup hexagonal grid covering lamella area};
\node [block, below = 0.3cm of identify.south] (mutate) {Beam-imageshift to tile n=i};
\node [block, below = 0.3cm of mutate.south] (transfer) {Capture exposure and determine defocus by CTF fitting};
\node [block, below = 0.3cm of transfer.south] (evaluate) {Adjust defocus by difference between expected and measured value};

% Draw edges
\path [line] (init) -- (identify);
\path [line] (identify) -- (mutate);
\path [line] (mutate) -- (transfer);
\path [line] (transfer) -- (evaluate);
\path [line] (evaluate) -|  ([xshift=0.5cm, yshift=0cm]transfer.east) |- (mutate);

\node (anchor) at (11.5,9.35) {}; 
\node[labelNode] {C};
% X axis is 9.4um
\node[graphicNode] (image) {\includegraphics[width=8cm]{content/code/img/acquisition_positions.png}};
\draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]image.south west) -- ([xshift=1.35cm, yshift=0.3cm]image.south west);
\draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]image.south west) -- ([xshift=1.35cm, yshift=0.3cm]image.south west);
\iftoggle{draft}{\node [redAnchorNode] {};};


\node (anchor) at (0.5,-0.5) {}; 
\node[labelNode] {D};
\node[graphicNode] (euc) {\includegraphics[height=4.0cm]{content/images/fringebeam.png}};
\node[anchor=center] at ([shift={(0.0cm,0.3cm)}]euc.north) {Eucentric Focus};
\draw [line width=1.25mm,white] ([xshift=-0.05cm, yshift=0.1cm]euc.south east) -- ([xshift=-0.8cm, yshift=0.1cm]euc.south east);
\iftoggle{draft}{\node [redAnchorNode] {};};


\node (anchor) at (5.5,-0.5) {}; 
\node[labelNode] {E};
\node[graphicNode] (fff) {\includegraphics[height=4.0cm]{content/images/nofringebeam.png}};
\node[anchor=center] at ([shift={(0.0cm,0.3cm)}]fff.north) {Fringe-free Focus};
\draw [line width=1.25mm,white] ([xshift=-0.05cm, yshift=0.1cm]fff.south east) -- ([xshift=-0.8cm, yshift=0.1cm]fff.south east);
\iftoggle{draft}{\node [redAnchorNode] {};};

\node (anchor) at (10.5,-0.5) {}; 
\node[labelNode] {F};
\node[anchor=north west, below right=-0.5cm and 0cm of anchor] {\includegraphics[height=4.8cm]{content/code/img/defocusplot.pdf}};
\iftoggle{draft}{\node [redAnchorNode] {};};</code></pre>
<figure>
<p><img src="tikz:lamella_spatial_info" id="fig:lamella_spatial_info" data-tag="3 - figure supplement 1" alt="Defocus estimation of individual tiles of DeCo-LACE montages (A) Defocus values of individual micrographs taken using the DeCo-LACE approach plotted as a function of the beam image-shift values. (B) Defocus astigmatism of individual micrographs taken using the DeCo-LACE approach plotted as a function of the beam image-shift values." /></p>
<figcaption aria-hidden="true">
<p>Defocus estimation of individual tiles of DeCo-LACE montages (A)
Defocus values of individual micrographs taken using the DeCo-LACE
approach plotted as a function of the beam image-shift values. (B)
Defocus astigmatism of individual micrographs taken using the DeCo-LACE
approach plotted as a function of the beam image-shift values.</p>
</figcaption>
</figure>
<pre class="tikz-figure"><code>\newlength{\panelheight}
\setlength{\panelheight}{4.6cm}


\node (anchor) at (0.5,9.5) {}; 
\node[labelNode] {A};
\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/defocus_defocus_vs_bs_plot_euc_lamella1.png}};
\node (anchor) at ([shift={(-0.4cm,0.0cm)}]panel.north east) {}; 
\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/defocus_defocus_vs_bs_plot_euc_lamella2.png}};
\node (anchor) at ([shift={(-0.4cm,0.0cm)}]panel.north east) {}; 
\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/defocus_defocus_vs_bs_plot_euc_lamella3.png}};
\node (anchor) at ([shift={(-0.4cm,0.0cm)}]panel.north east) {}; 
\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/defocus_defocus_vs_bs_plot_euc_lamella4.png}};

\node (anchor) at ([shift={(0.0cm,-0.5cm)}]0.5,0 |- panel.south) {}; 
\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/defocus_defocus_vs_bs_plot_fff_lamella1.png}};
\node (anchor) at ([shift={(-0.4cm,0.0cm)}]panel.north east) {}; 
\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/defocus_defocus_vs_bs_plot_fff_lamella2.png}};
\node (anchor) at ([shift={(-0.4cm,0.0cm)}]panel.north east) {}; 
\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/defocus_defocus_vs_bs_plot_fff_lamella3.png}};
\node (anchor) at ([shift={(-0.4cm,0.0cm)}]panel.north east) {}; 
\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/defocus_defocus_vs_bs_plot_fff_lamella4.png}};
\node (anchor) at ([shift={(-0.4cm,4.0cm)}]panel.north east) {}; 
\node[graphicNode] (key) {\includegraphics[height=\panelheight]{content/code/img/defocus_defocus_vs_bs_plot_colorbar.png}};

\node (anchor) at ([shift={(0.0cm,-1.0cm)}]0.5,0 |- panel.south) {}; 
\node[labelNode] {B};
\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/defocus_astigmatism_vs_bs_plot_euc_lamella1.png}};
\node (anchor) at ([shift={(-0.4cm,0.0cm)}]panel.north east) {}; 
\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/defocus_astigmatism_vs_bs_plot_euc_lamella2.png}};
\node (anchor) at ([shift={(-0.4cm,0.0cm)}]panel.north east) {}; 
\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/defocus_astigmatism_vs_bs_plot_euc_lamella3.png}};
\node (anchor) at ([shift={(-0.4cm,0.0cm)}]panel.north east) {}; 
\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/defocus_astigmatism_vs_bs_plot_euc_lamella4.png}};

\node (anchor) at ([shift={(0.0cm,-0.5cm)}]0.5,0 |- panel.south) {}; 
\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/defocus_astigmatism_vs_bs_plot_fff_lamella1.png}};
\node (anchor) at ([shift={(-0.4cm,0.0cm)}]panel.north east) {}; 
\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/defocus_astigmatism_vs_bs_plot_fff_lamella2.png}};
\node (anchor) at ([shift={(-0.4cm,0.0cm)}]panel.north east) {}; 
\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/defocus_astigmatism_vs_bs_plot_fff_lamella3.png}};
\node (anchor) at ([shift={(-0.4cm,0.0cm)}]panel.north east) {}; 
\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/defocus_astigmatism_vs_bs_plot_fff_lamella4.png}};
\node (anchor) at ([shift={(-0.4cm,4.0cm)}]panel.north east) {}; 
\node[graphicNode] (key) {\includegraphics[height=\panelheight]{content/code/img/defocus_astigmatism_vs_bs_plot_colorbar.png}};</code></pre>
<figure>
<p><img src="tikz:assembly" id="fig:assembly" alt="Assembling DeCo-LACE exposures into montages (A) Overview image of LamellaEUC 1 taken at low magnification. Scalebar corresponds to 1 μm. (B) Overview of LamellaEUC 1 created by montaging high magnification images taken with the DeCo-LACE approach. Scalebar corresponds to 1 μm. (C) Zoom-in into red box in panel B. Slight beam-fringe artifacts are visible. Scalebar corresponds to 100 nm. (D) Overview image of LamellaFFF 4 taken at low magnification. Scalebar corresponds to 1 μm. (E) Overview of LamellaFFF 4 created by montaging high magnification images taken with the DeCo-LACE approach. Scalebar corresponds to 1 μm. (F) Zoom-in into red box in panel E. No beam-fringe artifacts are visible. Scalebar corresponds to 100 nm." /></p>
<figcaption aria-hidden="true">
<p>Assembling DeCo-LACE exposures into montages (A) Overview image of
Lamella<sub>EUC</sub> 1 taken at low magnification. Scalebar corresponds
to 1 μm. (B) Overview of Lamella<sub>EUC</sub> 1 created by montaging
high magnification images taken with the DeCo-LACE approach. Scalebar
corresponds to 1 μm. (C) Zoom-in into red box in panel B. Slight
beam-fringe artifacts are visible. Scalebar corresponds to 100 nm. (D)
Overview image of Lamella<sub>FFF</sub> 4 taken at low magnification.
Scalebar corresponds to 1 μm. (E) Overview of Lamella<sub>FFF</sub> 4
created by montaging high magnification images taken with the DeCo-LACE
approach. Scalebar corresponds to 1 μm. (F) Zoom-in into red box in
panel E. No beam-fringe artifacts are visible. Scalebar corresponds to
100 nm.</p>
</figcaption>
</figure>
<pre class="tikz-figure"><code>\def\panelheight{6.0cm}

\node (anchor) at (0.5,13.0) {}; 
\node[labelNode] {A};
\node[graphicNode] (img) {\includegraphics[height=\panelheight,trim=0.0 0 0cm 0, clip]{content/images/euc_lamella1_view.png}};
% Title
\node[anchor=center,align=center] at ([shift={(0.0cm,0.6cm)}]img.north) {\baselineskip=8pt   \\ 38Å pixel size \\ 300μm defocus};
% scalebar - physical height is 12810 nm
\draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]img.south west) -- ([xshift=0.5cm + \panelheight / 12.81 , yshift=0.3cm]img.south west);
\draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]img.south west) -- ([xshift=0.5cm + \panelheight / 12.81, yshift=0.3cm]img.south west);

\node (anchor) at (5.5,13.0) {}; 
\node[labelNode] {B};
\node[graphicNode] (img) {\includegraphics[height=6.0cm,trim=0 0 0cm 0, clip]{content/images/euc_lamella1_assm.png}};
\node[anchor=center,align=center] at ([shift={(0.0cm,0.7cm)}]img.north) {\baselineskip=8pt Eucentric Focus \\ 1.76Å pixel size \\ 800nm defocus};
% Red Box
\node (rect) at (6.52,10.78) [anchor=center,draw,thick,minimum width=0.375cm,minimum height=0.469cm,color=red] {};
% scalebar - physical height is 12810 nm
\draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]img.south west) -- ([xshift=0.5cm + \panelheight / 12.81 , yshift=0.3cm]img.south west);
\draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]img.south west) -- ([xshift=0.5cm + \panelheight / 12.81, yshift=0.3cm]img.south west);

\node (anchor) at (10.6,13.0) {}; 
\node[labelNode] {C};
% height is 750 nm
\node[graphicNode] (image) {\includegraphics[height=6.0cm]{content/images/euc_lam1_zoomin.png}};
\node (n3) [box=red, fit=(image)] {};
\draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]image.south west) -- ([xshift=1.3cm, yshift=0.3cm]image.south west);
\draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]image.south west) -- ([xshift=1.3cm, yshift=0.3cm]image.south west);


\node (anchor) at (0.5,5.0) {}; 
\node[labelNode] {D};
\node[graphicNode] (img) {\includegraphics[height=6.0cm,trim=0 0 0cm 0, clip]{content/images/fff_lamella4_view.png}};
\node[anchor=center,align=center] at ([shift={(0.0cm,0.6cm)}]img.north) {\baselineskip=8pt  \\ 38Å pixel size \\ 300μm defocus};
% scalebar - physical height is 9337.5 nm
\draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]img.south west) -- ([xshift=0.5cm + \panelheight / 9.3375 , yshift=0.3cm]img.south west);
\draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]img.south west) -- ([xshift=0.5cm + \panelheight / 9.3375, yshift=0.3cm]img.south west);


\node (anchor) at (5.8,5.0) {}; 
\node[labelNode] {E};
\node[graphicNode] (img) {\includegraphics[height=6.0cm,trim=0 0 0cm 0, clip]{content/images/fff_lamella4_assm.png}};
\node[anchor=center,align=center] at ([shift={(0.0cm,0.7cm)}]img.north) {\baselineskip=2pt Fringe-free Focus \\ 1.76Å pixel size \\ 800nm defocus};
\node (rect) at (8.738,3.70) [anchor=center,draw,thick,minimum width=0.375cm,minimum height=0.469cm,color=red] {};
% scalebar - physical height is 9337.5 nm
\draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]img.south west) -- ([xshift=0.5cm + \panelheight / 9.3375 , yshift=0.3cm]img.south west);
\draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]img.south west) -- ([xshift=0.5cm + \panelheight / 9.3375, yshift=0.3cm]img.south west);

\node (anchor) at (11.0,5.0) {}; 
\node[labelNode] {F};
% height is 750 nm
\node[graphicNode] (image) {\includegraphics[height=6.0cm]{content/images/fff_lam4_zoomin.png}};
\node (n3) [box=red, fit=(image)] {};
\draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]image.south west) -- ([xshift=1.3cm, yshift=0.3cm]image.south west);
\draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]image.south west) -- ([xshift=1.3cm, yshift=0.3cm]image.south west);</code></pre>
<figure>
<p><img src="tikz:crop_unblur" id="fig:crop_unblur" data-tag="4 - figure supplement 1" alt="Motion correction of movies with condensed beams. At the top of each panel is an average of the movie that was motion-corrected with a red dashed box indicating the region that was used to estimate shifts. Below is a graph indicating the estimated shifts of the individual frames of the movie. Below this is the MIP of 2DTM using the large subunit of the mouse ribosome. (A) Motion correction of the whole movie (B) Notion correction of a cropped region of the movie that eliminates the beam edges (C) Motion correction of the whole movie, using only the central region to estimate the shifts (D) Representative result of 2DTM (botttom) on uncorrected micrographs. Detections along the beam edge are artifacts due to the high-contrast signal. (E) Micrographs after preprocessing for 2DTM (top). The micrograph has been cropped to the perimeter of the beam and unilluminated areas of the image have been replace by Gaussian noise. After running 2DTM on this micrograph (bottom) no more artifactual detections are observed." /></p>
<figcaption aria-hidden="true">
<p>Motion correction of movies with condensed beams. At the top of each
panel is an average of the movie that was motion-corrected with a red
dashed box indicating the region that was used to estimate shifts. Below
is a graph indicating the estimated shifts of the individual frames of
the movie. Below this is the MIP of 2DTM using the large subunit of the
mouse ribosome. (A) Motion correction of the whole movie (B) Notion
correction of a cropped region of the movie that eliminates the beam
edges (C) Motion correction of the whole movie, using only the central
region to estimate the shifts<ins> (D) Representative result of 2DTM
(botttom) on uncorrected micrographs. Detections along the beam edge are
artifacts due to the high-contrast signal. (E) Micrographs after
preprocessing for 2DTM (top). The micrograph has been cropped to the
perimeter of the beam and unilluminated areas of the image have been
replace by Gaussian noise. After running 2DTM on this micrograph
(bottom) no more artifactual detections are observed.</ins></p>
</figcaption>
</figure>
<pre class="tikz-figure"><code>\def\panelheight{4.6cm}

\node (anchor) at (0.5,9.5) {}; 
\node[labelNode] {A};
\node[graphicNode] (panel) {\includegraphics[width=5cm]{content/images/crop_unblur_initial.png}};
\node (box1) [box=red, fit=(panel), dashed] {};

\node (anchor) at ([shift={(0.0,-0.5cm)}]panel.south west) {}; 
\node[graphicNode] (panel) {\includegraphics[width=5cm]{content/code/img/crop_unblur_motiontrace_initial.png}};

\node (anchor) at ([shift={(0.0,-0.5cm)}]panel.south west) {}; 
\node[graphicNode] (panel) {\includegraphics[width=5cm]{content/code/img/crop_unblur_mip_initial_alignment.png}};



\node (anchor) at ([shift={(1.5,0.0cm)}]0.5,9.5 -| panel.north east) {}; 
\node[labelNode] {B};
\node[graphicNode] (panel) {\includegraphics[width=5cm]{content/images/crop_unblur_cropped.png}};
\node (box2) [box=red, fit=(panel), dashed] {};

\node (anchor) at ([shift={(0.0,-0.5cm)}]panel.south west) {}; 
\node[graphicNode] (panel) {\includegraphics[width=5cm]{content/code/img/crop_unblur_motiontrace_cropped.png}};

\node (anchor) at ([shift={(0.0,-0.5cm)}]panel.south west) {}; 
\node[graphicNode] (panel) {\includegraphics[width=5cm]{content/code/img/crop_unblur_mip_cropped_alignment.png}};

\node (anchor) at ([shift={(1.5,0.0cm)}]0.5,9.5 -| panel.north east) {}; 
\node[labelNode] {C};
\node[graphicNode] (panel) {\includegraphics[width=5cm]{content/images/crop_unblur_initial.png}};
\draw[red, dashed, line width=0.8mm] ([shift={(-0.7,-1.2cm)}]panel.north east) -- 
                                     ([shift={(0.7,-1.2cm)}]panel.north west) --
                                     ([shift={(0.7,1.2cm)}]panel.south west) --
                                     ([shift={(-0.7,1.2cm)}]panel.south east) --
                                     ([shift={(-0.7,-1.2cm)}]panel.north east);

\node (anchor) at ([shift={(0.0,-0.5cm)}]panel.south west) {}; 
\node[graphicNode] (panel) {\includegraphics[width=5cm]{content/code/img/crop_unblur_motiontrace_final.png}};

\node (anchor) at ([shift={(0.0,-0.5cm)}]panel.south west) {}; 
\node[graphicNode] (panel) {\includegraphics[width=5cm]{content/code/img/crop_unblur_mip_final_alignment.png}};

\node (anchor) at ([shift={(0.0,-1.0cm)}]0.5,9.5 |- panel.south east) {}; 
\node[labelNode] {D};
\node[graphicNode] (panel) {\includegraphics[height=5cm]{content/images/deco_unblur/zap000.png}};
\node (anchor) at ([shift={(0.0,-0.5cm)}]panel.south west) {}; 
\node[graphicNode] (panel2) {\includegraphics[height=5cm]{content/images/deco_unblur/zap002.png}};

\node (anchor) at ([shift={(1.5,0.0cm)}] panel.north east) {}; 
\node[labelNode] {E};
\node[graphicNode] (panel) {\includegraphics[height=5cm]{content/images/deco_unblur/zap001.png}};
\node (anchor) at ([shift={(0.0,-0.5cm)}]panel.south west) {}; 
\node[graphicNode] (panel) {\includegraphics[height=5cm]{content/images/deco_unblur/zap004.png}};</code></pre>
<figure>
<p><img src="tikz:lamella_motion_thickness" id="fig:lamella_motion_thickness" data-tag="4 - figure supplement 2" alt="Motion correction of individual tiles imaged using the DeCo-LACE approach (A) Total estimated motion of individual micrographs taken using the DeCo-LACE approach plotted as a function of the beam image-shift values. (B) Electron intensity of individual micrographs taken using the DeCo-LACE approach plotted as a function of the beam image-shift values." /></p>
<figcaption aria-hidden="true">
<p>Motion correction of individual tiles imaged using the DeCo-LACE
approach (A) Total estimated motion of individual micrographs taken
using the DeCo-LACE approach plotted as a function of the beam
image-shift values. (B) Electron intensity of individual micrographs
taken using the DeCo-LACE approach plotted as a function of the beam
image-shift values.</p>
</figcaption>
</figure>
<div class="sourceCode" id="cb7"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a> \newlength{\panelheight}</span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a> \setlength{\panelheight}{4.6cm}</span>
<span id="cb7-3"><a href="#cb7-3" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb7-4"><a href="#cb7-4" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb7-5"><a href="#cb7-5" aria-hidden="true" tabindex="-1"></a> \node (anchor) at (0.5,9.5) {}; </span>
<span id="cb7-6"><a href="#cb7-6" aria-hidden="true" tabindex="-1"></a> \node[labelNode] {A};</span>
<span id="cb7-7"><a href="#cb7-7" aria-hidden="true" tabindex="-1"></a> \node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/movement_vs_bs_plot_euc_lamella1.png}};</span>
<span id="cb7-8"><a href="#cb7-8" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(-0.4,0.0cm)}]panel.north east) {}; </span>
<span id="cb7-9"><a href="#cb7-9" aria-hidden="true" tabindex="-1"></a> \node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/movement_vs_bs_plot_euc_lamella2.png}};</span>
<span id="cb7-10"><a href="#cb7-10" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(-0.4,0.0cm)}]panel.north east) {}; </span>
<span id="cb7-11"><a href="#cb7-11" aria-hidden="true" tabindex="-1"></a> \node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/movement_vs_bs_plot_euc_lamella3.png}};</span>
<span id="cb7-12"><a href="#cb7-12" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(-0.4,0.0cm)}]panel.north east) {}; </span>
<span id="cb7-13"><a href="#cb7-13" aria-hidden="true" tabindex="-1"></a> \node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/movement_vs_bs_plot_euc_lamella4.png}};</span>
<span id="cb7-14"><a href="#cb7-14" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb7-15"><a href="#cb7-15" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(0.0cm,-0.5cm)}]0.5,0 |- panel.south) {}; </span>
<span id="cb7-16"><a href="#cb7-16" aria-hidden="true" tabindex="-1"></a> \node[graphicNode]  (panel) {\includegraphics[height=\panelheight]{content/code/img/movement_vs_bs_plot_fff_lamella1.png}};</span>
<span id="cb7-17"><a href="#cb7-17" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(-0.4,0.0cm)}]panel.north east) {}; </span>
<span id="cb7-18"><a href="#cb7-18" aria-hidden="true" tabindex="-1"></a> \node[graphicNode]  (panel) {\includegraphics[height=\panelheight]{content/code/img/movement_vs_bs_plot_fff_lamella2.png}};</span>
<span id="cb7-19"><a href="#cb7-19" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(-0.4,0.0cm)}]panel.north east) {};</span>
<span id="cb7-20"><a href="#cb7-20" aria-hidden="true" tabindex="-1"></a> \node[graphicNode]  (panel) {\includegraphics[height=\panelheight]{content/code/img/movement_vs_bs_plot_fff_lamella3.png}};</span>
<span id="cb7-21"><a href="#cb7-21" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(-0.4,0.0cm)}]panel.north east) {}; </span>
<span id="cb7-22"><a href="#cb7-22" aria-hidden="true" tabindex="-1"></a> \node[graphicNode]  (panel) {\includegraphics[height=\panelheight]{content/code/img/movement_vs_bs_plot_fff_lamella4.png}};</span>
<span id="cb7-23"><a href="#cb7-23" aria-hidden="true" tabindex="-1"></a><span class="st">-\node (anchor) at ([shift={(-1.0cm,4.0cm)}]panel.north east) {};</span></span>
<span id="cb7-24"><a href="#cb7-24" aria-hidden="true" tabindex="-1"></a><span class="va">+\node (anchor) at ([shift={(-0.5cm,4.0cm)}]panel.north east) {}; </span></span>
<span id="cb7-25"><a href="#cb7-25" aria-hidden="true" tabindex="-1"></a><span class="va">+\begin{scope}[on background layer]</span></span>
<span id="cb7-26"><a href="#cb7-26" aria-hidden="true" tabindex="-1"></a> \node[graphicNode] (key) {\includegraphics[height=\panelheight]{content/code/img/movement_vs_bs_plot_colorbar.png}};</span>
<span id="cb7-27"><a href="#cb7-27" aria-hidden="true" tabindex="-1"></a><span class="va">+\end{scope}</span></span>
<span id="cb7-28"><a href="#cb7-28" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb7-29"><a href="#cb7-29" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(0.0cm,-1.0cm)}]0.5,0 |- panel.south) {}; </span>
<span id="cb7-30"><a href="#cb7-30" aria-hidden="true" tabindex="-1"></a> \node[labelNode] {B};</span>
<span id="cb7-31"><a href="#cb7-31" aria-hidden="true" tabindex="-1"></a><span class="st">-\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/thickness_by_intensity_vs_bs_plot_euc_lamella1.png}};</span></span>
<span id="cb7-32"><a href="#cb7-32" aria-hidden="true" tabindex="-1"></a><span class="va">+\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/thickness_by_intensity_vs_bs_plot_euc_lamella1_lblaw.png};</span></span>
<span id="cb7-33"><a href="#cb7-33" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(-0.4cm,0.0cm)}]panel.north east) {}; </span>
<span id="cb7-34"><a href="#cb7-34" aria-hidden="true" tabindex="-1"></a><span class="st">-\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/thickness_by_intensity_vs_bs_plot_euc_lamella2.png}};</span></span>
<span id="cb7-35"><a href="#cb7-35" aria-hidden="true" tabindex="-1"></a><span class="va">+\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/thickness_by_intensity_vs_bs_plot_euc_lamella2_lblaw.png};</span></span>
<span id="cb7-36"><a href="#cb7-36" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(-0.4cm,0.0cm)}]panel.north east) {}; </span>
<span id="cb7-37"><a href="#cb7-37" aria-hidden="true" tabindex="-1"></a><span class="st">-\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/thickness_by_intensity_vs_bs_plot_euc_lamella3.png}};</span></span>
<span id="cb7-38"><a href="#cb7-38" aria-hidden="true" tabindex="-1"></a><span class="va">+\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/thickness_by_intensity_vs_bs_plot_euc_lamella3_lblaw.png};</span></span>
<span id="cb7-39"><a href="#cb7-39" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(-0.4cm,0.0cm)}]panel.north east) {}; </span>
<span id="cb7-40"><a href="#cb7-40" aria-hidden="true" tabindex="-1"></a><span class="st">-\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/thickness_by_intensity_vs_bs_plot_euc_lamella4.png}};</span></span>
<span id="cb7-41"><a href="#cb7-41" aria-hidden="true" tabindex="-1"></a><span class="va">+\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/thickness_by_intensity_vs_bs_plot_euc_lamella4_lblaw.png};</span></span>
<span id="cb7-42"><a href="#cb7-42" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb7-43"><a href="#cb7-43" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(0.0cm,-0.5cm)}]0.5,0 |- panel.south) {}; </span>
<span id="cb7-44"><a href="#cb7-44" aria-hidden="true" tabindex="-1"></a><span class="st">-\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/thickness_by_intensity_vs_bs_plot_fff_lamella1.png}};</span></span>
<span id="cb7-45"><a href="#cb7-45" aria-hidden="true" tabindex="-1"></a><span class="va">+\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/thickness_by_intensity_vs_bs_plot_fff_lamella1_lblaw.png};</span></span>
<span id="cb7-46"><a href="#cb7-46" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(-0.4cm,0.0cm)}]panel.north east) {}; </span>
<span id="cb7-47"><a href="#cb7-47" aria-hidden="true" tabindex="-1"></a><span class="st">-\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/thickness_by_intensity_vs_bs_plot_fff_lamella2.png}};</span></span>
<span id="cb7-48"><a href="#cb7-48" aria-hidden="true" tabindex="-1"></a><span class="va">+\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/thickness_by_intensity_vs_bs_plot_fff_lamella2_lblaw.png};</span></span>
<span id="cb7-49"><a href="#cb7-49" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(-0.4cm,0.0cm)}]panel.north east) {}; </span>
<span id="cb7-50"><a href="#cb7-50" aria-hidden="true" tabindex="-1"></a><span class="st">-\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/thickness_by_intensity_vs_bs_plot_fff_lamella3.png}};</span></span>
<span id="cb7-51"><a href="#cb7-51" aria-hidden="true" tabindex="-1"></a><span class="va">+\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/thickness_by_intensity_vs_bs_plot_fff_lamella3_lblaw.png};</span></span>
<span id="cb7-52"><a href="#cb7-52" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(-0.4cm,0.0cm)}]panel.north east) {}; </span>
<span id="cb7-53"><a href="#cb7-53" aria-hidden="true" tabindex="-1"></a><span class="st">-\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/thickness_by_intensity_vs_bs_plot_fff_lamella4.png}};</span></span>
<span id="cb7-54"><a href="#cb7-54" aria-hidden="true" tabindex="-1"></a><span class="st">-\node (anchor) at ([shift={(-1.0cm,4.0cm)}]panel.north east) {};</span></span>
<span id="cb7-55"><a href="#cb7-55" aria-hidden="true" tabindex="-1"></a><span class="st">-\node[graphicNode] (key) {\includegraphics[height=\panelheight]{content/code/img/thickness_by_intensity_vs_bs_plot_colorbar.png}};</span></span>
<span id="cb7-56"><a href="#cb7-56" aria-hidden="true" tabindex="-1"></a><span class="va">+\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/thickness_by_intensity_vs_bs_plot_fff_lamella4_lblaw.png};</span></span>
<span id="cb7-57"><a href="#cb7-57" aria-hidden="true" tabindex="-1"></a><span class="va">+\node (anchor) at ([shift={(-0.5cm,4.0cm)}]panel.north east) {}; </span></span>
<span id="cb7-58"><a href="#cb7-58" aria-hidden="true" tabindex="-1"></a><span class="va">+\begin{scope}[on background layer]</span></span>
<span id="cb7-59"><a href="#cb7-59" aria-hidden="true" tabindex="-1"></a><span class="va">+\node[graphicNode] (key) {\includegraphics[height=\panelheight]{content/code/img/thickness_by_intensity_vs_bs_plot_colorbar_lblaw.png};</span></span>
<span id="cb7-60"><a href="#cb7-60" aria-hidden="true" tabindex="-1"></a><span class="va">+\end{scope}</span></span>
<span id="cb7-61"><a href="#cb7-61" aria-hidden="true" tabindex="-1"></a><span class="va">+\node (anchor) at ([shift={(0.0cm,-1.0cm)}]0.5,0 |- panel.south) {}; </span></span>
<span id="cb7-62"><a href="#cb7-62" aria-hidden="true" tabindex="-1"></a><span class="va">+\node[labelNode] {C};</span></span>
<span id="cb7-63"><a href="#cb7-63" aria-hidden="true" tabindex="-1"></a><span class="va">+\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/code/img/matchrange_vs_thickness_plot.png}};</span></span></code></pre></div>
<figure>
<p><img src="tikz:gain" id="fig:gain" data-tag="4 - figure supplement 3" alt="Averages of micrographs taken with a condensed beam over vacuum using a Gatan K3 detector. Contrast and Brightness have been adjusted to highlight uneven dose response. (A) Eucentric Focus (B) Fringe-free Focus" /></p>
<figcaption aria-hidden="true">
<p>Averages of micrographs taken with a condensed beam over vacuum using
a Gatan K3 detector. Contrast and Brightness have been adjusted to
highlight uneven dose response. (A) Eucentric Focus (B) Fringe-free
Focus</p>
</figcaption>
</figure>
<pre class="tikz-figure"><code>\newlength{\panelheight}
\setlength{\panelheight}{7cm}


\node (anchor) at (0.5,3.5) {}; 
\node[labelNode] {A};
\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/images/euc_gain.png}};

\node (anchor) at ([shift={(0.5cm,0.0cm)}]panel.north east) {}; 
\node[labelNode] {B};
\node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/images/fff_gain.png}};</code></pre>
<figure>
<p><img src="tikz:lamella_images" id="fig:lamella_images" data-tag="4 - figure supplement 4" style="height:23cm" alt="Overview images of lamellae imaged using the DeCo-LACE approach taken at low-magnification (A) Overviews taken at low magnification. Scalebar corresponds to 1 μm. (B) Overviews assembled using the DeCo-LACE approach. Scalebar corresponds to 1 μm. (C) Representative examples of a class of granules containing a putatively cytosolic inclusion. Scalebar corresponds to 100 nm." /></p>
<figcaption aria-hidden="true">
<p>Overview images of lamellae imaged using the DeCo-LACE approach taken
at low-magnification (A) Overviews taken at low magnification. Scalebar
corresponds to 1 μm. (B) Overviews assembled using the DeCo-LACE
approach. Scalebar corresponds to 1 μm. (C) Representative examples of a
class of granules containing a putatively cytosolic inclusion. Scalebar
corresponds to 100 nm.</p>
</figcaption>
</figure>
<div class="sourceCode" id="cb9"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a> \newlength{\panelheight}</span>
<span id="cb9-2"><a href="#cb9-2" aria-hidden="true" tabindex="-1"></a> \setlength{\panelheight}{4.4cm}</span>
<span id="cb9-3"><a href="#cb9-3" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb9-4"><a href="#cb9-4" aria-hidden="true" tabindex="-1"></a> \node (anchor) at (0.5,4.5) {}; </span>
<span id="cb9-5"><a href="#cb9-5" aria-hidden="true" tabindex="-1"></a> \node[labelNode] {A};</span>
<span id="cb9-6"><a href="#cb9-6" aria-hidden="true" tabindex="-1"></a> \node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/images/euc_lamella1_view.png}};</span>
<span id="cb9-7"><a href="#cb9-7" aria-hidden="true" tabindex="-1"></a> % Title</span>
<span id="cb9-8"><a href="#cb9-8" aria-hidden="true" tabindex="-1"></a> \node[anchor=center,align=center] at ([shift={(0.0cm,0.3cm)}]panel.north) {Lamella$_\textrm{EUC}$ 1};</span>
<span id="cb9-9"><a href="#cb9-9" aria-hidden="true" tabindex="-1"></a> % scalebar - physical height is 12810 nm</span>
<span id="cb9-10"><a href="#cb9-10" aria-hidden="true" tabindex="-1"></a> \draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 12.81 , yshift=0.3cm]panel.south west);</span>
<span id="cb9-11"><a href="#cb9-11" aria-hidden="true" tabindex="-1"></a> \draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 12.81, yshift=0.3cm]panel.south west);</span>
<span id="cb9-12"><a href="#cb9-12" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(0.5cm,0.0cm)}]panel.north east) {}; </span>
<span id="cb9-13"><a href="#cb9-13" aria-hidden="true" tabindex="-1"></a> \node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/images/euc_lamella2_view.png}};</span>
<span id="cb9-14"><a href="#cb9-14" aria-hidden="true" tabindex="-1"></a> % Title</span>
<span id="cb9-15"><a href="#cb9-15" aria-hidden="true" tabindex="-1"></a> \node[anchor=center,align=center] at ([shift={(0.0cm,0.3cm)}]panel.north) {Lamella$_\textrm{EUC}$ 2};</span>
<span id="cb9-16"><a href="#cb9-16" aria-hidden="true" tabindex="-1"></a> % scalebar - physical height is 7027.5 nm</span>
<span id="cb9-17"><a href="#cb9-17" aria-hidden="true" tabindex="-1"></a> \draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 7.0275 , yshift=0.3cm]panel.south west);</span>
<span id="cb9-18"><a href="#cb9-18" aria-hidden="true" tabindex="-1"></a> \draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 7.0275, yshift=0.3cm]panel.south west);</span>
<span id="cb9-19"><a href="#cb9-19" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(0.5cm,0.0cm)}]panel.north east) {}; </span>
<span id="cb9-20"><a href="#cb9-20" aria-hidden="true" tabindex="-1"></a> \node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/images/euc_lamella3_view.png}};</span>
<span id="cb9-21"><a href="#cb9-21" aria-hidden="true" tabindex="-1"></a> % Title</span>
<span id="cb9-22"><a href="#cb9-22" aria-hidden="true" tabindex="-1"></a> \node[anchor=center,align=center] at ([shift={(0.0cm,0.3cm)}]panel.north) {Lamella$_\textrm{EUC}$ 3};</span>
<span id="cb9-23"><a href="#cb9-23" aria-hidden="true" tabindex="-1"></a> % scalebar - physical height is 9615 nm</span>
<span id="cb9-24"><a href="#cb9-24" aria-hidden="true" tabindex="-1"></a> \draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 9.615 , yshift=0.3cm]panel.south west);</span>
<span id="cb9-25"><a href="#cb9-25" aria-hidden="true" tabindex="-1"></a> \draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 9.615, yshift=0.3cm]panel.south west);</span>
<span id="cb9-26"><a href="#cb9-26" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(0.5cm,0.0cm)}]panel.north east) {}; </span>
<span id="cb9-27"><a href="#cb9-27" aria-hidden="true" tabindex="-1"></a> \node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/images/euc_lamella4_view.png}};</span>
<span id="cb9-28"><a href="#cb9-28" aria-hidden="true" tabindex="-1"></a> % Title</span>
<span id="cb9-29"><a href="#cb9-29" aria-hidden="true" tabindex="-1"></a> \node[anchor=center,align=center] at ([shift={(0.0cm,0.3cm)}]panel.north) {Lamella$_\textrm{EUC}$ 4};</span>
<span id="cb9-30"><a href="#cb9-30" aria-hidden="true" tabindex="-1"></a> % scalebar - physical height is 8625 nm</span>
<span id="cb9-31"><a href="#cb9-31" aria-hidden="true" tabindex="-1"></a> \draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 8.625 , yshift=0.3cm]panel.south west);</span>
<span id="cb9-32"><a href="#cb9-32" aria-hidden="true" tabindex="-1"></a> \draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 8.625, yshift=0.3cm]panel.south west);</span>
<span id="cb9-33"><a href="#cb9-33" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb9-34"><a href="#cb9-34" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(0.0cm,-0.5cm)}]0.5,0 |- panel.south) {}; </span>
<span id="cb9-35"><a href="#cb9-35" aria-hidden="true" tabindex="-1"></a> \node[graphicNode]  (panel) {\includegraphics[height=\panelheight]{content/images/fff_lamella1_view.png}};</span>
<span id="cb9-36"><a href="#cb9-36" aria-hidden="true" tabindex="-1"></a> % Title</span>
<span id="cb9-37"><a href="#cb9-37" aria-hidden="true" tabindex="-1"></a> \node[anchor=center,align=center] at ([shift={(0.0cm,0.3cm)}]panel.north) {Lamella$_\textrm{FFF}$ 1};</span>
<span id="cb9-38"><a href="#cb9-38" aria-hidden="true" tabindex="-1"></a> % scalebar - physical height is 8032.5 nm</span>
<span id="cb9-39"><a href="#cb9-39" aria-hidden="true" tabindex="-1"></a> \draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 8.0325 , yshift=0.3cm]panel.south west);</span>
<span id="cb9-40"><a href="#cb9-40" aria-hidden="true" tabindex="-1"></a> \draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 8.0325, yshift=0.3cm]panel.south west);</span>
<span id="cb9-41"><a href="#cb9-41" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(0.5cm,0.0cm)}]panel.north east) {}; </span>
<span id="cb9-42"><a href="#cb9-42" aria-hidden="true" tabindex="-1"></a> \node[graphicNode]  (panel) {\includegraphics[height=\panelheight]{content/images/fff_lamella2_view.png}};</span>
<span id="cb9-43"><a href="#cb9-43" aria-hidden="true" tabindex="-1"></a> % Title</span>
<span id="cb9-44"><a href="#cb9-44" aria-hidden="true" tabindex="-1"></a> \node[anchor=center,align=center] at ([shift={(0.0cm,0.3cm)}]panel.north) {Lamella$_\textrm{FFF}$ 2};</span>
<span id="cb9-45"><a href="#cb9-45" aria-hidden="true" tabindex="-1"></a> % scalebar - physical height is 10000.5 nm</span>
<span id="cb9-46"><a href="#cb9-46" aria-hidden="true" tabindex="-1"></a> \draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 10.0005 , yshift=0.3cm]panel.south west);</span>
<span id="cb9-47"><a href="#cb9-47" aria-hidden="true" tabindex="-1"></a> \draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 10.0005, yshift=0.3cm]panel.south west);</span>
<span id="cb9-48"><a href="#cb9-48" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(0.5cm,0.0cm)}]panel.north east) {}; </span>
<span id="cb9-49"><a href="#cb9-49" aria-hidden="true" tabindex="-1"></a> \node[graphicNode]  (panel) {\includegraphics[height=\panelheight]{content/images/fff_lamella3_view.png}};</span>
<span id="cb9-50"><a href="#cb9-50" aria-hidden="true" tabindex="-1"></a> % Title</span>
<span id="cb9-51"><a href="#cb9-51" aria-hidden="true" tabindex="-1"></a> \node[anchor=center,align=center] at ([shift={(0.0cm,0.3cm)}]panel.north) {Lamella$_\textrm{FFF}$ 3};</span>
<span id="cb9-52"><a href="#cb9-52" aria-hidden="true" tabindex="-1"></a> % scalebar - physical height is 9975 nm</span>
<span id="cb9-53"><a href="#cb9-53" aria-hidden="true" tabindex="-1"></a> \draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 9.975 , yshift=0.3cm]panel.south west);</span>
<span id="cb9-54"><a href="#cb9-54" aria-hidden="true" tabindex="-1"></a> \draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 9.975, yshift=0.3cm]panel.south west);</span>
<span id="cb9-55"><a href="#cb9-55" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(0.5cm,0.0cm)}]panel.north east) {}; </span>
<span id="cb9-56"><a href="#cb9-56" aria-hidden="true" tabindex="-1"></a> \node[graphicNode]  (panel) {\includegraphics[height=\panelheight]{content/images/fff_lamella4_view.png}};</span>
<span id="cb9-57"><a href="#cb9-57" aria-hidden="true" tabindex="-1"></a> % Title</span>
<span id="cb9-58"><a href="#cb9-58" aria-hidden="true" tabindex="-1"></a> \node[anchor=center,align=center] at ([shift={(0.0cm,0.3cm)}]panel.north) {Lamella$_\textrm{FFF}$ 4};</span>
<span id="cb9-59"><a href="#cb9-59" aria-hidden="true" tabindex="-1"></a> % scalebar - physical height is 9337.5 nm</span>
<span id="cb9-60"><a href="#cb9-60" aria-hidden="true" tabindex="-1"></a> \draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 9.3375 , yshift=0.3cm]panel.south west);</span>
<span id="cb9-61"><a href="#cb9-61" aria-hidden="true" tabindex="-1"></a> \draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 9.3375, yshift=0.3cm]panel.south west);</span>
<span id="cb9-62"><a href="#cb9-62" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb9-63"><a href="#cb9-63" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(0.0cm,-1.0cm)}]0.5,0 |- panel.south) {}; </span>
<span id="cb9-64"><a href="#cb9-64" aria-hidden="true" tabindex="-1"></a> \node[labelNode] {B};</span>
<span id="cb9-65"><a href="#cb9-65" aria-hidden="true" tabindex="-1"></a> \node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/images/euc_lamella1_assm.png}};</span>
<span id="cb9-66"><a href="#cb9-66" aria-hidden="true" tabindex="-1"></a> % Title</span>
<span id="cb9-67"><a href="#cb9-67" aria-hidden="true" tabindex="-1"></a> \node[anchor=center,align=center] at ([shift={(0.0cm,0.3cm)}]panel.north) {Lamella$_\textrm{EUC}$ 1: 909 tiles};</span>
<span id="cb9-68"><a href="#cb9-68" aria-hidden="true" tabindex="-1"></a> % scalebar - physical height is 12810 nm</span>
<span id="cb9-69"><a href="#cb9-69" aria-hidden="true" tabindex="-1"></a> \draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 12.81 , yshift=0.3cm]panel.south west);</span>
<span id="cb9-70"><a href="#cb9-70" aria-hidden="true" tabindex="-1"></a> \draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 12.81, yshift=0.3cm]panel.south west);</span>
<span id="cb9-71"><a href="#cb9-71" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(0.5cm,0.0cm)}]panel.north east) {}; </span>
<span id="cb9-72"><a href="#cb9-72" aria-hidden="true" tabindex="-1"></a> \node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/images/euc_lamella2_assm.png}};</span>
<span id="cb9-73"><a href="#cb9-73" aria-hidden="true" tabindex="-1"></a> % Title</span>
<span id="cb9-74"><a href="#cb9-74" aria-hidden="true" tabindex="-1"></a> \node[anchor=center,align=center] at ([shift={(0.0cm,0.3cm)}]panel.north) {Lamella$_\textrm{EUC}$ 2: 451 tiles};</span>
<span id="cb9-75"><a href="#cb9-75" aria-hidden="true" tabindex="-1"></a> % scalebar - physical height is 7027.5 nm</span>
<span id="cb9-76"><a href="#cb9-76" aria-hidden="true" tabindex="-1"></a> \draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 7.0275 , yshift=0.3cm]panel.south west);</span>
<span id="cb9-77"><a href="#cb9-77" aria-hidden="true" tabindex="-1"></a> \draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 7.0275, yshift=0.3cm]panel.south west);</span>
<span id="cb9-78"><a href="#cb9-78" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(0.5cm,0.0cm)}]panel.north east) {}; </span>
<span id="cb9-79"><a href="#cb9-79" aria-hidden="true" tabindex="-1"></a> \node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/images/euc_lamella3_assm.png}};</span>
<span id="cb9-80"><a href="#cb9-80" aria-hidden="true" tabindex="-1"></a> % Title</span>
<span id="cb9-81"><a href="#cb9-81" aria-hidden="true" tabindex="-1"></a> \node[anchor=center,align=center] at ([shift={(0.0cm,0.3cm)}]panel.north) {Lamella$_\textrm{EUC}$ 3: 513 tiles};</span>
<span id="cb9-82"><a href="#cb9-82" aria-hidden="true" tabindex="-1"></a> % scalebar - physical height is 9615 nm</span>
<span id="cb9-83"><a href="#cb9-83" aria-hidden="true" tabindex="-1"></a> \draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 9.615 , yshift=0.3cm]panel.south west);</span>
<span id="cb9-84"><a href="#cb9-84" aria-hidden="true" tabindex="-1"></a> \draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 9.615, yshift=0.3cm]panel.south west);</span>
<span id="cb9-85"><a href="#cb9-85" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(0.5cm,0.0cm)}]panel.north east) {}; </span>
<span id="cb9-86"><a href="#cb9-86" aria-hidden="true" tabindex="-1"></a> \node[graphicNode] (panel) {\includegraphics[height=\panelheight]{content/images/euc_lamella4_assm.png}};</span>
<span id="cb9-87"><a href="#cb9-87" aria-hidden="true" tabindex="-1"></a> % Title</span>
<span id="cb9-88"><a href="#cb9-88" aria-hidden="true" tabindex="-1"></a> \node[anchor=center,align=center] at ([shift={(0.0cm,0.3cm)}]panel.north) {Lamella$_\textrm{EUC}$ 4: 497 tiles};</span>
<span id="cb9-89"><a href="#cb9-89" aria-hidden="true" tabindex="-1"></a> % scalebar - physical height is 8625 nm</span>
<span id="cb9-90"><a href="#cb9-90" aria-hidden="true" tabindex="-1"></a> \draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 8.625 , yshift=0.3cm]panel.south west);</span>
<span id="cb9-91"><a href="#cb9-91" aria-hidden="true" tabindex="-1"></a> \draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 8.625, yshift=0.3cm]panel.south west);</span>
<span id="cb9-92"><a href="#cb9-92" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb9-93"><a href="#cb9-93" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(0.0cm,-0.5cm)}]0.5,0 |- panel.south) {}; </span>
<span id="cb9-94"><a href="#cb9-94" aria-hidden="true" tabindex="-1"></a> \node[graphicNode]  (panel) {\includegraphics[height=\panelheight]{content/images/fff_lamella1_assm.png}};</span>
<span id="cb9-95"><a href="#cb9-95" aria-hidden="true" tabindex="-1"></a> % Title</span>
<span id="cb9-96"><a href="#cb9-96" aria-hidden="true" tabindex="-1"></a> \node[anchor=center,align=center] at ([shift={(0.0cm,0.3cm)}]panel.north) {Lamella$_\textrm{FFF}$ 1: 438 tiles};</span>
<span id="cb9-97"><a href="#cb9-97" aria-hidden="true" tabindex="-1"></a> % scalebar - physical height is 10000.5 nm</span>
<span id="cb9-98"><a href="#cb9-98" aria-hidden="true" tabindex="-1"></a> \draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 10.0005 , yshift=0.3cm]panel.south west);</span>
<span id="cb9-99"><a href="#cb9-99" aria-hidden="true" tabindex="-1"></a> \draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 10.0005, yshift=0.3cm]panel.south west);</span>
<span id="cb9-100"><a href="#cb9-100" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(0.5cm,0.0cm)}]panel.north east) {}; </span>
<span id="cb9-101"><a href="#cb9-101" aria-hidden="true" tabindex="-1"></a> \node[graphicNode]  (panel) {\includegraphics[height=\panelheight]{content/images/fff_lamella2_assm.png}};</span>
<span id="cb9-102"><a href="#cb9-102" aria-hidden="true" tabindex="-1"></a> % Title</span>
<span id="cb9-103"><a href="#cb9-103" aria-hidden="true" tabindex="-1"></a> \node[anchor=center,align=center] at ([shift={(0.0cm,0.3cm)}]panel.north) {Lamella$_\textrm{FFF}$ 2: 474 tiles};</span>
<span id="cb9-104"><a href="#cb9-104" aria-hidden="true" tabindex="-1"></a> % scalebar - physical height is 10000.5 nm</span>
<span id="cb9-105"><a href="#cb9-105" aria-hidden="true" tabindex="-1"></a> \draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 10.0005 , yshift=0.3cm]panel.south west);</span>
<span id="cb9-106"><a href="#cb9-106" aria-hidden="true" tabindex="-1"></a> \draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 10.0005, yshift=0.3cm]panel.south west);</span>
<span id="cb9-107"><a href="#cb9-107" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(0.5cm,0.0cm)}]panel.north east) {}; </span>
<span id="cb9-108"><a href="#cb9-108" aria-hidden="true" tabindex="-1"></a> \node[graphicNode]  (panel) {\includegraphics[height=\panelheight]{content/images/fff_lamella3_assm.png}};</span>
<span id="cb9-109"><a href="#cb9-109" aria-hidden="true" tabindex="-1"></a> % Title</span>
<span id="cb9-110"><a href="#cb9-110" aria-hidden="true" tabindex="-1"></a> \node[anchor=center,align=center] at ([shift={(0.0cm,0.3cm)}]panel.north) {Lamella$_\textrm{FFF}$ 3: 501 tiles};</span>
<span id="cb9-111"><a href="#cb9-111" aria-hidden="true" tabindex="-1"></a> % scalebar - physical height is 9975 nm</span>
<span id="cb9-112"><a href="#cb9-112" aria-hidden="true" tabindex="-1"></a> \draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 9.975 , yshift=0.3cm]panel.south west);</span>
<span id="cb9-113"><a href="#cb9-113" aria-hidden="true" tabindex="-1"></a> \draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 9.975, yshift=0.3cm]panel.south west);</span>
<span id="cb9-114"><a href="#cb9-114" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(0.5cm,0.0cm)}]panel.north east) {}; </span>
<span id="cb9-115"><a href="#cb9-115" aria-hidden="true" tabindex="-1"></a> \node[graphicNode]  (panel) {\includegraphics[height=\panelheight]{content/images/fff_lamella4_assm.png}};</span>
<span id="cb9-116"><a href="#cb9-116" aria-hidden="true" tabindex="-1"></a> % Title</span>
<span id="cb9-117"><a href="#cb9-117" aria-hidden="true" tabindex="-1"></a> \node[anchor=center,align=center] at ([shift={(0.0cm,0.3cm)}]panel.north) {Lamella$_\textrm{FFF}$ 4: 425 tiles};</span>
<span id="cb9-118"><a href="#cb9-118" aria-hidden="true" tabindex="-1"></a> % scalebar - physical height is 9337.5 nm</span>
<span id="cb9-119"><a href="#cb9-119" aria-hidden="true" tabindex="-1"></a> \draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 9.3375 , yshift=0.3cm]panel.south west);</span>
<span id="cb9-120"><a href="#cb9-120" aria-hidden="true" tabindex="-1"></a> \draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + \panelheight / 9.3375, yshift=0.3cm]panel.south west);</span>
<span id="cb9-121"><a href="#cb9-121" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb9-122"><a href="#cb9-122" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb9-123"><a href="#cb9-123" aria-hidden="true" tabindex="-1"></a> \newlength{\padding}</span>
<span id="cb9-124"><a href="#cb9-124" aria-hidden="true" tabindex="-1"></a> \setlength{\padding}{0.2cm}</span>
<span id="cb9-125"><a href="#cb9-125" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(0.0cm,-1.0cm)}]0.5,0 |- panel.south) {}; </span>
<span id="cb9-126"><a href="#cb9-126" aria-hidden="true" tabindex="-1"></a> \node[labelNode] {C};</span>
<span id="cb9-127"><a href="#cb9-127" aria-hidden="true" tabindex="-1"></a><span class="st">-\node[graphicNode] (panel) {\includegraphics[height=0.75\panelheight]{content/images/granules/zap004.png}};</span></span>
<span id="cb9-128"><a href="#cb9-128" aria-hidden="true" tabindex="-1"></a><span class="va">+\node[graphicNode] (panel) {\includegraphics[height=0.75\panelheight]{content/images/granules/zap004_lp.png}};</span></span>
<span id="cb9-129"><a href="#cb9-129" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(\padding,0.0cm)}]panel.north east) {}; </span>
<span id="cb9-130"><a href="#cb9-130" aria-hidden="true" tabindex="-1"></a><span class="st">-\node[graphicNode] (panel) {\includegraphics[height=0.75\panelheight]{content/images/granules/zap012.png}};</span></span>
<span id="cb9-131"><a href="#cb9-131" aria-hidden="true" tabindex="-1"></a><span class="va">+\node[graphicNode] (panel) {\includegraphics[height=0.75\panelheight]{content/images/granules/zap012_lp.png}};</span></span>
<span id="cb9-132"><a href="#cb9-132" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(\padding,0.0cm)}]panel.north east) {}; </span>
<span id="cb9-133"><a href="#cb9-133" aria-hidden="true" tabindex="-1"></a><span class="st">-\node[graphicNode] (panel) {\includegraphics[height=0.75\panelheight]{content/images/granules/zap015.png}};</span></span>
<span id="cb9-134"><a href="#cb9-134" aria-hidden="true" tabindex="-1"></a><span class="va">+\node[graphicNode] (panel) {\includegraphics[height=0.75\panelheight]{content/images/granules/zap015_lp.png}};</span></span>
<span id="cb9-135"><a href="#cb9-135" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(\padding,0.0cm)}]panel.north east) {}; </span>
<span id="cb9-136"><a href="#cb9-136" aria-hidden="true" tabindex="-1"></a><span class="st">-\node[graphicNode] (panel) {\includegraphics[height=0.75\panelheight]{content/images/granules/zap020.png}};</span></span>
<span id="cb9-137"><a href="#cb9-137" aria-hidden="true" tabindex="-1"></a><span class="va">+\node[graphicNode] (panel) {\includegraphics[height=0.75\panelheight]{content/images/granules/zap020_lp.png}};</span></span>
<span id="cb9-138"><a href="#cb9-138" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(\padding,0.0cm)}]panel.north east) {}; </span>
<span id="cb9-139"><a href="#cb9-139" aria-hidden="true" tabindex="-1"></a><span class="st">-\node[graphicNode] (panel) {\includegraphics[height=0.75\panelheight]{content/images/granules/zap006.png}};</span></span>
<span id="cb9-140"><a href="#cb9-140" aria-hidden="true" tabindex="-1"></a><span class="va">+\node[graphicNode] (panel) {\includegraphics[height=0.75\panelheight]{content/images/granules/zap006_lp.png}};</span></span>
<span id="cb9-141"><a href="#cb9-141" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb9-142"><a href="#cb9-142" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(0.0cm,-\padding)}]0.5,0 |- panel.south) {}; </span>
<span id="cb9-143"><a href="#cb9-143" aria-hidden="true" tabindex="-1"></a><span class="st">-\node[graphicNode]  (panel) {\includegraphics[height=0.75\panelheight]{content/images/granules/zap021.png}};</span></span>
<span id="cb9-144"><a href="#cb9-144" aria-hidden="true" tabindex="-1"></a><span class="va">+\node[graphicNode]  (panel) {\includegraphics[height=0.75\panelheight]{content/images/granules/zap021_lp.png}};</span></span>
<span id="cb9-145"><a href="#cb9-145" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(\padding,0.0cm)}]panel.north east) {}; </span>
<span id="cb9-146"><a href="#cb9-146" aria-hidden="true" tabindex="-1"></a><span class="st">-\node[graphicNode]  (panel) {\includegraphics[height=0.75\panelheight]{content/images/granules/zap022.png}};</span></span>
<span id="cb9-147"><a href="#cb9-147" aria-hidden="true" tabindex="-1"></a><span class="va">+\node[graphicNode]  (panel) {\includegraphics[height=0.75\panelheight]{content/images/granules/zap022_lp.png}};</span></span>
<span id="cb9-148"><a href="#cb9-148" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(\padding,0.0cm)}]panel.north east) {}; </span>
<span id="cb9-149"><a href="#cb9-149" aria-hidden="true" tabindex="-1"></a><span class="st">-\node[graphicNode]  (panel) {\includegraphics[height=0.75\panelheight]{content/images/granules/zap027.png}};</span></span>
<span id="cb9-150"><a href="#cb9-150" aria-hidden="true" tabindex="-1"></a><span class="va">+\node[graphicNode]  (panel) {\includegraphics[height=0.75\panelheight]{content/images/granules/zap027_lp.png}};</span></span>
<span id="cb9-151"><a href="#cb9-151" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(\padding,0.0cm)}]panel.north east) {}; </span>
<span id="cb9-152"><a href="#cb9-152" aria-hidden="true" tabindex="-1"></a><span class="st">-\node[graphicNode]  (panel) {\includegraphics[height=0.75\panelheight]{content/images/granules/zap029.png}};</span></span>
<span id="cb9-153"><a href="#cb9-153" aria-hidden="true" tabindex="-1"></a><span class="va">+\node[graphicNode]  (panel) {\includegraphics[height=0.75\panelheight]{content/images/granules/zap029_lp.png}};</span></span>
<span id="cb9-154"><a href="#cb9-154" aria-hidden="true" tabindex="-1"></a> \node (anchor) at ([shift={(\padding,0.0cm)}]panel.north east) {}; </span>
<span id="cb9-155"><a href="#cb9-155" aria-hidden="true" tabindex="-1"></a><span class="st">-\node[graphicNode]  (panel) {\includegraphics[height=0.75\panelheight]{content/images/granules/zap023.png}};</span></span>
<span id="cb9-156"><a href="#cb9-156" aria-hidden="true" tabindex="-1"></a><span class="va">+\node[graphicNode]  (panel) {\includegraphics[height=0.75\panelheight]{content/images/granules/zap023_lp.png}};</span></span>
<span id="cb9-157"><a href="#cb9-157" aria-hidden="true" tabindex="-1"></a> % scalebar - physical height is 375 nm</span>
<span id="cb9-158"><a href="#cb9-158" aria-hidden="true" tabindex="-1"></a> \draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + 0.75\panelheight / 3.75 , yshift=0.3cm]panel.south west);</span>
<span id="cb9-159"><a href="#cb9-159" aria-hidden="true" tabindex="-1"></a> \draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]panel.south west) -- ([xshift=0.5cm + 0.75\panelheight / 3.75, yshift=0.3cm]panel.south west);</span>
<span id="cb9-160"><a href="#cb9-160" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb9-161"><a href="#cb9-161" aria-hidden="true" tabindex="-1"></a> </span></code></pre></div>
<figure>
<p><img src="tikz:matching_euc" id="fig:matching_euc" alt="Template matching in lamella imaged using the DeCo-LACE approach at eucentric focus (A) Montage of Lamella_\textrm{EUC} 1 overlaid with detected targets colored in orange. Scalebar corresponds to 1 μm. (B) Side view of detected targets in the lamella, such that the direction of the electron beam is horizontal. (C-F) Magnified area of panel A showing rough ER with associated ribosomes (C), outer nuclear membrane with associated ribosomes (D), ribsomes arranged in a circular fashion (E), ribosomes enclosed in a less electron dense inclusion in a granule (F). Ribosomes are colored in white with the surface of the peptide exit tunnel colored in green and the A, P, and E sites colored in blue, purple, and red, respectively.Scalebar corresponds to 100 nm." /></p>
<figcaption aria-hidden="true">
<p>Template matching in lamella imaged using the DeCo-LACE approach at
eucentric focus (A) Montage of Lamella<span class="math inline"><sub>EUC</sub></span> 1 overlaid with detected
targets colored in orange. Scalebar corresponds to 1 μm. (B) Side view
of detected targets in the lamella, such that the direction of the
electron beam is horizontal. (C-F) Magnified area of panel A showing
rough ER with associated ribosomes (C), outer nuclear membrane with
associated ribosomes (D), ribsomes arranged in a circular fashion (E),
ribosomes enclosed in a less electron dense inclusion in a granule (F).
Ribosomes are colored in white with the surface of the peptide exit
tunnel colored in green and the A, P, and E sites colored in blue,
purple, and red, respectively.Scalebar corresponds to 100 nm.</p>
</figcaption>
</figure>
<pre class="tikz-figure"><code>\newlength{\panelheight}
\setlength{\panelheight}{12cm}

\node (anchor) at (0.5,9.5) {}; 
\node[labelNode] {A};
\node[graphicNode] (assm) {\includegraphics[height=\panelheight]{content/images/euc_lamella1_assm.png}};
\node[graphicNode] {\includegraphics[height=\panelheight]{content/images/euc_lamella1.png}};
\node (rect) at (7.4,3.5) [anchor=center,draw,thick,minimum width=0.3cm,minimum height=0.3cm,color=red] {};
\node (rect) at (1.75,4.5) [anchor=center,draw,thick,minimum width=0.3cm,minimum height=0.3cm,color=green] {};
\node (rect) at (7.2,2.15) [anchor=center,draw,thick,minimum width=0.3cm,minimum height=0.3cm,color=blue!60] {};
\node (rect) at (6.45,6.0) [anchor=center,draw,thick,minimum width=0.47cm,minimum height=0.47cm,color=cyan] {};
% scalebar - physical height is 12810 nm
\draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]assm.south west) -- ([xshift=0.5cm + \panelheight / 12.81 , yshift=0.3cm]assm.south west);
\draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]assm.south west) -- ([xshift=0.5cm + \panelheight / 12.81, yshift=0.3cm]assm.south west);


\node (anchor) at (11.0,9.5) {}; 
\node[labelNode] {B};
\node[anchor=north west, below right=0cm and 0.0cm of anchor] (side) {\includegraphics[height=12cm, trim=20.5cm 0 2cm 0]{content/images/euc_lamella1_side.png}};

\draw [-&gt;, thick] ($ (assm.east) + (0.1,0) $) -- node[above=0.2cm, align=center] (c) {} ($ (side.west) + (0.2,0) $);

\centerarc [canvas is zx plane at y=0.75, -&gt;](c.center)(-90:180:0.15);
\draw [very thick] ($ (c.center) + (0,-0.1) $) -- ($ (c.center) + (0,0.2) $);
\node at ($ (c.center) + (0.5,0) $) {90°};

\newlength{\panelheightm}
\setlength{\panelheightm}{2.8cm}
\node (anchor) at (13.0,9.5) {}; 
\node[labelNode] {C};
\node[graphicNode] (pan) {\includegraphics[height=\panelheightm]{content/images/euc_lamella1_detail1.png}};
\node (n3) [box=red, fit=(pan)] {};
% scalebar - physical height is 300 nm
\draw [line width=1.55mm,white] ([xshift=0.2cm, yshift=0.2cm]pan.south west) -- ([xshift=0.2cm + \panelheightm / 3 , yshift=0.2cm]pan.south west);
\draw [line width=0.7mm,black] ([xshift=0.2cm, yshift=0.2cm]pan.south west) -- ([xshift=0.2cm + \panelheightm / 3, yshift=0.2cm]pan.south west);

\node (anchor) at (13.0,6.5) {}; 
\node[labelNode] {D};
\node[graphicNode] (pan) {\includegraphics[height=\panelheightm]{content/images/euc_lamella1_detail2.png}};
\node (n3) [box=green, fit=(pan)] {};

% scalebar - physical height is 300 nm
\draw [line width=1.55mm,white] ([xshift=0.2cm, yshift=0.2cm]pan.south west) -- ([xshift=0.2cm + \panelheightm / 3 , yshift=0.2cm]pan.south west);
\draw [line width=0.7mm,black] ([xshift=0.2cm, yshift=0.2cm]pan.south west) -- ([xshift=0.2cm + \panelheightm / 3, yshift=0.2cm]pan.south west);

\node (anchor) at (13.,3.5) {}; 
\node[labelNode] {E};
\node[graphicNode] (pan) {\includegraphics[height=\panelheightm]{content/images/euc_lamella1_detail3.png}};
\node (n3) [box=blue!60, fit=(pan)] {};

% scalebar - physical height is 300 nm
\draw [line width=1.55mm,white] ([xshift=0.2cm, yshift=0.2cm]pan.south west) -- ([xshift=0.2cm + \panelheightm / 3 , yshift=0.2cm]pan.south west);
\draw [line width=0.7mm,black] ([xshift=0.2cm, yshift=0.2cm]pan.south west) -- ([xshift=0.2cm + \panelheightm / 3, yshift=0.2cm]pan.south west);

\node (anchor) at (13.0,0.5) {}; 
\node[labelNode] {F};
\node[graphicNode] (pan) {\includegraphics[height=\panelheightm]{content/images/euc_lamella1_detail4.png}};
\node (n3) [box=cyan, fit=(pan)] {};

% scalebar - physical height is 500 nm
\draw [line width=1.55mm,white] ([xshift=0.2cm, yshift=0.2cm]pan.south west) -- ([xshift=0.2cm + \panelheightm / 5 , yshift=0.2cm]pan.south west);
\draw [line width=0.7mm,black] ([xshift=0.2cm, yshift=0.2cm]pan.south west) -- ([xshift=0.2cm + \panelheightm / 5, yshift=0.2cm]pan.south west);</code></pre>
<figure>
<p><img src="tikz:matching_fff" id="fig:matching_fff" alt="Template matching in lamella imaged using the DeCo-LACE approach at fringe-free focus (A) Montage of Lamella_\textrm{FFF} 4 overlaid with detected targets colored in orange. Scalebar corresponds to 1 μm. (B) Side view of detected targets in the lamella, such that the direction of the electron beam is horizontal. (C-E) Magnified area of panel A showing rough ER with associated ribosomes (C) and ribosomes enclosed in a less electron dense inclusion in a granule (D,E). (F) Side view of panel E. Ribosomes are colored in white with the surface of the peptide exit tunnel colored in green and the A, P, and E sites colored in blue, purple, and red, respectively. Scalebar corresponds to 100 nm." /></p>
<figcaption aria-hidden="true">
<p>Template matching in lamella imaged using the DeCo-LACE approach at
fringe-free focus (A) Montage of Lamella<span class="math inline"><sub>FFF</sub></span> 4 overlaid with detected
targets colored in orange. Scalebar corresponds to 1 μm. (B) Side view
of detected targets in the lamella, such that the direction of the
electron beam is horizontal. (C-E) Magnified area of panel A showing
rough ER with associated ribosomes (C) and ribosomes enclosed in a less
electron dense inclusion in a granule (D,E). (F) Side view of panel E.
Ribosomes are colored in white with the surface of the peptide exit
tunnel colored in green and the A, P, and E sites colored in blue,
purple, and red, respectively. Scalebar corresponds to 100 nm.</p>
</figcaption>
</figure>
<pre class="tikz-figure"><code>\newlength{\panelheight}
\setlength{\panelheight}{12cm}

\node (anchor) at (0.5,9.5) {}; 
\node[labelNode] {A};
\node[graphicNode] (assm) {\includegraphics[height=\panelheight]{content/images/fff_lamella4_assm.png}};
\node[graphicNode] {\includegraphics[height=\panelheight]{content/images/fff_lamella4.png}};
\node (rect) at (5.35,5.5) [anchor=center,draw,thick,minimum width=0.4cm,minimum height=0.4cm,color=red] {};
\node (rect) at (8.05,1.75) [anchor=center,draw,thick,minimum width=0.6cm,minimum height=0.6cm,color=green] {};
\node (rect) at (6.45,-1.23) [anchor=center,draw,thick,minimum width=0.5cm,minimum height=0.5cm,color=blue!60] {};
% scalebar - physical height is 9337.5nm
\draw [line width=1.55mm,white] ([xshift=0.5cm, yshift=0.3cm]assm.south west) -- ([xshift=0.5cm + \panelheight / 9.3375 , yshift=0.3cm]assm.south west);
\draw [line width=0.7mm,black] ([xshift=0.5cm, yshift=0.3cm]assm.south west) -- ([xshift=0.5cm + \panelheight / 9.3375, yshift=0.3cm]assm.south west);

\node (anchor) at (11.0,9.5) {}; 
\node[labelNode] {B};
\node[anchor=north west, below right=0cm and 0.0cm of anchor] (side) {\includegraphics[height=12cm, trim=15.5cm 0 2cm 0]{content/images/fff_lamella4_side.png}};

\draw [-&gt;, thick] ($ (assm.east) + (0.1,0) $) -- node[above=0.2cm, align=center] (c) {} ($ (side.west) + (0.2,0) $);

\centerarc [canvas is zx plane at y=0.75, -&gt;](c.center)(-90:180:0.15);
\draw [very thick] ($ (c.center) + (0,-0.1) $) -- ($ (c.center) + (0,0.2) $);
\node at ($ (c.center) + (0.5,0) $) {90°};

\newlength{\panelheightm}
\setlength{\panelheightm}{2.8cm}
\node (anchor) at (13.0,9.5) {}; 
\node[labelNode] {C};
\node[graphicNode] (pan) {\includegraphics[height=2.8cm]{content/images/fff_lamella4_detail2.png}};
\node (n3) [box=red, fit=(pan)] {};
% scalebar - physical height is 300 nm
\draw [line width=1.55mm,white] ([xshift=0.2cm, yshift=0.2cm]pan.south west) -- ([xshift=0.2cm + \panelheightm / 3 , yshift=0.2cm]pan.south west);
\draw [line width=0.7mm,black] ([xshift=0.2cm, yshift=0.2cm]pan.south west) -- ([xshift=0.2cm + \panelheightm / 3, yshift=0.2cm]pan.south west);

\node (anchor) at (13.0,6.5) {}; 
\node[labelNode] {D};
\node[graphicNode] (pan) {\includegraphics[height=2.8cm]{content/images/fff_lamella4_detail1.png}};
\node (n3) [box=green, fit=(pan)] {};
% scalebar - physical height is 500 nm
\draw [line width=1.55mm,white] ([xshift=0.2cm, yshift=0.2cm]pan.south west) -- ([xshift=0.2cm + \panelheightm / 5 , yshift=0.2cm]pan.south west);
\draw [line width=0.7mm,black] ([xshift=0.2cm, yshift=0.2cm]pan.south west) -- ([xshift=0.2cm + \panelheightm / 5, yshift=0.2cm]pan.south west);

\node (anchor) at (13.0,3.5) {}; 
\node[labelNode] {E};
\node[graphicNode] (pan){\includegraphics[height=2.8cm]{content/images/fff_lamella4_detail3_back.png}};
\node[graphicNode] {\includegraphics[height=2.8cm]{content/images/fff_lamella4_detail3_close.png}};
\node[graphicNode] {\includegraphics[height=2.8cm]{content/images/fff_lamella4_detail3_far.png}};
\node (n3) [box=blue!60, fit=(pan)] {};
% scalebar - physical height is 400 nm
\draw [line width=1.55mm,white] ([xshift=0.2cm, yshift=0.2cm]pan.south west) -- ([xshift=0.2cm + \panelheightm / 4 , yshift=0.2cm]pan.south west);
\draw [line width=0.7mm,black] ([xshift=0.2cm, yshift=0.2cm]pan.south west) -- ([xshift=0.2cm + \panelheightm / 4, yshift=0.2cm]pan.south west);

\node (anchor) at (13,0.5) {}; 
\node[labelNode] {F};
\node[graphicNode] {\includegraphics[height=2.8cm]{content/images/fff_lamella4_detail3_far_side.png}};
\node[graphicNode] {\includegraphics[height=2.8cm]{content/images/fff_lamella4_detail3_close_side.png}};</code></pre>
<figure>
<p><img src="tikz:matching_stat" id="fig:matching_stat" alt="Statistics of 2DTM on lamella imaged using DeCo-LACE (A) Number of detected targets in each lamella (B) Distribution of targets per tile in each lamella. Only tiles with two or more detected targets were included (C) Distribution of SNRs in each lamella (D) For each lamella an average of all tiles is shown. Overlaid is a scatterplot of all detected targets in these tiles according to their in-tile coordinates. Scatterplot is colored according to the 2DTM SNR. There are no detected targets in the top circle-circle intersection due to radiation damage from previous exposures. (E) 2D histogram of number of detected targets as a function of beam-image shift (F) Mean 2DTM SNR as a function of beam-image shift" /></p>
<figcaption aria-hidden="true">
<p>Statistics of 2DTM on lamella imaged using DeCo-LACE (A) Number of
detected targets in each lamella (B) Distribution of targets per tile in
each lamella. Only tiles with two or more detected targets were included
(C) Distribution of SNRs in each lamella (D) For each lamella an average
of all tiles is shown. Overlaid is a scatterplot of all detected targets
in these tiles according to their in-tile coordinates. Scatterplot is
colored according to the 2DTM SNR. There are no detected targets in the
top circle-circle intersection due to radiation damage from previous
exposures. (E) 2D histogram of number of detected targets as a function
of beam-image shift (F) Mean 2DTM SNR as a function of beam-image
shift</p>
</figcaption>
</figure>
<pre class="tikz-figure"><code>
\node (anchor) at (0.5,2.5) {}; 
\node[labelNode] {A};
\node[graphicNode] (panel) {\includegraphics[width=4.5cm]{content/code/img/num_matches_plot.png}};
\node (anchor) at ([shift={(0.5cm,0.0)}]panel.north east) {}; 
\node[labelNode] {B};
\node[graphicNode] (panel) {\includegraphics[width=4.5cm]{content/code/img/matches_per_tile_plot.png}};
\node (anchor) at ([shift={(0.5cm,0.0)}]panel.north east) {}; 
\node[labelNode] {C};
\node[graphicNode] (panel) {\includegraphics[width=4.5cm]{content/code/img/scores_plot.png}};
\node (anchor) at ([shift={(0.0cm,-0.2)}]0.5,2.5 |- panel.south east) {}; 
\node[labelNode] {D};
\node[graphicNode] (panel) {\includegraphics[width=14cm]{content/code/img/micrograph_with_matches.png}};

\node (anchor) at ([shift={(0.0cm,-0.2)}]0.5,2.5 |- panel.south east) {}; 
\node[labelNode] {E};
\node[graphicNode] (panel) {\includegraphics[width=6cm]{content/code/img/num_matches_vs_bis.png}};

\node (anchor) at ([shift={(1.0cm,0.0)}]panel.north east) {}; 
\node[labelNode] {F};
\node[graphicNode] {\includegraphics[width=6cm]{content/code/img/snrs_vs_bis.png}};</code></pre>
<figure>
<p><img src="tikz:deco_lace_workflow" id="fig:deco_lace_workflow" alt="Workflow of DeCo-LACE processing" /></p>
<figcaption aria-hidden="true">
<p>Workflow of DeCo-LACE processing</p>
</figcaption>
</figure>
<div class="sourceCode" id="cb13"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb13-1"><a href="#cb13-1" aria-hidden="true" tabindex="-1"></a> \begin{scope}[</span>
<span id="cb13-2"><a href="#cb13-2" aria-hidden="true" tabindex="-1"></a>           blocks/.style = {rectangle, draw, fill=blue!20, text width=15em, align=center, rounded corners, minimum height=2em,inner sep=0.5em},</span>
<span id="cb13-3"><a href="#cb13-3" aria-hidden="true" tabindex="-1"></a>           inputs/.style = {rectangle,draw,fill=cyan!20, inner sep=0.5em},</span>
<span id="cb13-4"><a href="#cb13-4" aria-hidden="true" tabindex="-1"></a>           every path/.style={line width=1pt}</span>
<span id="cb13-5"><a href="#cb13-5" aria-hidden="true" tabindex="-1"></a> ]</span>
<span id="cb13-6"><a href="#cb13-6" aria-hidden="true" tabindex="-1"></a> \node [inputs,anchor=north] (struc) at (3.0,17.5) {\textbf{6SWA Structure}};</span>
<span id="cb13-7"><a href="#cb13-7" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-8"><a href="#cb13-8" aria-hidden="true" tabindex="-1"></a> \node [inputs,anchor=north] (movies) at (9.0,17.5) {\textbf{Movies}};</span>
<span id="cb13-9"><a href="#cb13-9" aria-hidden="true" tabindex="-1"></a> \node [inputs,anchor=north] (meta) at (15.0,17.5) {\textbf{SerialEM Metadata}};</span>
<span id="cb13-10"><a href="#cb13-10" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-11"><a href="#cb13-11" aria-hidden="true" tabindex="-1"></a> \node [inputs,anchor=north] (over) at (3.0,1.5) {\textbf{Overview Image}};</span>
<span id="cb13-12"><a href="#cb13-12" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-13"><a href="#cb13-13" aria-hidden="true" tabindex="-1"></a> \node [blocks,anchor=north] (simu) at (3.0,15.5) {\textbf{Generate Template} \\ \textit{simulate}</span>
<span id="cb13-14"><a href="#cb13-14" aria-hidden="true" tabindex="-1"></a> \begin{itemize}</span>
<span id="cb13-15"><a href="#cb13-15" aria-hidden="true" tabindex="-1"></a><span class="st">-    \item   Generate electron density map at 1.5{\AA}  pixel-size</span></span>
<span id="cb13-16"><a href="#cb13-16" aria-hidden="true" tabindex="-1"></a><span class="va">+    \item   Generate density map at 1.5{\AA}  pixel-size</span></span>
<span id="cb13-17"><a href="#cb13-17" aria-hidden="true" tabindex="-1"></a> \end{itemize}};</span>
<span id="cb13-18"><a href="#cb13-18" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-19"><a href="#cb13-19" aria-hidden="true" tabindex="-1"></a> \node [blocks,anchor=north] (mc) at (9.0,15.5) {\textbf{Motion correction} \\ \textit{unblur\_decolace}</span>
<span id="cb13-20"><a href="#cb13-20" aria-hidden="true" tabindex="-1"></a> \begin{itemize}</span>
<span id="cb13-21"><a href="#cb13-21" aria-hidden="true" tabindex="-1"></a>     \item   Motion correction using central area of movie</span>
<span id="cb13-22"><a href="#cb13-22" aria-hidden="true" tabindex="-1"></a>     \item   Mask out unilluminated areas and replace with noise</span>
<span id="cb13-23"><a href="#cb13-23" aria-hidden="true" tabindex="-1"></a> \end{itemize}};</span>
<span id="cb13-24"><a href="#cb13-24" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-25"><a href="#cb13-25" aria-hidden="true" tabindex="-1"></a> \node [inputs,below = 0.5 cm of mc.south] (images) {\textbf{Images}};</span>
<span id="cb13-26"><a href="#cb13-26" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-27"><a href="#cb13-27" aria-hidden="true" tabindex="-1"></a> \node [blocks,below = 0.5 cm of images.south] (ctf) {\textbf{Ctf estimation} \\ \textit{ctffind4} </span>
<span id="cb13-28"><a href="#cb13-28" aria-hidden="true" tabindex="-1"></a> \begin{itemize}</span>
<span id="cb13-29"><a href="#cb13-29" aria-hidden="true" tabindex="-1"></a>     \item Standard CTF estimation</span>
<span id="cb13-30"><a href="#cb13-30" aria-hidden="true" tabindex="-1"></a> \end{itemize}};</span>
<span id="cb13-31"><a href="#cb13-31" aria-hidden="true" tabindex="-1"></a> \node [blocks,below = 0.7 cm of ctf.south] (match) {\textbf{Template matching} \\ \textit{match\_template} \\ Template matching using 6swa as template};</span>
<span id="cb13-32"><a href="#cb13-32" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-33"><a href="#cb13-33" aria-hidden="true" tabindex="-1"></a> \begin{scope}[on background layer]</span>
<span id="cb13-34"><a href="#cb13-34" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-35"><a href="#cb13-35" aria-hidden="true" tabindex="-1"></a> \node[draw,very thick, dashed, inner sep=0.5em, rounded corners, fill=blue!10,</span>
<span id="cb13-36"><a href="#cb13-36" aria-hidden="true" tabindex="-1"></a>     fit=(mc) (ctf) (match)] (gui) {};</span>
<span id="cb13-37"><a href="#cb13-37" aria-hidden="true" tabindex="-1"></a> \end{scope}</span>
<span id="cb13-38"><a href="#cb13-38" aria-hidden="true" tabindex="-1"></a> \node[below left=0.1cm and 0cm of gui.south east, font=\fontannot] {cisTEM GUI};</span>
<span id="cb13-39"><a href="#cb13-39" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-40"><a href="#cb13-40" aria-hidden="true" tabindex="-1"></a> \node [blocks,below = 1.4 cm of match.south] (refine) {\textbf{Refine matches} \\ \textit{refine\_matches.py/refine\_template} \\ Refine matches and compile list of all matches in a lamella};</span>
<span id="cb13-41"><a href="#cb13-41" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-42"><a href="#cb13-42" aria-hidden="true" tabindex="-1"></a> \node [inputs,below = 0.5 cm of refine.south] (refstar) {\textbf{matches\_in\_tiles.star}};</span>
<span id="cb13-43"><a href="#cb13-43" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-44"><a href="#cb13-44" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-45"><a href="#cb13-45" aria-hidden="true" tabindex="-1"></a> \node [blocks,anchor=center] (assem) at (images.east -| meta.south) {\textbf{Assemble montage} \\ \textit{assemble\_montage.py} </span>
<span id="cb13-46"><a href="#cb13-46" aria-hidden="true" tabindex="-1"></a> \begin{itemize}</span>
<span id="cb13-47"><a href="#cb13-47" aria-hidden="true" tabindex="-1"></a>     \item Create list of all tiles and their respective coordinates</span>
<span id="cb13-48"><a href="#cb13-48" aria-hidden="true" tabindex="-1"></a>     \item Create binned montage image</span>
<span id="cb13-49"><a href="#cb13-49" aria-hidden="true" tabindex="-1"></a> \end{itemize}};</span>
<span id="cb13-50"><a href="#cb13-50" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-51"><a href="#cb13-51" aria-hidden="true" tabindex="-1"></a> \node [inputs,below = 0.5 cm of assem.south] (monstar) {\textbf{montage.star}};</span>
<span id="cb13-52"><a href="#cb13-52" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-53"><a href="#cb13-53" aria-hidden="true" tabindex="-1"></a> \node [blocks,anchor=center] (assemma) at (refstar.east -| monstar.south) {\textbf{Assemble matches} \\ \textit{assemble\_matches.py} </span>
<span id="cb13-54"><a href="#cb13-54" aria-hidden="true" tabindex="-1"></a> \begin{itemize}</span>
<span id="cb13-55"><a href="#cb13-55" aria-hidden="true" tabindex="-1"></a>     \item Transfer coordinates of matches into the montage coordinate system</span>
<span id="cb13-56"><a href="#cb13-56" aria-hidden="true" tabindex="-1"></a> \end{itemize}};</span>
<span id="cb13-57"><a href="#cb13-57" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-58"><a href="#cb13-58" aria-hidden="true" tabindex="-1"></a> \node [inputs,below = 0.5 cm of assemma.south] (monmastar) {\textbf{matches\_in\_montage.star}};</span>
<span id="cb13-59"><a href="#cb13-59" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-60"><a href="#cb13-60" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-61"><a href="#cb13-61" aria-hidden="true" tabindex="-1"></a> %% Draw lines</span>
<span id="cb13-62"><a href="#cb13-62" aria-hidden="true" tabindex="-1"></a> \draw[-&gt;] (struc) -- (simu);</span>
<span id="cb13-63"><a href="#cb13-63" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-64"><a href="#cb13-64" aria-hidden="true" tabindex="-1"></a> \draw[-&gt;] (movies) -- (mc);</span>
<span id="cb13-65"><a href="#cb13-65" aria-hidden="true" tabindex="-1"></a> \draw[-&gt;] (mc) -- (images);</span>
<span id="cb13-66"><a href="#cb13-66" aria-hidden="true" tabindex="-1"></a> \draw[-&gt;] (images) -- (ctf);</span>
<span id="cb13-67"><a href="#cb13-67" aria-hidden="true" tabindex="-1"></a> \draw[-&gt;] (ctf) -- (match);</span>
<span id="cb13-68"><a href="#cb13-68" aria-hidden="true" tabindex="-1"></a> \draw[-&gt;] (match) -- (refine);</span>
<span id="cb13-69"><a href="#cb13-69" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-70"><a href="#cb13-70" aria-hidden="true" tabindex="-1"></a> \draw[-&gt;] (meta) -- (assem);</span>
<span id="cb13-71"><a href="#cb13-71" aria-hidden="true" tabindex="-1"></a> \draw[-&gt;] (images) -- (assem);</span>
<span id="cb13-72"><a href="#cb13-72" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-73"><a href="#cb13-73" aria-hidden="true" tabindex="-1"></a> \draw[-&gt;] (simu) |- (match);</span>
<span id="cb13-74"><a href="#cb13-74" aria-hidden="true" tabindex="-1"></a> \draw[-&gt;] (simu) |- (refine);</span>
<span id="cb13-75"><a href="#cb13-75" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-76"><a href="#cb13-76" aria-hidden="true" tabindex="-1"></a> \draw[-&gt;] (assem) -- (monstar);</span>
<span id="cb13-77"><a href="#cb13-77" aria-hidden="true" tabindex="-1"></a> \draw[-&gt;] (monstar) -- (assemma);</span>
<span id="cb13-78"><a href="#cb13-78" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-79"><a href="#cb13-79" aria-hidden="true" tabindex="-1"></a> \draw[-&gt;] (refine) -- (refstar);</span>
<span id="cb13-80"><a href="#cb13-80" aria-hidden="true" tabindex="-1"></a> \draw[-&gt;] (refstar) -- (assemma);</span>
<span id="cb13-81"><a href="#cb13-81" aria-hidden="true" tabindex="-1"></a> \draw[-&gt;] (assemma) -- (monmastar);</span>
<span id="cb13-82"><a href="#cb13-82" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb13-83"><a href="#cb13-83" aria-hidden="true" tabindex="-1"></a> \end{scope}</span></code></pre></div>
<figure>
<p><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkYAAAAAYAAjCB0C8AAAAASUVORK5CYII=" id="fig:movie1" data-tag="Movie 1" alt="Movie of detected LSU targets in Lamella_\textrm{EUC} 1, corresponding to Figure 5" /></p>
<figcaption aria-hidden="true">
<p>Movie of detected LSU targets in Lamella<span class="math inline"><sub>EUC</sub></span> 1, corresponding to Figure
5</p>
</figcaption>
</figure>
<figure>
<p><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkYAAAAAYAAjCB0C8AAAAASUVORK5CYII=" id="fig:movie2" data-tag="Movie 2" alt="Movie of detected LSU targets in Lamella_\textrm{FFF} 4, corresponding to Figure 6" /></p>
<figcaption aria-hidden="true">
<p>Movie of detected LSU targets in Lamella<span class="math inline"><sub>FFF</sub></span> 4, corresponding to Figure
6</p>
</figcaption>
</figure>
<h2 id="references">References</h2>
</article>
</body>
</html>
