<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Johannes Elferich" />
  <meta name="author" content="Giulia Schiroli" />
  <meta name="author" content="David Scadden" />
  <meta name="author" content="Nikolaus Grigorieff" />
  <meta name="dcterms.date" content="2022-06-06" />
  <meta name="keywords" content="cryo-EM, visual protemics, ribosome" />
  <title>Defocus Corrected Large Area Cryo-EM (DeCo-LACE) for Label-Free Detection of Molecules across Entire Cell Sections</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta name="dc.title" content="Defocus Corrected Large Area Cryo-EM (DeCo-LACE) for Label-Free Detection of Molecules across Entire Cell Sections" />
  <meta name="citation_title" content="Defocus Corrected Large Area Cryo-EM (DeCo-LACE) for Label-Free Detection of Molecules across Entire Cell Sections" />
  <meta property="og:title" content="Defocus Corrected Large Area Cryo-EM (DeCo-LACE) for Label-Free Detection of Molecules across Entire Cell Sections" />
  <meta property="twitter:title" content="Defocus Corrected Large Area Cryo-EM (DeCo-LACE) for Label-Free Detection of Molecules across Entire Cell Sections" />
  <meta name="dc.date" content="2022-06-06" />
  <meta name="citation_publication_date" content="2022-06-06" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Johannes Elferich" />
  <meta name="citation_author_institution" content="RNA Therapeutics Institute, University of Massachusetts Chan Medical School, Worcester, MA, USA" />
  <meta name="citation_author_institution" content="Howard Hughes Medical Institute, Ashburn, VA, USA" />
  <meta name="citation_author_orcid" content="0000-0002-9911-706X" />
  <meta name="citation_author" content="Giulia Schiroli" />
  <meta name="citation_author_institution" content="Center for Regenerative Medicine, Massachusetts General Hospital, Boston, MA 02114, USA" />
  <meta name="citation_author_orcid" content="0000-0002-9812-8728" />
  <meta name="citation_author" content="David Scadden" />
  <meta name="citation_author_institution" content="Center for Regenerative Medicine, Massachusetts General Hospital, Boston, MA 02114, USA" />
  <meta name="citation_author_institution" content="Harvard Stem Cell Institute, Cambridge, MA 02138, USA" />
  <meta name="citation_author_institution" content="Department of Stem Cell and Regenerative Biology, Harvard University, Cambridge, MA 02138, USA" />
  <meta name="citation_author" content="Nikolaus Grigorieff" />
  <meta name="citation_author_institution" content="RNA Therapeutics Institute, University of Massachusetts Chan Medical School, Worcester, MA, USA" />
  <meta name="citation_author_institution" content="Howard Hughes Medical Institute, Ashburn, VA, USA" />
  <meta name="citation_author_orcid" content="0000-0002-1506-909X" />
  <p><link rel="canonical" href="https://jojoelfe.github.io/deco_lace_template_matching_manuscript/" /></p>
  <meta property="og:url" content="https://jojoelfe.github.io/deco_lace_template_matching_manuscript/" />
  <meta property="twitter:url" content="https://jojoelfe.github.io/deco_lace_template_matching_manuscript/" />
  <meta name="citation_fulltext_html_url" content="https://jojoelfe.github.io/deco_lace_template_matching_manuscript/" />
  <meta name="citation_pdf_url" content="https://jojoelfe.github.io/deco_lace_template_matching_manuscript/manuscript.pdf" />
  <p><link rel="alternate" type="application/pdf" href="https://jojoelfe.github.io/deco_lace_template_matching_manuscript/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://jojoelfe.github.io/deco_lace_template_matching_manuscript/v/a321def4b341f6e6f2c8d66936f53cf5e9398eae/" /></p>
  <meta name="manubot_html_url_versioned" content="https://jojoelfe.github.io/deco_lace_template_matching_manuscript/v/a321def4b341f6e6f2c8d66936f53cf5e9398eae/" />
  <meta name="manubot_pdf_url_versioned" content="https://jojoelfe.github.io/deco_lace_template_matching_manuscript/v/a321def4b341f6e6f2c8d66936f53cf5e9398eae/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <p><link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" /></p>
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Defocus Corrected Large Area Cryo-EM (DeCo-LACE) for Label-Free Detection of Molecules across Entire Cell Sections</h1>
</header>
<!--
Manubot generated metadata rendered from header-includes-template.html.
Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
-->
<meta name="dc.format" content="text/html" />
<meta name="dc.title" content="Defocus Corrected Large Area Cryo-EM (DeCo-LACE) for Label-Free Detection of Molecules across Entire Cell Sections" />
<meta name="citation_title" content="Defocus Corrected Large Area Cryo-EM (DeCo-LACE) for Label-Free Detection of Molecules across Entire Cell Sections" />
<meta property="og:title" content="Defocus Corrected Large Area Cryo-EM (DeCo-LACE) for Label-Free Detection of Molecules across Entire Cell Sections" />
<meta property="twitter:title" content="Defocus Corrected Large Area Cryo-EM (DeCo-LACE) for Label-Free Detection of Molecules across Entire Cell Sections" />
<meta name="dc.date" content="2022-06-06" />
<meta name="citation_publication_date" content="2022-06-06" />
<meta name="dc.language" content="en-US" />
<meta name="citation_language" content="en-US" />
<meta name="dc.relation.ispartof" content="Manubot" />
<meta name="dc.publisher" content="Manubot" />
<meta name="citation_journal_title" content="Manubot" />
<meta name="citation_technical_report_institution" content="Manubot" />
<meta name="citation_author" content="Johannes Elferich" />
<meta name="citation_author_institution" content="RNA Therapeutics Institute, University of Massachusetts Chan Medical School, Worcester, MA, USA" />
<meta name="citation_author_institution" content="Howard Hughes Medical Institute, Ashburn, VA, USA" />
<meta name="citation_author_orcid" content="0000-0002-9911-706X" />
<meta name="citation_author" content="Giulia Schiroli" />
<meta name="citation_author_institution" content="Center for Regenerative Medicine, Massachusetts General Hospital, Boston, MA 02114, USA" />
<meta name="citation_author_orcid" content="0000-0002-9812-8728" />
<meta name="citation_author" content="David Scadden" />
<meta name="citation_author_institution" content="Center for Regenerative Medicine, Massachusetts General Hospital, Boston, MA 02114, USA" />
<meta name="citation_author_institution" content="Harvard Stem Cell Institute, Cambridge, MA 02138, USA" />
<meta name="citation_author_institution" content="Department of Stem Cell and Regenerative Biology, Harvard University, Cambridge, MA 02138, USA" />
<meta name="citation_author" content="Nikolaus Grigorieff" />
<meta name="citation_author_institution" content="RNA Therapeutics Institute, University of Massachusetts Chan Medical School, Worcester, MA, USA" />
<meta name="citation_author_institution" content="Howard Hughes Medical Institute, Ashburn, VA, USA" />
<meta name="citation_author_orcid" content="0000-0002-1506-909X" />
<p><link rel="canonical" href="https://jojoelfe.github.io/deco_lace_template_matching_manuscript/" /></p>
<meta property="og:url" content="https://jojoelfe.github.io/deco_lace_template_matching_manuscript/" />
<meta property="twitter:url" content="https://jojoelfe.github.io/deco_lace_template_matching_manuscript/" />
<meta name="citation_fulltext_html_url" content="https://jojoelfe.github.io/deco_lace_template_matching_manuscript/" />
<meta name="citation_pdf_url" content="https://jojoelfe.github.io/deco_lace_template_matching_manuscript/manuscript.pdf" />
<p><link rel="alternate" type="application/pdf" href="https://jojoelfe.github.io/deco_lace_template_matching_manuscript/manuscript.pdf" />
<link rel="alternate" type="text/html" href="https://jojoelfe.github.io/deco_lace_template_matching_manuscript/v/a321def4b341f6e6f2c8d66936f53cf5e9398eae/" /></p>
<meta name="manubot_html_url_versioned" content="https://jojoelfe.github.io/deco_lace_template_matching_manuscript/v/a321def4b341f6e6f2c8d66936f53cf5e9398eae/" />
<meta name="manubot_pdf_url_versioned" content="https://jojoelfe.github.io/deco_lace_template_matching_manuscript/v/a321def4b341f6e6f2c8d66936f53cf5e9398eae/manuscript.pdf" />
<meta property="og:type" content="article" />
<meta property="twitter:card" content="summary_large_image" />
<p><link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
<link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" /></p>
<meta name="theme-color" content="#ad1457" />
<!-- end Manubot generated metadata -->
<p><small><em>
This manuscript
(<a href="https://jojoelfe.github.io/deco_lace_template_matching_manuscript/v/a321def4b341f6e6f2c8d66936f53cf5e9398eae/">permalink</a>)
was automatically generated
from <a href="https://github.com/jojoelfe/deco_lace_template_matching_manuscript/tree/a321def4b341f6e6f2c8d66936f53cf5e9398eae">jojoelfe/deco_lace_template_matching_manuscript@a321def</a>
on June 6, 2022.
</em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>Johannes Elferich</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9911-706X">0000-0002-9911-706X</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/jojoelfe">jojoelfe</a><br>
<small>
RNA Therapeutics Institute, University of Massachusetts Chan Medical School, Worcester, MA, USA; Howard Hughes Medical Institute, Ashburn, VA, USA
</small></p></li>
<li><p><strong>Giulia Schiroli</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9812-8728">0000-0002-9812-8728</a><br>
<small>
Center for Regenerative Medicine, Massachusetts General Hospital, Boston, MA 02114, USA
</small></p></li>
<li><p><strong>David Scadden</strong><br><br>
<small>
Center for Regenerative Medicine, Massachusetts General Hospital, Boston, MA 02114, USA; Harvard Stem Cell Institute, Cambridge, MA 02138, USA; Department of Stem Cell and Regenerative Biology, Harvard University, Cambridge, MA 02138, USA
</small></p></li>
<li><p><strong>Nikolaus Grigorieff</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-1506-909X">0000-0002-1506-909X</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/nikogrigorieff">nikogrigorieff</a><br>
<small>
RNA Therapeutics Institute, University of Massachusetts Chan Medical School, Worcester, MA, USA; Howard Hughes Medical Institute, Ashburn, VA, USA
</small></p></li>
</ul>
<h2 class="page_break_before" id="abstract">Abstract</h2>
<p>A major goal of biological imaging is localization of biomolecules inside a cell. Fluorescence microscopy can localize biomolecules inside whole cells
and tissues, but its ability to count biomolecules and accuracy of the spatial
coordinates is limited by the wavelength of visible light. Cryo-electron
microscopy (cryo-EM) provides highly accurate position and orientation
information of biomolecules but is often confined to small fields of view inside
a cell, limiting biological context. In this study we use a new data-acquisition
scheme called “Defocus-Corrected Large-Area cryo-EM” (DeCo-LACE) to collect
high-resolution images of entire sections (100 – 200 nm thick lamellae)
of neutrophil-like mouse cells, representing 1-2% of the total cellular
volume. We use 2D template matching (2DTM) to determine localization and orientation of the large ribosomal subunit in these sections. These data provide “maps” of ribosomes across entire sections of mammalian cells. This high-throughput cryo-EM data collection approach together with 2DTM will advance visual proteomics and provide biological insight that cannot be obtained by other methods.</p>
<h2 id="introduction">Introduction</h2>
<p>A major goal in understanding cellular processes is the knowledge of the
amounts, location, interactions, and conformations of biomolecules
inside the cell. This knowledge can be obtained by approaches broadly
divided into label- and label-free techniques. In label-dependent
techniques a probe is physically attached to a molecule of interest that
is able to be detected by its strong signal, such as a
fluorescent molecule. In label-free techniques, the physical properties
of molecules themselves are used for detection. An example for this is
proteomics using mass-spectrometry <span class="citation" data-cites="tSXIKPl7">[<a href="#ref-tSXIKPl7" role="doc-biblioref">1</a>]</span>. The advantage
of label-free techniques is that they can provide information over
thousands of molecules, while label-dependent techniques offer highly specific
information for a few molecules. in particular, spatial information is primarily achieved using label-dependent techniques, such as
fluorescence microscopy <span class="citation" data-cites="VBmW7Aot">[<a href="#ref-VBmW7Aot" role="doc-biblioref">2</a>]</span>.</p>
<p>Cryo-electron microscopy (cryo-EM) has the potential to directly visualize the
arrangement of atoms that compose biomolecules inside of cells, thereby
allowing label-free detection with high spatial accuracy. This has been
called “visual proteomics” <span class="citation" data-cites="tGQ6TSUo">[<a href="#ref-tGQ6TSUo" role="doc-biblioref">3</a>]</span>. Since cryo-EM
requires thin samples (&lt;500nm), imaging of biomolecules inside cells is
restricted to small organisms, thin regions of cells, or samples that
have been suitably thinned. Thinning can be achieved either by
mechanical sectioning <span class="citation" data-cites="g8QavfwP">[<a href="#ref-g8QavfwP" role="doc-biblioref">4</a>]</span> or by
milling using a focused ion beam (FIB) <span class="citation" data-cites="16IhS1Nc4">[<a href="#ref-16IhS1Nc4" role="doc-biblioref">5</a>]</span>. This complex workflow
leads to a low throughput of cryo-EM imaging of cells and is further
limited by the fact that at the required magnifications, typical fields
of view (FOV) are very small compared to mammalian cells, and the FOV
achieved by label-dependent techniques such as fluorescence light microscopy. The
predominant cryo-EM technique for the localization of biomolecules of
defined size and shape inside cells is cryo-electron tomography <span class="citation" data-cites="Rksh2dxu">[<a href="#ref-Rksh2dxu" role="doc-biblioref">6</a>]</span>. However, the requirement of a tilt series at
every imaged location and subsequent image alignment, severely limits
the throughput for molecular localization.</p>
<p>An alternative approach is to identify molecules by their structural
“fingerprint” in single projection using “2D template-matching” (2DTM)
<span class="citation" data-cites="Ynb3IP6I 18KGpXYPE 10bXZuF3G">[<a href="#ref-Ynb3IP6I" role="doc-biblioref">7</a>,<a href="#ref-18KGpXYPE" role="doc-biblioref">8</a>,<a href="#ref-10bXZuF3G" role="doc-biblioref">9</a>]</span>. In this
method, a 3D model of a biomolecule is used as a template to find 2D
projections that match the molecules visible in the electron
micrographs. This method requires a projection search on a fine angular
grid, and the projections are used to find local cross-correlation peaks
with the micrograph. Since the location of a biomolecule in the
z-direction causes predictable aberrations to the projection image, this
method can be used to calculate complete 3D coordinates and orientations
of a biomolecule in a cellular sample
<span class="citation" data-cites="18KGpXYPE">[<a href="#ref-18KGpXYPE" role="doc-biblioref">8</a>]</span>.</p>
<p>Here we apply 2DTM of the ribosome large subunit (LSU) to conditionally immortalized <em>mus musculus</em> (mouse) cell line that gives rise to functional mature neutrophils <span class="citation" data-cites="1B9Vt9eYu">[<a href="#ref-1B9Vt9eYu" role="doc-biblioref">10</a>]</span> after thinning using cryo-FIB milling. We chose these
cells because genetic defects in the ribosome machinery often leads to
hematopoietic disease <span class="citation" data-cites="gRoY21jY">[<a href="#ref-gRoY21jY" role="doc-biblioref">11</a>]</span> and direct quantification of
ribosome location, number and conformational states in hematopoietic cells could
lead to new insight into hematopoietic disease <span class="citation" data-cites="KAJ7221k">[<a href="#ref-KAJ7221k" role="doc-biblioref">12</a>]</span>.
To increase the amount of collected data and to provide unbiased sampling of the
whole lamella, we devised a new data-acquisition scheme, “Defocus-Corrected
Large Area Cryo-Electron microscopy” (DeCo-LACE). 2DTM allows us to test whether
aberrations caused by large beam-image shifts and highly condensed beams
deteriorate the high-resolution signal. We find that these aberrations do not
impede LSU detection by 2DTM. The resulting data provide a description of
ribosome distribution in an entire lamella, which represent 1-2% of the cellular
volume. We find a highly heterogeneous density of ribosomes within the cell.
Analysis of the throughput in this method suggests that for the foreseeable
future computation will be the bottleneck for visual proteomics.</p>
<h2 id="results">Results</h2>
<h3 id="dtm-detects-large-ribosomal-subunits-in-cryo-fib-lamellae-of-mammalian-cells">2DTM detects large ribosomal subunits in cryo-FIB lamellae of mammalian cells</h3>
<p>FIB-milled <em>Saccharomyces cerevisiae</em> (yeast) cells are sufficiently well preserved to permit localization of 60S ribosomal subunits with 2DTM <span class="citation" data-cites="T1Qr0xyP">[<a href="#ref-T1Qr0xyP" role="doc-biblioref">13</a>]</span>. Due to the larger size of mammalian cells compared to yeast cells, it was unclear whether plunge freezing would be adequate to produce vitreous ice across the whole volume of the cell. To test this we prepared cryo-lamellae of mouse neutrophil cells. A low magnification image
of a representative lamella clearly shows cellular features consistent with a
neutrophile-like phenotype, mainly a segmented nucleus and a plethora of
membrane-organelles, corresponding to the granules and secretory vesicles of
neutrophils (Fig. [<a href="#fig:initmatching">1</a>]A). We then proceeded to acquire
micrographs on this lamella with a defocus of 0.5-1.0 μm, 30 e<sup>-</sup>/Å<span class="math inline">\(^2\)</span>/s
exposure and 1.76 Å pixel size. We manually selected multiple locations in the
lamella and acquired micrographs using standard low-dose techniques where
focusing is performed on a sacrificial area. The resulting micrographs showed
smooth bilayered membranes and no signs of crystalline ice (Fig.
[<a href="#fig:initmatching">1</a>]C,D), indicating successful vitrification throughout the
lamella.</p>
<p>We used an atomic model of the 60S mouse ribosomal subunit (6SWA) for 2DTM <span class="citation" data-cites="sbkam02a">[<a href="#ref-sbkam02a" role="doc-biblioref">14</a>]</span>. In a
subset of images, the distribution of cross-correlation scores significantly
exceeded the distribution expected from images devoid of detectable targets. In the
resulting scaled maximum-intensity projections (MIPs), clear peaks with SNR
values up to 10 were apparent (Fig. [<a href="#fig:initmatching2">2 - figure supplement 1</a>]A). Using a
threshold criterion to select significant targets (see Methods), we found that in
images of cytosolic compartments there were 10-500 ribosomes within one micrograph
(Fig. [<a href="#fig:initmatching">1</a>]B-E). Notably, we found no targets in areas
corresponding to the nucleus (Fig. [<a href="#fig:initmatching">1</a>]B) or mitochondria
(Fig. <a href="#fig:initmatching">1</a>D). In the cytoplasm, we found a highly variable
number of targets, only ~ 50 in some exposures (Fig. [<a href="#fig:initmatching">1</a>]E) and
up to 500 in others (Fig. [<a href="#fig:initmatching">1</a>]C). However, it is unclear whether this ten-fold difference in local ribosome concentration is due to technical variation, such as sample thickness, or biological variation. To differentiate between the two we reasoned it was important to not manually choose imaging regions and to collect larger amounts of data. We therefore set out to collect cryo-EM data for 2DTM from mammalian cell lamellae in a high-throughput unbiased fashion.</p>
<h3 id="deco-lace-for-2d-imaging-of-whole-lamellae">DeCo-LACE for 2D imaging of whole lamellae</h3>
<p>In order to obtain high-resolution data from complete lamellae, we developed a new
approach for data collection. This approach uses three key strategies: (1) every
electron that exposes a fresh area of the sample is collected on the camera (2)
image shift is used to precisely and quickly raster the surface of a lamella and
(3) focusing is done without using a sacrificial area (Fig. [<a href="#fig:approach">2</a>]A).</p>
<p>To ensure that every electron exposing a fresh area of the sample is captured by
the detector, we adjusted the electron beam size to be entirely contained by the
detector area. During canonical low-dose imaging, the microscope is configured so
that the focal plane is identical to the eucentric plane of the specimen stage.
This leaves the C2 aperture out of focus, resulting in ripples at the edge of
the beam (Fig. [<a href="#fig:approach">2</a>]D). While these ripples are low-resolution
features that likely do not interfere with 2DTM <span class="citation" data-cites="Ynb3IP6I">[<a href="#ref-Ynb3IP6I" role="doc-biblioref">7</a>]</span>, we also tested data
collection under conditions where the C2 aperture is in focus (“fringe-free”,
Fig. [<a href="#fig:approach">2</a>]E) <span class="citation" data-cites="wsFP5c54">[<a href="#ref-wsFP5c54" role="doc-biblioref">15</a>]</span>.</p>
<p>We then centered a lamella on the optical axis of the microscope and used the
image shift controls of the microscope to systematically scan the whole surface
of the lamella in a hexagonal pattern (Fig. [<a href="#fig:approach">2</a>]A,C). Instead of
focusing on a sacrificial area, we determined the defocus from every exposure
after it was taken. The defocus was then adjusted based on the difference
between desired and measured defocus (Fig. [<a href="#fig:approach">2</a>]B). Since we used a
serpentine pattern for data collection, every exposure was close to the previous
exposure, making large changes in the defocus unlikely. Furthermore, we started
our acquisition pattern on the platinum deposition edge to make sure that the
initial exposure where the defocus was not yet adjusted did not contain any
biologically relevant information.</p>
<p>We used this strategy to collect data on eight lamellae, four using the
eucentric focus condition, hereafter referred to as Lamella<sub>EUC</sub>, and four using
the fringe-free condition, hereafter referred to as Lamella<sub>FFF</sub>(Fig.
[<a href="#fig:assembly">3</a>] A+D, Fig.
[<a href="#fig:lamella_images">4 - figure supplement 4</a>]A). We were able to collect data with a highly
consistent defocus of 800 nm (Fig. [<a href="#fig:approach">2</a>]F), both in the eucentric
focus and fringe-free focus condition. To ensure that data were collected
consistently, we mapped defocus values as a function of the applied image shift
(Fig. [<a href="#fig:lamella_spatial_info">3 - figure supplement 1</a>]A). This demonstrated that the defocus was
consistent across a lamella, except for rare outliers and in images containing
contamination. We also plotted the measured objective astigmatism of each
lamella and found that it varies with the applied image shift, becoming more
astigmatic mostly due to image shift in the x direction (Fig.
[<a href="#fig:lamella_spatial_info">3 - figure supplement 1</a>]B). While approaches exist to correct for this
during the data collection <span class="citation" data-cites="APSL9LmU">[<a href="#ref-APSL9LmU" role="doc-biblioref">16</a>]</span>, we opted to not use these approaches in our
initial experiments. We reasoned that because 2DTM depends on high-resolution
information, this would be an excellent test of how much these aberration affect
imaging.</p>
<p>We assembled the tile micrographs into a montage using the image-shift values
and the SerialEM calibration followed by cross-correlation based refinement (see
Methods). In the resulting montages, the same cellular features visible in the
overview images are apparent (Fig. [<a href="#fig:assembly">3</a>]B+E, Fig. [<a href="#fig:lamella_images">4 - figure supplement 4</a>]B), however
due to the high magnification and low defocus many more details, such as the
membrane bilayer separation, can be observed (Fig. [<a href="#fig:assembly">3</a>]C+F). For montages collected using the
eucentric condition, there are clearly visible fringes at the edges between the
tiles (Fig. [<a href="#fig:assembly">3</a>]C), which are absent in the fringe-free focus montages (Fig. [<a href="#fig:assembly">3</a>]F). In our analysis
below, we show that these fringes do not impede target detection by 2DTM, making
them primarily an aesthetic issue. We also note that the tiling
pattern is visible in the montages (Fig. [<a href="#fig:assembly">3</a>]B+E), which we believe is due to the non-linear
behavior of the K3 camera since we can observe these shading artifacts in micrographs of a condensed beam over vacuum (Fig. [<a href="#fig:gain">4 - figure supplement 3</a>]).</p>
<p>The montages show membrane vesicles and granules with highly variable sizes and
density. We found that a substantial number of granules, which are characterized by higher
density inside the the surrounding cytosol <span class="citation" data-cites="xE2EhAcH">[<a href="#ref-xE2EhAcH" role="doc-biblioref">17</a>]</span>, seemed to contain a membrane-enclosed inclusion with density similar to
the surrounding cytosol (Fig. [<a href="#fig:lamella_images">4 - figure supplement 4</a>]C) and could therefore be formed
by inward budding of the granule membrane. These granules were
150-300 nm in diameter and the inclusions were 100-200 nm in diameter. Based on
these dimensions the granules are either azurophil or specific granules <span class="citation" data-cites="xE2EhAcH">[<a href="#ref-xE2EhAcH" role="doc-biblioref">17</a>]</span>. To our
knowledge, these inclusion have not been described in granulocytes and are
further described and discussed below.</p>
<h3 id="dtm-of-deco-lace-data-reveals-large-ribosomal-subunit-distribution-in-cellular-cross-sections">2DTM of DeCo-LACE data reveals large ribosomal subunit distribution in cellular cross-sections</h3>
<p>In our initial attempts of using 2DTM on micrographs acquired with the DeCo-LACE
protocol, we did not observe any SNR peaks above threshold using the large
subunit of the mouse ribosome (Fig. [<a href="#fig:crop_unblur">4 - figure supplement 1</a>]A). We reasoned that
the edges of the beam might interfere with motion-correction of the movies as
they represent strong low-resolution features that do not move with the sample.
When we cropped the movie frames to exclude the beam edges, the estimated amount
of motion increased (Fig. [<a href="#fig:crop_unblur">4 - figure supplement 1</a>]B), consistent with successful
tracking of sample motion. Furthermore, in the motion-corrected average we could
identify significant SNR peaks (Fig. [<a href="#fig:crop_unblur">4 - figure supplement 1</a>]B), confirming the
high sensitivity of 2DTM to the presence of high-resolution signal preserved in
the images by the motion correction. To streamline data processing, we
implemented a function in unblur to consider only a defined central area of a
movie for estimation of sample motion, while still averaging the complete movie
frames (Fig. [<a href="#fig:crop_unblur">4 - figure supplement 1</a>]C). Using this approach, we motion-corrected
all tiles in the eight lamellae and found consistently total motion below 1 Å
per frame (Fig. [<a href="#fig:lamella_motion_thickness">4 - figure supplement 2</a>] A). In some lamellae we
found increased motion in the lamella center, which indicates areas of variable
mechanical stability within FIB-milled lamellae. In some micrographs we also
observed that the beam edges gave rise to artifacts in the MIP and numerous
false-positive detections at the edge of the illuminated area (Fig. [<a href="#fig:crop_unblur">4 - figure supplement 1</a>]D). A
similar phenomenon was observed on isolated “hot” pixels in unilluminated areas.
To overcome this issue we implemented a function in unblur to replace
dark areas in the micrograph with Gaussian noise (see Methods), with
mean and standard deviation matching the illuminated portion of the micrograph
(Fig. [<a href="#fig:crop_unblur">4 - figure supplement 1</a>]D+E). Together, these pre-processing steps enabled us to perform 2DTM on
all tiles of the eight lamellae.</p>
<p>We used the tile positions to calculate the positions of the detected
LSUs in the lamellae (Fig. [<a href="#fig:matching_euc">4</a>]A, Fig. [<a href="#fig:matching_fff">5</a>]A). Overlaying these positions of the lamellae montages reveals LSU
distribution throughout the FIB-milled slices of individual cells. Consistent with prior observations imaging selected views in yeast <span class="citation" data-cites="T1Qr0xyP">[<a href="#ref-T1Qr0xyP" role="doc-biblioref">13</a>]</span>, organelles
like the nucleus and mitochondria only showed sporadic targets detected with low
SNRs, consistent with the estimated false-positive rate of one per tile. For
each detected target we also calculated the Z positions from the individual
estimated defocus and defocus offset for each tile. When viewed from the side,
the ribosome positions therefore show the slight tilts of the lamellae relative
to the microscope frame of reference (Fig. [<a href="#fig:matching_euc">4</a>]B, Fig.
[<a href="#fig:matching_fff">5</a>]B). Furthermore, the side views indicated that lamellae
were thinner at the leading edge. Indeed, when plotting the transmitted beam
intensities in individual tiles as a function of beam image-shift, we observed
substantially higher intensities at the leading edge (Fig. [<a href="#fig:lamella_motion_thickness">4 - figure supplement 2</a>]B), which in energy-filtered
TEM indicates a thinner sample <span class="citation" data-cites="17IP4Lhz1">[<a href="#ref-17IP4Lhz1" role="doc-biblioref">18</a>]</span>. Even though we prepared the lamellae
with the “overtilt” approach <span class="citation" data-cites="hr3dyUeI">[<a href="#ref-hr3dyUeI" role="doc-biblioref">19</a>]</span>, this means that LSU densities across
the lamellae can be skewed by a change in thickness, and better sample
preparation methods are needed to generate more even samples.</p>
<p>Close inspection of the LSU positions in the lamellae revealed several
interesting features. LSUs could be seen associating with membranes, in
patterns reminiscent of the rough endoplasmic reticulum (Fig.
[<a href="#fig:matching_euc">4</a>]C, Fig. [<a href="#fig:matching_fff">5</a>]C) or the outer nuclear
membrane (Fig. [<a href="#fig:matching_euc">4</a>]D). We also observed LSUs forming
ring-like structures (Fig. [<a href="#fig:matching_euc">4</a>]E), potentially indicating
circularized mRNAs <span class="citation" data-cites="cie13Q8F">[<a href="#ref-cie13Q8F" role="doc-biblioref">20</a>]</span>. While ribosomes were for the most part excluded
from the numerous granules observed in the cytoplasm, in some cases we observed
clusters of LSUs in the inclusions of double-membraned granules
described earlier (Fig. [<a href="#fig:matching_euc">4</a>]F, Fig. [<a href="#fig:matching_fff">5</a>]D,E). It is, in principle, possible that these targets are situated above or
below the imaged granules, since the granule positions in z cannot be determined
using 2D projections. However, in the case of Fig. [<a href="#fig:matching_fff">5</a>]E, the
detected LSUs span the whole lamella in the z direction (Fig. [<a href="#fig:matching_fff">5</a>]F), while positions
above or below a granule would result in LSUs situated exlusively at the
top or bottom of the lamella. This is consistent with the earlier hypothesis that the inclusions are of
cytoplasmic origin.</p>
<h3 id="does-deco-lace-induce-aberrations-that-affect-2dtm">Does DeCo-LACE induce aberrations that affect 2DTM?</h3>
<p>Within the eight lamellae we found different numbers of detected targets, ranging from 1089 to 6433 per lamella (Fig.
[<a href="#fig:matching_stat">6</a>]A). Lamella<sub>EUC</sub> 1 had the most detected targets, but also has
the largest surface area and contained cytoplasm from two cells. Lamella<sub>FFF</sub> 4
had the fewest detected targets, but this particular lamella was dominated by a
circular section of the nucleus, with only small pockets of cytoplasm (Fig.
[<a href="#fig:lamella_images">4 - figure supplement 4</a>]). In an attempt to normalize for these differences in
area containing cytoplasm, we compared the number of detected targets per tile in
tiles that contained more than one target, which should exclude tiles with
non-cytosolic content (Fig. [<a href="#fig:matching_stat">6</a>]B). While this measure had
less variability, there were still differences. Lamella<sub>EUC</sub> 4 had not only the
fewest targets, but also the lowest density, which could be due to this lamella being the
thinnest, or due to it sectioning the cell in an area with a lower concentration of
ribosomes. Lamella<sub>FFF</sub> 3 had a substantially higher number of ribosomes per
tile. Since all of these lamellae were made from a cell-line under identical
conditions, this underscores the necessity to collect data from large numbers of
lamellae to overcome the inherent variability. When comparing the distribution of
scores between lamellae, we found them to be fairly comparable with median
SNRs ranging from 8.7 to 9.7 (Fig. [<a href="#fig:matching_stat">6</a>]C). Lamella<sub>EUC</sub> 1 had slightly lower scores compared
to
the rest, potentially due to its large size and connected mechanical instability
during imaging. Overall, we did not observe differences in the number or SNR of
detected targets between eucentric or fringe-free illumination conditions that
were bigger than the observed inter-lamella variability.</p>
<p>Since the SNR values of 2DTM are highly sensitive to image quality, we reasoned
we could use them to verify that DeCo-LACE does not introduce a systematic loss
of image quality. We considered non-parallel illumination introduced by the
unusually condensed beam and uncharacterized aberrations near the beam
periphery. When plotting the SNR values of detected targets in all eight
lamellae as a function of their location in the tiles, we found uniformly high
SNR values throughout the illuminated areas for both eucentric and fringe-free
focus illumination, demonstrating that both illumination schemed are suitable
for DeCo-LACE (Fig. [<a href="#fig:matching_stat">6</a>]D).</p>
<p>We also wondered whether large image shifts would lead aberration due to
astigmatism or beam tilt <span class="citation" data-cites="APSL9LmU">[<a href="#ref-APSL9LmU" role="doc-biblioref">16</a>]</span>. We reasoned that if
that was the case the number of detected targets should be highest in the center of the
lamella where the applied beam image-shift is 0. Instead, we observed that in
both eucentric and fringe-free focus conditions more targets were detected at the
“back” edge of the lamella (Fig. [<a href="#fig:matching_stat">6</a>]E]). This may be due to the center of the cell being predominantly
occupied by the nucleus, despite its segmentation in neutrophil-like cells. The
increase in matches at the “back” of the lamellae compared to the “front” can also
be explained by the thickness gradient of the lamellae (Fig. [<a href="#fig:lamella_motion_thickness">4 - figure supplement 2</a>]B,
Fig. [<a href="#fig:matching_euc">4</a>]B, Fig. [<a href="#fig:matching_fff">5</a>]B). In addition, aberrations would be expected to cause average 2DTM SNRs to be higher when beam-image shift
values are small. Instead, we found that SNRs where on average the highest at the
“front” edge of the lamellae, presumably due to the thinner sample. We therefore
conclude that factors other that beam image-shift or beam
condensation aberrations are limiting 2DTM SNRS, predominantly the thickness of
the lamellae.</p>
<h3 id="computation-is-the-bottleneck-of-visual-proteomics">Computation is the bottleneck of visual proteomics</h3>
<p>All lamellae described above were derived from a clonal cell line under identical condition and thinned with the same parameters. This means that the substantial variability of detected targets between the lamellae must be due to
technical variability, including area, thickness, mechanical stability, and location of the section within the cell. We therefore predict that further studies that want to draw quantitative and statistically relevant conclusions about the number and location of
molecules under different experimental conditions, will require collection of orders of magnitude more data than in this study to gain enough statistical power given this variability. The samples used were
prepared in two 24 h sessions on a FIB/SEM instrument, and imaging was performed
during another two 24h session on the TEM microscope. Inspections of the
timestamps of the raw data files revealed that the milling time per lamella was
~30 minutes and TEM imaging was accomplished in ~10 seconds per tile or 90
minutes for a ~ 6x6 μm lamella. Processing of the data, however, took
substantially longer. Specifically, 2DTM of all tiles took approximately one
week per lamella on 32 Nvidia A6000 GPUs. Computation is therefore a bottleneck in our
current workflow, and further optimizations of the algorithm may be necessary
increase throughput. Alternatively, this bottleneck could be reduced by
increasing the number of processing units.</p>
<h2 id="discussion">Discussion</h2>
<p>In this study we developed an approach to image entire cellular cross-section using cryo-EM at high enough resolution to allow for 2DTM detection of the LSU. The two main advantages compared to previous approaches are a high throughput of imaging and the biological context for detected molecules. The requirement to increase throughput in cryo-EM data collection of cellular samples has been recognized in the recent literature. Most approaches described so far are tailored towards tomography. Peck et al. <span class="citation" data-cites="1GPPSX111">[<a href="#ref-1GPPSX111" role="doc-biblioref">21</a>]</span> and Yang et al. <span class="citation" data-cites="ujizcpVn">[<a href="#ref-ujizcpVn" role="doc-biblioref">22</a>]</span> developed approaches to increase the FOV of tomogram data-collection by using a montaging technique. Peck et al. used a similar “condensed-beam” approach as described here. However, the montages are substantially smaller in scope, covering carbon film holes of 2 um diameter. Bouvette et al. <span class="citation" data-cites="1Gwx5ScUa">[<a href="#ref-1Gwx5ScUa" role="doc-biblioref">23</a>]</span> and Eisenstein et al. <span class="citation" data-cites="xvouNMBy">[<a href="#ref-xvouNMBy" role="doc-biblioref">24</a>]</span> are using beam image-shift to collect tilt-series in multiple locations in parallel to increase throughput. However, none of these approaches provide the full coverage of a cellular cross-section that can be achieved using DeCo-Lace.</p>
<p>Since we observed substantial variation in LSU density within and between lamellae, visual proteomics studies that use cryo-EM to establish changes in molecular organization within cells will require orders of magnitude more data than used in this study. One milestone would be to image enough data to represent one cellular volume, which for a small eukaryotic cells requires imaging approximately 100 lamella. While data collection throughput on the TEM is fundamentally limited by the exposure time, this amount of data could be collected within 12 hours by improving the data acquisition scheme to perform all necessary calculations in parallel with actual exposure of the camera. Sample preparation using a FIB/SEM is also currently a bottleneck, but preparation of large lamellae with multiple cellular cross-sections using methods like WAFFLE <span class="citation" data-cites="17H2LWiIu">[<a href="#ref-17H2LWiIu" role="doc-biblioref">25</a>]</span> might allow sufficient throughput. As stated in the results, at least for 2DTM computation will remain challenging and approximately 17,000 GPU hours would be required for a 100 lamellae dataset.</p>
<p>As described in <span class="citation" data-cites="Ynb3IP6I">[<a href="#ref-Ynb3IP6I" role="doc-biblioref">7</a>]</span> the 2DTM SNR threshold for detecting a target is chosen to result in one false positive detection per image searched. We would therefore expect to find one false positive detection per tile. We reasoned that the large nuclear area imaged by DeCo-Lace could be used to test whether this assumption is true. In the 670 tiles containing exclusively nucleus (as manually annotated from the overview image) we detected 247 targets, making the false-positive rate more than twofold lower than expected. Since earlier work shows that 2DTM with the LSU can produce matches to nuclear ribosome biogenesis intermediates <span class="citation" data-cites="T1Qr0xyP">[<a href="#ref-T1Qr0xyP" role="doc-biblioref">13</a>]</span>, this could even be an overestimate of the false-positive rate. This suggests that the detection threshold could be even lower, which is an area of ongoing research.</p>
<p>We found that even though we used beam image-shift extensively (up to 7 um), we did not see substantially reduced 2DTM SNR values in tiles acquired at high beam image-shift compared to tiles acquired with low or no beam image-shift. This is in contrast to reports in single-particle analysis (SPA) <span class="citation" data-cites="44fG6kcy">[<a href="#ref-44fG6kcy" role="doc-biblioref">26</a>]</span> where the induced beam tilt substantially reduced the resolution if it was not corrected during processing. It is possible that 2DTM is less sensitive to beam-tilt aberrations, since the template is free of any aberration and only the image is distorted, while in SPA the beam tilt will affect both the images and the reconstructed template.</p>
<p>As mentioned in the results, we found a consistent shading artifact pattern in our montages, that we believe is the result of non-linear behavior of the K3 camera. Indeed, when we average images with a condensed beam taken over vacuum we found in both focus conditions a consistent background pattern with a brighter region on the periphery of the illuminated area (Fig [<a href="#fig:gain">4 - figure supplement 3</a>]). This might be caused by dynamic adjustment of the internal camera counting threshold which expects columns of the sensor to be evenly illuminated as is the case for SPA applications. Since the signal of this pattern has mainly low-resolution components it is unlikely to affect 2DTM. However, it highlights that the non-linear behavior of the camera has to be taken into account when imaging samples with strongly varying density and unusual illumination schemes.</p>
<p>We observed granules containing a vesicle of putative cytosolic origin. We speculate that upon degranulation, the process in which granules fuse with the plasma membrane, these vesicles would be released into the extracelullar space. The main types of extracellular vesicles of this size are exosomes, up to 100 nm large vesicles derived from fusion of multivesicular bodies with the plasma membrane, and microvesicles, which are derived from direct budding of the plasma membrane <span class="citation" data-cites="kncnYGLD">[<a href="#ref-kncnYGLD" role="doc-biblioref">27</a>]</span>. We suggest that granulocytes could release a third type of extracellular vesicle, granule-derived vesicles (GDV), into the extracellular space. 2DTM showed that a subset of GDVs can contain ribosomes (Fig. [<a href="#fig:matching_euc">4</a>]F, Fig. [<a href="#fig:matching_fff">5</a>]D,E). This could indicate that these vesicles are transporting translation-capable mRNAs, as has been described for exosomes <span class="citation" data-cites="agVdpX76">[<a href="#ref-agVdpX76" role="doc-biblioref">28</a>]</span>. Further studies will be necessary to confirm the existence of GDVs in granulocytes isolated from mammals and to understand their functional significance.</p>
<h2 id="materials-and-methods">Materials and Methods</h2>
<h3 id="grid-preparation">Grid preparation</h3>
<p>ER-HoxB8 cells were maintained in RPMI medium supplemented with 10% FBS,
penicillin/streptomycin, SCF, and estrogen <span class="citation" data-cites="1B9Vt9eYu">[<a href="#ref-1B9Vt9eYu" role="doc-biblioref">10</a>]</span> at
37 °C and 5% CO2. 120 h prior to grid preparation, cells were washed twice in PBS
and cultured in the same medium except without estrogen. Differentiation was
verified by staining with Hoechst-dye and inspection of nuclear morphology.
Cells were then counted and diluted to <span class="math inline">\(1\cdot10^6\)</span> cells/ml. Grids (either 200
mesh copper grids, with a sillicone-oxide and 2 µm holes with a 2 µm spacing or
200 mesh gold grids with a thin gold film and 2 µm holes in 2 µm spacing) were
glow-discharged from both sides using a 15 mA for 45 s. 3.5 µl of cell suspension
was added to grids on the thin-film side and grids were blotted from the back
side using a GP2 cryoplunger (Leica) for 8 s and rapidly plunged into liquid
ethane at -185 °C.</p>
<h3 id="fib-milling">FIB-milling</h3>
<p>Grids were loaded into an Aquilos 2 FIB/SEM (Thermo Fisher) instrument with a
stage cooled to -190 °C. Grids were sputter-coated with platinum for 15 s at 45 mA
and then coated with a layer of platinum-precursor by opening the GIS-valve for
45 s. An overview of the grid was created by montaging SEM images and isolated
cells at the center of gridsquares were selected for FIB-milling. Lamellae were
generated automatically using the AutoTEM software (Thermo Fisher), with the
following parameters:</p>
<ul>
<li>Milling angle: 20°</li>
<li>Rough milling: 3.2 µm thickness, 0.5 nA current</li>
<li>Medium milling: 1.8 µm thickness, 0.3 nA current, 1.0° overtilt</li>
<li>Fine milling: 1.0 µm tchickness, 0.1 nA current, 0.5° overtilt</li>
<li>Finer milling: 700 nm thickness, 0.1 nA curent, 0.2° overtilt</li>
<li>Polish 1: 450 nm thickness, 50 pA current</li>
<li>Polish 2: 200 nm thickness, 30 pA current</li>
</ul>
<p>This resulted in 6-10 µm wide lamella with 150-250 nm thickness as determined by
FIB-imaging of the lamella edges.</p>
<h3 id="data-collection">Data collection</h3>
<p>Grids were loaded into a Titan Krios TEM (Thermo Fisher) operated at 300 keV and
equipped with a BioQuantum energy filter (Gatan) and K3 camera (Gatan). The
microscope was aligned using a cross-grating grid on the stage. Prior to each
session, we carefully performed the “Image/Beam” calibration in nanoprobe. We
set the magnification to a pixel size of 1.76 Å and condensed the beam to ~ 900 nm
diameter, resulting in the beam being completely visible on the camera. To
establish fringe-free conditions, the “Fine eucentric” procedure of SerialEM <span class="citation" data-cites="19ZFerhph">[<a href="#ref-19ZFerhph" role="doc-biblioref">29</a>]</span> was
used to move a square of the cross-grating grid to the eucentric position of the
microscope. The effective defocus was then set to 2 µm, using the “autofocus”
routine of SerialEM. The objective focus of the microscope was changed until no
fringes were visible. The stage was then moved in Z until images had an apparent
defocus of 2 µm. The difference in stage Z-position between the eucentric and
fringe-free conditions was used to move other areas into fringe-free condition.</p>
<p>Low magnification montages were used to find lamellae and lamellae that were
sufficiently thin and free of contamination were selected for automated data
collection. Overview images of each lamella were taken at 2250x magnification
(38 Å pixel size). The corners of the lamella in the overview image were manually
annotated in SerialEM and translated into beam image-shift values using SerialEM’s
calibration. A hexagonal pattern of beam image-shift positions was calculated
that covered the area between the four corners in a serpentine way, with a
<span class="math inline">\(\sqrt{3}\cdot425\)</span> nm horizontal spacing and <span class="math inline">\(3/4\cdot 850\)</span> nm vertical spacing. Exposures were
taken at each position with a 30 e<sup>-</sup>/Å<span class="math inline">\(^2\)</span> total dose. After each exposure, the
defocus was estimated using the ctffind function of SerialEM and the focus for
the next exposure was corrected by the difference between the estimated focus
and the desired defocus of 800 nm. Furthermore, after each exposure the
deviation of the beam from the center of the camera was measured and corrected
using the “CenterBeamFromImage” command of SerialEM.</p>
<p>After data collection, a 20 s exposure at 2250x magnification of the lamella at
200 μm defocus was taken for visualization purposes. A Python script implementing
this procedure is available at [Link to repo].</p>
<h3 id="deco-lace-data-processing">DeCo-LACE data processing</h3>
<p>An overview of the data analysis pipeline is shown in Fig.
<a href="#fig:deco_lace_workflow">7</a>.</p>
<h4 id="pre-processing">Pre-processing</h4>
<p>Motion-correction, dose weighting and other preprocessing as detailed below was performed using <em>cis</em>TEM <span class="citation" data-cites="MUY42yxp">[<a href="#ref-MUY42yxp" role="doc-biblioref">30</a>]</span>. To avoid influence of the beam-edge on motion-correction, only a quarter of the
movie in the center of the camera was considered for calculation of the
estimated motion. After movie frames were aligned and summed, a mask for the
illuminated area was calculated by lowpass filtering the image with a 100 Å
resolution cutoff, thresholding the image at 10% of the maximal value and then lowpass
filtering the mask again with a 100 Å resolution cutoff to smooth the mask edges. This mask was
then used to select dark areas in the image and fill the pixels with
Gaussian noise, with the same mean and standard deviation as the illuminated
area. A custom version of the unblur program <span class="citation" data-cites="1G4Y94qCy">[<a href="#ref-1G4Y94qCy" role="doc-biblioref">31</a>]</span>
implementing this procedure is available at [link to decolace branch]. During motion correction images were resampled to a pixel size of 1.5 Å. The
contrast-transfer function (CTF) was estimated using ctffind
<span class="citation" data-cites="n43f3Sqs">[<a href="#ref-n43f3Sqs" role="doc-biblioref">32</a>]</span>, searching between 0.2 and 2 μm defocus.</p>
<h4 id="dtm">2DTM</h4>
<p>The search template was generated from the atomic model of the mouse LSU
(PDB 6SWA, exluding the Epb1 subunit) using the cryo-EM simulator implemented in <em>cis</em>TEM
<span class="citation" data-cites="q2PSamkS">[<a href="#ref-q2PSamkS" role="doc-biblioref">33</a>]</span>. The
match_template program <span class="citation" data-cites="10bXZuF3G">[<a href="#ref-10bXZuF3G" role="doc-biblioref">9</a>]</span> was used to search for this
template in the movie-aligned, exposure-filtered and masked images, using a 1.5°
angular step in out-of-plane angles and a 1.0° angular step in-plane. 11 defocus
planes in 20 nm steps centered around the ctffind-determined defocus were searched. Targets
were defined as detected when their matches with the template produced peaks
with a singal-to-noise ratio (SNR) above a threshold of 7.75, which was chosen
based on the one-false-positive-per-tile criterion <span class="citation" data-cites="Ynb3IP6I">[<a href="#ref-Ynb3IP6I" role="doc-biblioref">7</a>]</span>.</p>
<h4 id="montage-assembly">Montage assembly</h4>
<p>The coordinates of each tile <span class="math inline">\(i\)</span>,
<span class="math inline">\(\mathbf{c}_{i}\)</span> [2D Vector in pixels] were initialized using beam image-shift of the tile, <span class="math inline">\(\mathbf{b}_i\)</span> [2D Vector in μm],
and the ISToCamera matrix <span class="math inline">\(\mathbf{IC}\)</span>, as calibrated by SerialEM:</p>
<p><span class="math display">\[\mathbf{c}_{i} = \mathbf{IC} \cdot \mathbf{b}_i\]</span></p>
<p>A list of tile pairs <span class="math inline">\(i,j\)</span> that overlap were assembled by selecting images where
<span class="math inline">\(|\mathbf{c}_i-\mathbf{c}_j| &lt; D_{Beam}\)</span>. In order to calculate the precise offset between tiles <span class="math inline">\(i\)</span> and <span class="math inline">\(j\)</span>, <span class="math inline">\(\mathbf{r}_{i,j}\)</span>, we calculated the cross-correlation between the two tiles, masked to the overlapping illuminated area using the scikit-image
package <span class="citation" data-cites="stvWEJeu">[<a href="#ref-stvWEJeu" role="doc-biblioref">34</a>]</span> was used to calculate refined offsets . The coordinates <span class="math inline">\(\mathbf{c}_{i}\)</span> were then refined by a least-square minimization against <span class="math inline">\(\mathbf{r}_{i,j}\)</span>:</p>
<p><span class="math display">\[ \displaystyle{\min_{\mathbf{c}} \sum_{pairs}{(\mathbf{r}_{i,j} - (\mathbf{c}_i-\mathbf{c}_j))^2}}\]</span></p>
<p>using the scipy package <span class="citation" data-cites="8Miti2Gz">[<a href="#ref-8Miti2Gz" role="doc-biblioref">35</a>]</span>. The masked cross-correlation and the least-square minimization was repeated once more to arrive at the final tile alignment.</p>
<p>The x,y coordinates of target <span class="math inline">\(n\)</span> detected by 2DTM in the tile <span class="math inline">\(i\)</span>,
<span class="math inline">\(\textbf{m}^\textrm{T}_{n,i}\)</span>, was transformed into the montage frame by adding
the coordinate of the tile.</p>
<p><span class="math display">\[ \textbf{m}^\textrm{M}_n = \textbf{m}^\textrm{T}_{n,i} + \textbf{c}_i\]</span></p>
<p>The z coordinate of each target was calculated as the sum of the defocus offset
for the target, the estimated defocus of the tile, and the nominal defocus of
the microscope when the tile was acquired.</p>
<p>Images were rendered using UCSF ChimeraX <span class="citation" data-cites="cYCuuZxG">[<a href="#ref-cYCuuZxG" role="doc-biblioref">36</a>]</span> using a custom extension to render 2DTM results available at [https://github.com/jojoelfe/tempest]. The Python scripts used for data processing are available
under [https://github.com/jojoelfe/deco_lace_template_matching_manuscript].</p>
<h2 id="acknowledgments">Acknowledgments</h2>
<p>The authors would like to thank Bronwyn Lucas, Carsten Sachse, and Chen Xue for helpful suggestions and careful reading of the manuscript. Data was collected at the UMass Chan medical School CryoEM core with help by Kankang song, Christna, and Chen Xue.</p>
<h2 id="data-availability">Data availability</h2>
<p>Cryo-EM movies, motion-corrected images and template matching results have been deposited in EMPIAR under accession code [CODE]. The custom cisTEM version is available here [] until features have been integrated into a cisTEM release. The ChimeraX extension for rendering is avaibale here []. This manuscript was prepared using the manubot package [@ cite] . Custom python scripts and all files nescessary for manuscript generation are available here [].</p>
<h2 id="figures">Figures</h2>
<div id="fig:initmatching" class="fignos">
<figure>
<img src="figures/initmatching.png" alt="Figure 1: 2D template matching of the large subunit of the ribosome in fib-milled neutrophil-like cells (A) Overview image of the lamella. Major cellular regions are labeled, as Nucleus (Nuc), Mitochondria (M), and granular cytoplasm (GrCyt). FOVs where high-magnification images for template matching where acquired are indicated as boxes with the number of detected targets indicated on the bottom right. FOVs displayed in Panels B-E are color-coded. Scalebar corresponds to 1 μm. (B-E) FOVs with projection of detected LSUs shown in cyan. (B) Perinuclear region, the only detected targets are in the cytoplasmic half. (C) Cytoplasmic region with high density of ribosomes (D) Mitochondrium, as expected there are only detected LSUs in the cytoplasmic region (E) Cytoplasm, with low density of ribosomes." />
<figcaption aria-hidden="true"><span>Figure 1:</span> 2D template matching of the large subunit of the ribosome in fib-milled neutrophil-like cells
(A) Overview image of the lamella. Major cellular regions are labeled, as Nucleus (Nuc), Mitochondria (M), and granular cytoplasm (GrCyt). FOVs where high-magnification images for template matching where acquired are indicated as boxes with the number of detected targets indicated on the bottom right. FOVs displayed in Panels B-E are color-coded. Scalebar corresponds to 1 μm.
(B-E) FOVs with projection of detected LSUs shown in cyan. (B) Perinuclear region, the only detected targets are in the cytoplasmic half. (C) Cytoplasmic region with high density of ribosomes (D) Mitochondrium, as expected there are only detected LSUs in the cytoplasmic region (E) Cytoplasm, with low density of ribosomes.</figcaption>
</figure>
</div>
<div id="fig:initmatching2" class="fignos">
<figure>
<img src="figures/initmatching2.png" data-tag="2 - figure supplement 1" alt="Figure 2 - figure supplement 1: 2D template matching of the large subunit of the ribosome in fib-milled neutrophil-like cells (A) Maximum intensity projection (MIP) cross-correlation map of micrograph shown in Figure 1 (B+C) 3D plot of MIP regions indicated by color boxes in Panel A" />
<figcaption aria-hidden="true"><span>Figure 2 - figure supplement 1:</span> 2D template matching of the large subunit of the ribosome in fib-milled
neutrophil-like cells (A) Maximum intensity projection (MIP) cross-correlation map of
micrograph shown in Figure
<a href="#fig:initmatching">1</a> (B+C) 3D plot of MIP regions indicated by color boxes in Panel A</figcaption>
</figure>
</div>
<div id="fig:approach" class="fignos">
<figure>
<img src="figures/approach.png" alt="Figure 2: DeCo-LACE approach (A) Graphic demonstrating the data-collection strategy for DeCo-LACE. The electron beam is condensed to a diameter D_{Beam} that allows captured of the whole illuminated area on the camera. Beam-image shift along X and Y (BIS_X,$BIS_Y) is used to scan the whole lamella (B) Diagram of the collection algorithm (C) Example overview image of a lamella with the designated acquisition positions and the used beam diameter indicated with red circles. Scalebar corresponds to 1 μm. (D+E) Representative micrographs takne with a condensed beam at eucentric focus (D) or fringe-free focus (E). Scalebar corresponds to 100 nm. (F) Boxplot of defocus measured by ctffind of micrographs taken by the DeCo-Lace approach on four lamellae images at eucentric focus and four lamellae imaged with fringe-free focus." />
<figcaption aria-hidden="true"><span>Figure 2:</span> DeCo-LACE approach (A) Graphic demonstrating the data-collection strategy for
DeCo-LACE. The electron beam is condensed to a diameter <span class="math inline">\(D_{Beam}\)</span> that allows captured of
the whole illuminated area on the camera. Beam-image shift along X and Y
(<span class="math inline">\(BIS_X\)</span>,$<span class="math inline">\(BIS_Y\)</span>) is used to scan the whole lamella
(B) Diagram of the collection algorithm
(C) Example overview image of a lamella with the designated acquisition
positions and the used beam diameter indicated with red circles. Scalebar corresponds to 1 μm.
(D+E) Representative micrographs takne with a condensed beam at eucentric focus
(D) or fringe-free focus (E). Scalebar corresponds to 100 nm.
(F) Boxplot of defocus measured by ctffind of micrographs taken by the DeCo-Lace
approach on four lamellae images at eucentric focus and four lamellae imaged with
fringe-free focus.</figcaption>
</figure>
</div>
<div id="fig:lamella_spatial_info" class="fignos">
<figure>
<img src="figures/lamella_spatial_info.png" data-tag="3 - figure supplement 1" alt="Figure 3 - figure supplement 1: Defocus estimation of individual tiles of DeCo-Lace montages (A) Defocus values of individual micrographs taken using the DeCo-Lace approach plotted as a function of the beam image-shift values. (B) Defocus astigmatism of individual micrographs taken using the DeCo-Lace approach plotted as a function of the beam image-shift values." />
<figcaption aria-hidden="true"><span>Figure 3 - figure supplement 1:</span> Defocus estimation of individual tiles of DeCo-Lace montages
(A) Defocus values of individual micrographs taken using the DeCo-Lace approach
plotted as a function of the beam image-shift values.
(B) Defocus astigmatism of individual micrographs taken using the DeCo-Lace approach
plotted as a function of the beam image-shift values.</figcaption>
</figure>
</div>
<div id="fig:assembly" class="fignos">
<figure>
<img src="figures/assembly.png" alt="Figure 3: Assembling DeCo-LACE exposures into montages (A) Overview image of LamellaEUC 1 taken at low magnification. Scalebar corresponds to 1 μm. (B) Overview of LamellaEUC 1 created by montaging high magnification images taken with the DeCo-Lace approach. Scalebar corresponds to 1 μm. (C) Zoom-in into red box in panel B. Slight beam-fringe artifacts are visible. Scalebar corresponds to 100 nm. (D) Overview image of LamellaFFF 4 taken at low magnification. Scalebar corresponds to 1 μm. (E) Overview of LamellaFFF 4 created by montaging high magnification images taken with the DeCo-Lace approach. Scalebar corresponds to 1 μm. (F) Zoom-in into red box in panel E. No beam-fringe artifacts are visible. Scalebar corresponds to 100 nm." />
<figcaption aria-hidden="true"><span>Figure 3:</span> Assembling DeCo-LACE exposures into montages (A) Overview
image of Lamella<sub>EUC</sub> 1 taken at low magnification. Scalebar corresponds to 1 μm. (B) Overview of Lamella<sub>EUC</sub> 1 created by
montaging high magnification images taken with the DeCo-Lace approach. Scalebar corresponds to 1 μm. (C) Zoom-in into red box in panel B. Slight beam-fringe artifacts are visible. Scalebar corresponds to 100 nm.
(D) Overview
image of Lamella<sub>FFF</sub> 4 taken at low magnification. Scalebar corresponds to 1 μm. (E) Overview of Lamella<sub>FFF</sub> 4 created by
montaging high magnification images taken with the DeCo-Lace approach. Scalebar corresponds to 1 μm. (F) Zoom-in into red box in panel E. No beam-fringe artifacts are visible. Scalebar corresponds to 100 nm.</figcaption>
</figure>
</div>
<div id="fig:crop_unblur" class="fignos">
<figure>
<img src="figures/crop_unblur.png" data-tag="4 - figure supplement 1" alt="Figure 4 - figure supplement 1: Motion correction of movies with condensed beams. At the top of each panel is an average of the movie that was motion-corrected with a red dashed box indicating the region that was used to estimate shifts. Below is a graph indicating the estimated shifts of the individual frames of the movie. Below this is the MIP of 2DTM using the large subunit of the mouse ribosome. (A) Motion correction of the whole movie (B) Notion correction of a cropped region of the movie that eliminates the beam edges (C) Motion correction of the whole movie, using only the central region to estimate the shifts" />
<figcaption aria-hidden="true"><span>Figure 4 - figure supplement 1:</span> Motion correction of movies with condensed beams.
At the top of each panel is an average of the movie that was motion-corrected
with a red dashed box indicating the region that was used to estimate shifts.
Below is a graph indicating the estimated shifts of the individual frames of the
movie. Below this is the MIP of 2DTM using the large subunit of the mouse ribosome.
(A) Motion correction of the whole movie
(B) Notion correction of a cropped region of the movie that eliminates the beam
edges
(C) Motion correction of the whole movie, using only the central region to
estimate the shifts</figcaption>
</figure>
</div>
<div id="fig:lamella_motion_thickness" class="fignos">
<figure>
<img src="figures/lamella_motion_thickness.png" data-tag="4 - figure supplement 2" alt="Figure 4 - figure supplement 2: Motion correction of individual tiles imaged using the DeCo-LACE approach (A) Total estimated motion of individual micrographs taken using the DeCo-Lace approach plotted as a function of the beam image-shift values. (B) Electron intensity of individual micrographs taken using the DeCo-Lace approach plotted as a function of the beam image-shift values." />
<figcaption aria-hidden="true"><span>Figure 4 - figure supplement 2:</span> Motion correction of individual tiles imaged using the DeCo-LACE approach
(A) Total estimated motion of individual micrographs taken using the DeCo-Lace approach
plotted as a function of the beam image-shift values.
(B) Electron intensity of individual micrographs taken using the DeCo-Lace approach
plotted as a function of the beam image-shift values.</figcaption>
</figure>
</div>
<div id="fig:gain" class="fignos">
<figure>
<img src="figures/gain.png" data-tag="4 - figure supplement 3" alt="Figure 4 - figure supplement 3: Averages of micrographs taken with a condensed beam over vacuum using a Gatan K3 detector. Contrast and Brightness have been adjusted to highlight uneven dose response. (A) Eucentric Focus (B) Fringe-free Focus" />
<figcaption aria-hidden="true"><span>Figure 4 - figure supplement 3:</span> Averages of micrographs taken with a condensed beam over vacuum using a Gatan K3 detector. Contrast and Brightness have been adjusted to highlight uneven dose response. (A) Eucentric Focus (B) Fringe-free Focus</figcaption>
</figure>
</div>
<div id="fig:lamella_images" class="fignos">
<figure>
<img src="figures/lamella_images.png" data-tag="4 - figure supplement 4" alt="Figure 4 - figure supplement 4: Overview images of lamellae imaged using the DeCo-LACE approach taken at low-magnification (A) Overviews taken at low magnification. Scalebar corresponds to 1 μm. (B) Overviews assembled using the DeCo-LACE approach. Scalebar corresponds to 1 μm. (C) Representative examples of a class of granules containing a putatively cytosolic inclusion. Scalebar corresponds to 100 nm." />
<figcaption aria-hidden="true"><span>Figure 4 - figure supplement 4:</span> Overview images of lamellae imaged using the DeCo-LACE approach taken at low-magnification (A) Overviews taken at low magnification. Scalebar corresponds to 1 μm. (B) Overviews assembled using the DeCo-LACE approach. Scalebar corresponds to 1 μm. (C) Representative examples of a class of granules containing a putatively cytosolic inclusion. Scalebar corresponds to 100 nm.</figcaption>
</figure>
</div>
<div id="fig:matching_euc" class="fignos">
<figure>
<img src="figures/matching_euc.png" alt="Figure 4: Template matching in lamella imaged using the DeCo-Lace approach at eucentric focus (A) Montage of Lamella_\textrm{EUC} 1 overlaid with detected targets colored in orange. Scalebar corresponds to 1 μm. (B) Side view of detected targets in the lamella, such that the direction of the electron beam is horizontal. (C-F) Magnified area of panel A showing rough ER with associated ribosomes (C), outer nuclear membrane with associated ribosomes (D), ribsomes arranged in a circular fashion (E), ribosomes enclosed in a less electron dense inclusion in a granule (F). Ribosomes are colored in white with the surface of the peptide exit tunnel colored in green and the A, P, and E sites colored in blue, purple, and red, respectively.Scalebar corresponds to 100 nm." />
<figcaption aria-hidden="true"><span>Figure 4:</span> Template matching in lamella imaged using the DeCo-Lace approach at eucentric
focus (A) Montage of Lamella<span class="math inline">\(_\textrm{EUC}\)</span> 1 overlaid with detected targets colored in orange. Scalebar corresponds to 1 μm. (B) Side view of detected targets in the lamella, such that the
direction of the electron beam is horizontal. (C-F) Magnified area of panel A
showing rough ER with associated ribosomes (C), outer nuclear membrane with
associated ribosomes (D), ribsomes arranged in a circular fashion (E), ribosomes
enclosed in a less electron dense inclusion in a granule (F). Ribosomes are colored in white with the surface of the peptide exit tunnel colored in green and the A, P, and E sites colored in blue, purple, and red, respectively.Scalebar corresponds to 100 nm.</figcaption>
</figure>
</div>
<div id="fig:matching_fff" class="fignos">
<figure>
<img src="figures/matching_fff.png" alt="Figure 5: Template matching in lamella imaged using the DeCo-Lace approach at fringe-free focus (A) Montage of Lamella_\textrm{FFF} 4 overlaid with detected targets colored in orange. Scalebar corresponds to 1 μm. (B) Side view of detected targets in the lamella, such that the direction of the electron beam is horizontal. (C-E) Magnified area of panel A showing rough ER with associated ribosomes (C) and ribosomes enclosed in a less electron dense inclusion in a granule (D,E). (F) Side view of panel E. Ribosomes are colored in white with the surface of the peptide exit tunnel colored in green and the A, P, and E sites colored in blue, purple, and red, respectively. Scalebar corresponds to 100 nm." />
<figcaption aria-hidden="true"><span>Figure 5:</span> Template matching in lamella imaged using the DeCo-Lace approach at fringe-free
focus (A) Montage of Lamella<span class="math inline">\(_\textrm{FFF}\)</span> 4 overlaid with detected targets colored in orange. Scalebar corresponds to 1 μm. (B) Side view of detected targets in the lamella, such that the
direction of the electron beam is horizontal. (C-E) Magnified area of panel A
showing rough ER with associated ribosomes (C) and ribosomes
enclosed in a less electron dense inclusion in a granule (D,E). (F) Side view of
panel E. Ribosomes are colored in white with the surface of the peptide exit tunnel colored in green and the A, P, and E sites colored in blue, purple, and red, respectively. Scalebar corresponds to 100 nm.</figcaption>
</figure>
</div>
<div id="fig:matching_stat" class="fignos">
<figure>
<img src="figures/matching_stat.png" alt="Figure 6: Statistics of 2DTM on lamella imaged using DeCo-LACE (A) Number of detected targets in each lamella (B) Distribution of targets per tile in each lamella. Only tiles with two or more detected targets were included (C) Distribution of SNRs in each lamella (D) For each lamella an average of all tiles is shown. Overlaid is a scatterplot of all detected targets in these tiles according to their in-tile coordinates. Scatterplot is colored according to the 2DTM SNR. There are no detected targets in the top circle-circle intersection due to radiation damage from previous exposures. (E) 2D histogram of number of detected targets as a function of beam-image shift (F) Mean 2DTM SNR as a function of beam-image shift" />
<figcaption aria-hidden="true"><span>Figure 6:</span> Statistics of 2DTM on lamella imaged using DeCo-LACE (A) Number of detected targets in
each lamella (B) Distribution of targets per tile in each lamella. Only tiles
with two or more detected targets were included (C) Distribution of SNRs in each lamella
(D) For each lamella an average of all tiles is shown. Overlaid is a scatterplot
of all detected targets in these tiles according to their in-tile coordinates.
Scatterplot is colored according to the 2DTM SNR. There are no detected targets in the top
circle-circle intersection due to radiation damage from previous exposures. (E) 2D histogram of number of detected targets as a function of beam-image shift (F) Mean 2DTM SNR as a function of beam-image shift</figcaption>
</figure>
</div>
<div id="fig:deco_lace_workflow" class="fignos">
<figure>
<img src="figures/deco_lace_workflow.png" alt="Figure 7: Workflow of DeCo-Lace processing" />
<figcaption aria-hidden="true"><span>Figure 7:</span> Workflow of DeCo-Lace processing</figcaption>
</figure>
</div>
<div id="fig:movie1" class="fignos">
<figure>
<img src="images/1x1-00000000.png" data-tag="Movie 1" alt="Figure Movie 1: Movie of detected LSU targets in Lamella_\textrm{EUC} 1, corresponding to Figure 5" />
<figcaption aria-hidden="true"><span>Figure Movie 1:</span> Movie of detected LSU targets in Lamella<span class="math inline">\(_\textrm{EUC}\)</span> 1, corresponding to Figure 5</figcaption>
</figure>
</div>
<div id="fig:movie2" class="fignos">
<figure>
<img src="images/1x1-00000000.png" data-tag="Movie 2" alt="Figure Movie 2: Movie of detected LSU targets in Lamella_\textrm{FFF} 4, corresponding to Figure 6" />
<figcaption aria-hidden="true"><span>Figure Movie 2:</span> Movie of detected LSU targets in Lamella<span class="math inline">\(_\textrm{FFF}\)</span> 4, corresponding to Figure 6</figcaption>
</figure>
</div>
<h2 class="page_break_before" id="references">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-tSXIKPl7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1. </div><div class="csl-right-inline"><strong>Label-free, normalized quantification of complex mass spectrometry data for proteomic analysis</strong> <div class="csl-block">Noelle M Griffin, Jingyi Yu, Fred Long, Phil Oh, Sabrina Shore, Yan Li, Jim A Koziol, Jan E Schnitzer</div> <em>Nature Biotechnology</em> (2010-01) <a href="https://doi.org/fshgnc">https://doi.org/fshgnc</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nbt.1592">10.1038/nbt.1592</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20010810">20010810</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2805705">PMC2805705</a></div></div>
</div>
<div id="ref-VBmW7Aot" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2. </div><div class="csl-right-inline"><strong>Fluorescence microscopy</strong> <div class="csl-block">Jeff W Lichtman, José-Angel Conchello</div> <em>Nature Methods</em> (2005-11-18) <a href="https://doi.org/bbpg4n">https://doi.org/bbpg4n</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth817">10.1038/nmeth817</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16299476">16299476</a></div></div>
</div>
<div id="ref-tGQ6TSUo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">3. </div><div class="csl-right-inline"><strong>A visual approach to proteomics</strong> <div class="csl-block">Stephan Nickell, Christine Kofler, Andrew P Leis, Wolfgang Baumeister</div> <em>Nature Reviews Molecular Cell Biology</em> (2006-02-15) <a href="https://doi.org/d6d5mq">https://doi.org/d6d5mq</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nrm1861">10.1038/nrm1861</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16482091">16482091</a></div></div>
</div>
<div id="ref-g8QavfwP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">4. </div><div class="csl-right-inline"><strong>Electron microscopy of frozen hydrated sections of vitreous ice and vitrified biological samples</strong> <div class="csl-block">AW McDowall, J-J Chang, R Freeman, J Lepault, CA Walter, J Dubochet</div> <em>Journal of Microscopy</em> (1983-07) <a href="https://doi.org/bdnzmv">https://doi.org/bdnzmv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1111/j.1365-2818.1983.tb04225.x">10.1111/j.1365-2818.1983.tb04225.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/6350598">6350598</a></div></div>
</div>
<div id="ref-16IhS1Nc4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">5. </div><div class="csl-right-inline"><strong>Opening windows into the cell: focused-ion-beam milling for cryo-electron tomography</strong> <div class="csl-block">Elizabeth Villa, Miroslava Schaffer, Jürgen M Plitzko, Wolfgang Baumeister</div> <em>Current Opinion in Structural Biology</em> (2013-10) <a href="https://doi.org/f537jp">https://doi.org/f537jp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.sbi.2013.08.006">10.1016/j.sbi.2013.08.006</a></div></div>
</div>
<div id="ref-Rksh2dxu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">6. </div><div class="csl-right-inline"><strong>Electron tomography of cells</strong> <div class="csl-block">Lu Gan, Grant J Jensen</div> <em>Quarterly Reviews of Biophysics</em> (2011-11-15) <a href="https://doi.org/czj9hr">https://doi.org/czj9hr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1017/s0033583511000102">10.1017/s0033583511000102</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22082691">22082691</a></div></div>
</div>
<div id="ref-Ynb3IP6I" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">7. </div><div class="csl-right-inline"><strong>Single-protein detection in crowded molecular environments in cryo-EM images</strong> <div class="csl-block">JPeter Rickgauer, Nikolaus Grigorieff, Winfried Denk</div> <em>eLife</em> (2017-05-03) <a href="https://doi.org/gnq4q4">https://doi.org/gnq4q4</a> <div class="csl-block">DOI: <a href="https://doi.org/10.7554/elife.25648">10.7554/elife.25648</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28467302">28467302</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5453696">PMC5453696</a></div></div>
</div>
<div id="ref-18KGpXYPE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">8. </div><div class="csl-right-inline"><strong>Label-free single-instance protein detection in vitrified cells</strong> <div class="csl-block">JPeter Rickgauer, Heejun Choi, Jennifer Lippincott-Schwartz, Winfried Denk</div> <em>Cold Spring Harbor Laboratory</em> (2020-04-24) <a href="https://doi.org/gpbjfd">https://doi.org/gpbjfd</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2020.04.22.053868">10.1101/2020.04.22.053868</a></div></div>
</div>
<div id="ref-10bXZuF3G" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">9. </div><div class="csl-right-inline"><strong>Locating macromolecular assemblies in cells by 2D template matching with cisTEM</strong> <div class="csl-block">Bronwyn A Lucas, Benjamin A Himes, Liang Xue, Timothy Grant, Julia Mahamid, Nikolaus Grigorieff</div> <em>eLife</em> (2021-06-11) <a href="https://doi.org/gkkc49">https://doi.org/gkkc49</a> <div class="csl-block">DOI: <a href="https://doi.org/10.7554/elife.68946">10.7554/elife.68946</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34114559">34114559</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8219381">PMC8219381</a></div></div>
</div>
<div id="ref-1B9Vt9eYu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">10. </div><div class="csl-right-inline"><strong>Inhibition of Dihydroorotate Dehydrogenase Overcomes Differentiation Blockade in Acute Myeloid Leukemia</strong> <div class="csl-block">David B Sykes, Youmna S Kfoury, François E Mercier, Mathias J Wawer, Jason M Law, Mark K Haynes, Timothy A Lewis, Amir Schajnovitz, Esha Jain, Dongjun Lee, … David T Scadden</div> <em>Cell</em> (2016-09) <a href="https://doi.org/f3r5jr">https://doi.org/f3r5jr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cell.2016.08.057">10.1016/j.cell.2016.08.057</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27641501">27641501</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7360335">PMC7360335</a></div></div>
</div>
<div id="ref-gRoY21jY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">11. </div><div class="csl-right-inline"><strong>Hallmarks of ribosomopathies</strong> <div class="csl-block">Kim R Kampen, Sergey O Sulima, Stijn Vereecke, Kim De Keersmaecker</div> <em>Nucleic Acids Research</em> (2019-07-27) <a href="https://doi.org/gpbjfm">https://doi.org/gpbjfm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/nar/gkz637">10.1093/nar/gkz637</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31350888">31350888</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7026650">PMC7026650</a></div></div>
</div>
<div id="ref-KAJ7221k" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">12. </div><div class="csl-right-inline"><strong>Diagnostic and prognostic implications of ribosomal protein transcript expression patterns in human cancers</strong> <div class="csl-block">James M Dolezal, Arie P Dash, Edward V Prochownik</div> <em>BMC Cancer</em> (2018-03-12) <a href="https://doi.org/gc87j9">https://doi.org/gc87j9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/s12885-018-4178-z">10.1186/s12885-018-4178-z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29530001">29530001</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5848553">PMC5848553</a></div></div>
</div>
<div id="ref-T1Qr0xyP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">13. </div><div class="csl-right-inline"><strong>&lt;i&gt;In situ&lt;/i&gt; single particle classification reveals distinct 60S maturation intermediates in cells</strong> <div class="csl-block">Bronwyn A Lucas, Kexin Zhang, Sarah Loerch, Nikolaus Grigorieff</div> <em>Cold Spring Harbor Laboratory</em> (2022-04-10) <a href="https://doi.org/gp4qrx">https://doi.org/gp4qrx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2022.04.10.487797">10.1101/2022.04.10.487797</a></div></div>
</div>
<div id="ref-sbkam02a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">14. </div><div class="csl-right-inline"><strong>Protein Synthesis in the Developing Neocortex at Near-Atomic Resolution Reveals Ebp1-Mediated Neuronal Proteostasis at the 60S Tunnel Exit</strong> <div class="csl-block">Matthew L Kraushar, Ferdinand Krupp, Dermot Harnett, Paul Turko, Mateusz C Ambrozkiewicz, Thiemo Sprink, Koshi Imami, Manuel Günnigmann, Ulrike Zinnall, Carlos H Vieira-Vieira, … Christian MT Spahn</div> <em>Molecular Cell</em> (2021-01) <a href="https://doi.org/gh7d72">https://doi.org/gh7d72</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.molcel.2020.11.037">10.1016/j.molcel.2020.11.037</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33357414">33357414</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8163098">PMC8163098</a></div></div>
</div>
<div id="ref-wsFP5c54" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">15. </div><div class="csl-right-inline"><strong>Advances in Single Particle Analysis Data Acquisition</strong> <div class="csl-block">Stan Konings, Maarten Kuijper, Jeroen Keizer, Fanis Grollios, Tjerk Spanjer, Peter Tiemeijer</div> <em>Microscopy and Microanalysis</em> (2019-08) <a href="https://doi.org/gp9bqm">https://doi.org/gp9bqm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1017/s1431927619005798">10.1017/s1431927619005798</a></div></div>
</div>
<div id="ref-APSL9LmU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">16. </div><div class="csl-right-inline"><strong>High-quality, high-throughput cryo-electron microscopy data collection via beam tilt and astigmatism-free beam-image shift</strong> <div class="csl-block">Chunling Wu, Xiaojun Huang, Jing Cheng, Dongjie Zhu, Xinzheng Zhang</div> <em>Journal of Structural Biology</em> (2019-12) <a href="https://doi.org/gp2nq9">https://doi.org/gp2nq9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jsb.2019.09.013">10.1016/j.jsb.2019.09.013</a></div></div>
</div>
<div id="ref-xE2EhAcH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">17. </div><div class="csl-right-inline"><strong>THE DEVELOPMENT OF NEUTROPHILIC POLYMORPHONUCLEAR LEUKOCYTES IN HUMAN BONE MARROW</strong> <div class="csl-block">Dorothy Ford Bainton, Joan L Ullyot, Marilyn G Farquhar</div> <em>Journal of Experimental Medicine</em> (1971-10-01) <a href="https://doi.org/chkbcm">https://doi.org/chkbcm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1084/jem.134.4.907">10.1084/jem.134.4.907</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/4106490">4106490</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2138991">PMC2138991</a></div></div>
</div>
<div id="ref-17IP4Lhz1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">18. </div><div class="csl-right-inline"><strong>Routine determination of ice thickness for cryo-EM grids</strong> <div class="csl-block">William J Rice, Anchi Cheng, Alex J Noble, Edward T Eng, Laura Y Kim, Bridget Carragher, Clinton S Potter</div> <em>Journal of Structural Biology</em> (2018-10) <a href="https://doi.org/gfbpj2">https://doi.org/gfbpj2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jsb.2018.06.007">10.1016/j.jsb.2018.06.007</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29981485">29981485</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6119488">PMC6119488</a></div></div>
</div>
<div id="ref-hr3dyUeI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">19. </div><div class="csl-right-inline"><strong>Optimized cryo-focused ion beam sample preparation aimed at in situ structural studies of membrane proteins</strong> <div class="csl-block">Miroslava Schaffer, Julia Mahamid, Benjamin D Engel, Tim Laugks, Wolfgang Baumeister, Jürgen M Plitzko</div> <em>Journal of Structural Biology</em> (2017-02) <a href="https://doi.org/f9qfr6">https://doi.org/f9qfr6</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jsb.2016.07.010">10.1016/j.jsb.2016.07.010</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27444390">27444390</a></div></div>
</div>
<div id="ref-cie13Q8F" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">20. </div><div class="csl-right-inline"><strong>Circularization of mRNA by eukaryotic translation initiation factors.</strong> <div class="csl-block">SE Wells, PE Hillner, RD Vale, AB Sachs</div> <em>Molecular cell</em> (1998-07) <a href="https://www.ncbi.nlm.nih.gov/pubmed/9702200">https://www.ncbi.nlm.nih.gov/pubmed/9702200</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/s1097-2765(00)80122-7">10.1016/s1097-2765(00)80122-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/9702200">9702200</a></div></div>
</div>
<div id="ref-1GPPSX111" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">21. </div><div class="csl-right-inline"><strong>Montage electron tomography of vitrified specimens</strong> <div class="csl-block">Ariana Peck, Stephen D Carter, Huanghao Mai, Songye Chen, Alister Burt, Grant J Jensen</div> <em>Journal of Structural Biology</em> (2022-06) <a href="https://doi.org/gp4qrs">https://doi.org/gp4qrs</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jsb.2022.107860">10.1016/j.jsb.2022.107860</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35487464">35487464</a></div></div>
</div>
<div id="ref-ujizcpVn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">22. </div><div class="csl-right-inline"><strong>Correlative cryogenic montage electron tomography for comprehensive in-situ whole-cell structural studies</strong> <div class="csl-block">Jie E Yang, Matthew R Larson, Bryan S Sibert, Joseph Y Kim, Daniel Parrell, Juan C Sanchez, Victoria Pappas, Anil Kumar, Kai Cai, Keith Thompson, Elizabeth R Wright</div> <em>Cold Spring Harbor Laboratory</em> (2022-01-02) <a href="https://doi.org/gp4qrv">https://doi.org/gp4qrv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2021.12.31.474669">10.1101/2021.12.31.474669</a></div></div>
</div>
<div id="ref-1Gwx5ScUa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">23. </div><div class="csl-right-inline"><strong>Beam image-shift accelerated data acquisition for near-atomic resolution single-particle cryo-electron tomography</strong> <div class="csl-block">Jonathan Bouvette, Hsuan-Fu Liu, Xiaochen Du, Ye Zhou, Andrew P Sikkema, Juliana da Fonseca Rezende e Mello, Bradley P Klemm, Rick Huang, Roel M Schaaper, Mario J Borgnia, Alberto Bartesaghi</div> <em>Nature Communications</em> (2021-03-30) <a href="https://doi.org/gjphkb">https://doi.org/gjphkb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41467-021-22251-8">10.1038/s41467-021-22251-8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33785757">33785757</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8009872">PMC8009872</a></div></div>
</div>
<div id="ref-xvouNMBy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">24. </div><div class="csl-right-inline"><strong>Parallel cryo electron tomography on &lt;i&gt;in situ&lt;/i&gt; lamellae</strong> <div class="csl-block">Fabian Eisenstein, Haruaki Yanagisawa, Hiroka Kashihara, Masahide Kikkawa, Sachiko Tsukita, Radostin Danev</div> <em>Cold Spring Harbor Laboratory</em> (2022-04-08) <a href="https://doi.org/gp4qrw">https://doi.org/gp4qrw</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2022.04.07.487557">10.1101/2022.04.07.487557</a></div></div>
</div>
<div id="ref-17H2LWiIu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">25. </div><div class="csl-right-inline"><strong>Waffle Method: A general and flexible approach for improving throughput in FIB-milling</strong> <div class="csl-block">Kotaro Kelley, Ashleigh M Raczkowski, Oleg Klykov, Pattana Jaroenlak, Daija Bobe, Mykhailo Kopylov, Edward T Eng, Gira Bhabha, Clinton S Potter, Bridget Carragher, Alex J Noble</div> <em>Nature Communications</em> (2022-04-06) <a href="https://doi.org/gp4qrt">https://doi.org/gp4qrt</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41467-022-29501-3">10.1038/s41467-022-29501-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35387991">35387991</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8987090">PMC8987090</a></div></div>
</div>
<div id="ref-44fG6kcy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">26. </div><div class="csl-right-inline"><strong>High-resolution cryo-EM using beam-image shift at 200 keV</strong> <div class="csl-block">Jennifer N Cash, Sarah Kearns, Yilai Li, Michael A Cianfrocco</div> <em>IUCrJ</em> (2020-10-29) <a href="https://doi.org/gjwcfq">https://doi.org/gjwcfq</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1107/s2052252520013482">10.1107/s2052252520013482</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33209328">33209328</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7642776">PMC7642776</a></div></div>
</div>
<div id="ref-kncnYGLD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">27. </div><div class="csl-right-inline"><strong>Shedding light on the cell biology of extracellular vesicles</strong> <div class="csl-block">Guillaume van Niel, Gisela D'Angelo, Graça Raposo</div> <em>Nature Reviews Molecular Cell Biology</em> (2018-01-17) <a href="https://doi.org/gct4df">https://doi.org/gct4df</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nrm.2017.125">10.1038/nrm.2017.125</a></div></div>
</div>
<div id="ref-agVdpX76" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">28. </div><div class="csl-right-inline"><strong>Exosome-mediated transfer of mRNAs and microRNAs is a novel mechanism of genetic exchange between cells</strong> <div class="csl-block">Hadi Valadi, Karin Ekström, Apostolos Bossios, Margareta Sjöstrand, James J Lee, Jan O Lötvall</div> <em>Nature Cell Biology</em> (2007-05-07) <a href="https://doi.org/d5df4s">https://doi.org/d5df4s</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/ncb1596">10.1038/ncb1596</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17486113">17486113</a></div></div>
</div>
<div id="ref-19ZFerhph" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">29. </div><div class="csl-right-inline"><strong>Automated electron microscope tomography using robust prediction of specimen movements</strong> <div class="csl-block">David N Mastronarde</div> <em>Journal of Structural Biology</em> (2005-10) <a href="https://doi.org/ff7gzx">https://doi.org/ff7gzx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jsb.2005.07.007">10.1016/j.jsb.2005.07.007</a></div></div>
</div>
<div id="ref-MUY42yxp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">30. </div><div class="csl-right-inline"><strong>cisTEM, user-friendly software for single-particle image processing</strong> <div class="csl-block">Timothy Grant, Alexis Rohou, Nikolaus Grigorieff</div> <em>eLife</em> (2018-03-07) <a href="https://doi.org/gf5cq7">https://doi.org/gf5cq7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.7554/elife.35383">10.7554/elife.35383</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29513216">29513216</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5854467">PMC5854467</a></div></div>
</div>
<div id="ref-1G4Y94qCy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">31. </div><div class="csl-right-inline"><strong>Measuring the optimal exposure for single particle cryo-EM using a 2.6 Å reconstruction of rotavirus VP6</strong> <div class="csl-block">Timothy Grant, Nikolaus Grigorieff</div> <em>eLife</em> (2015-05-29) <a href="https://doi.org/gf5cm7">https://doi.org/gf5cm7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.7554/elife.06980">10.7554/elife.06980</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26023829">26023829</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4471936">PMC4471936</a></div></div>
</div>
<div id="ref-n43f3Sqs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">32. </div><div class="csl-right-inline"><strong>CTFFIND4: Fast and accurate defocus estimation from electron micrographs</strong> <div class="csl-block">Alexis Rohou, Nikolaus Grigorieff</div> <em>Journal of Structural Biology</em> (2015-11) <a href="https://doi.org/f7xzgv">https://doi.org/f7xzgv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jsb.2015.08.008">10.1016/j.jsb.2015.08.008</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26278980">26278980</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6760662">PMC6760662</a></div></div>
</div>
<div id="ref-q2PSamkS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">33. </div><div class="csl-right-inline"><strong>Cryo-TEM simulations of amorphous radiation-sensitive samples using multislice wave propagation</strong> <div class="csl-block">Benjamin Himes, Nikolaus Grigorieff</div> <em>IUCrJ</em> (2021-09-30) <a href="https://doi.org/gpzs9r">https://doi.org/gpzs9r</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1107/s2052252521008538">10.1107/s2052252521008538</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34804546">34804546</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8562658">PMC8562658</a></div></div>
</div>
<div id="ref-stvWEJeu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">34. </div><div class="csl-right-inline"><strong>scikit-image: image processing in Python</strong> <div class="csl-block">Stéfan van der Walt, Johannes L Schönberger, Juan Nunez-Iglesias, François Boulogne, Joshua D Warner, Neil Yager, Emmanuelle Gouillart, Tony Yu</div> <em>PeerJ</em> (2014-06-19) <a href="https://doi.org/gftp3s">https://doi.org/gftp3s</a> <div class="csl-block">DOI: <a href="https://doi.org/10.7717/peerj.453">10.7717/peerj.453</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25024921">25024921</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4081273">PMC4081273</a></div></div>
</div>
<div id="ref-8Miti2Gz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">35. </div><div class="csl-right-inline"><strong>SciPy 1.0: fundamental algorithms for scientific computing in Python</strong> <div class="csl-block">Pauli Virtanen, Ralf Gommers, Travis E Oliphant, Matt Haberland, Tyler Reddy, David Cournapeau, Evgeni Burovski, Pearu Peterson, Warren Weckesser, Jonathan Bright, … </div> <em>Nature Methods</em> (2020-02-03) <a href="https://doi.org/ggj45f">https://doi.org/ggj45f</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41592-019-0686-2">10.1038/s41592-019-0686-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32015543">32015543</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7056644">PMC7056644</a></div></div>
</div>
<div id="ref-cYCuuZxG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">36. </div><div class="csl-right-inline"><strong>&lt;scp&gt;UCSF ChimeraX&lt;/scp&gt;
: Structure visualization for researchers, educators, and developers</strong> <div class="csl-block">Eric F Pettersen, Thomas D Goddard, Conrad C Huang, Elaine C Meng, Gregory S Couch, Tristan I Croll, John H Morris, Thomas E Ferrin</div> <em>Protein Science</em> (2020-10-22) <a href="https://doi.org/ghr6mn">https://doi.org/ghr6mn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pro.3943">10.1002/pro.3943</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32881101">32881101</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7737788">PMC7737788</a></div></div>
</div>
</div>
<!-- default theme -->

<style>
  /* import google fonts */
  @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
  @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

  /* -------------------------------------------------- */
  /* global */
  /* -------------------------------------------------- */

  /* all elements */
  * {
    /* force sans-serif font unless specified otherwise */
    font-family: "Open Sans", "Helvetica", sans-serif;

    /* prevent text inflation on some mobile browsers */
    -webkit-text-size-adjust: none !important;
    -moz-text-size-adjust: none !important;
    -o-text-size-adjust: none !important;
    text-size-adjust: none !important;
  }

  @media only screen {
    /* "page" element */
    body {
      position: relative;
      box-sizing: border-box;
      font-size: 12pt;
      line-height: 1.5;
      max-width: 8.5in;
      margin: 20px auto;
      padding: 40px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* "page" element */
    body {
      padding: 20px;
      margin: 0;
      border-radius: 0;
      border: none;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
      background: none;
    }
  }

  /* -------------------------------------------------- */
  /* headings */
  /* -------------------------------------------------- */

  /* all headings */
  h1,
  h2,
  h3,
  h4,
  h5,
  h6 {
    margin: 20px 0;
    padding: 0;
    font-weight: bold;
  }

  /* biggest heading */
  h1 {
    margin: 40px 0;
    text-align: center;
  }

  /* second biggest heading */
  h2 {
    margin-top: 30px;
    padding-bottom: 5px;
    border-bottom: solid 1px #bdbdbd;
  }

  /* heading font sizes */
  h1 {
    font-size: 2em;
  }
  h2 {
    font-size: 1.5em;
  }
  h3 {
    font-size: 1.35em;
  }
  h4 {
    font-size: 1.25em;
  }
  h5 {
    font-size: 1.15em;
  }
  h6 {
    font-size: 1em;
  }

  /* -------------------------------------------------- */
  /* manuscript header */
  /* -------------------------------------------------- */

  /* manuscript title */
  header > h1 {
    margin: 0;
  }

  /* manuscript title caption text (ie "automatically generated on") */
  header + p {
    text-align: center;
    margin-top: 10px;
  }

  /* -------------------------------------------------- */
  /* text elements */
  /* -------------------------------------------------- */

  /* links */
  a {
    color: #2196f3;
    overflow-wrap: break-word;
  }

  /* superscripts and subscripts */
  sub,
  sup {
    /* prevent from affecting line height */
    line-height: 0;
  }

  /* unordered and ordered lists*/
  ul,
  ol {
    padding-left: 20px;
  }

  /* class for styling text semibold */
  .semibold {
    font-weight: 600;
  }

  /* class for styling elements horizontally left aligned */
  .left {
    display: block;
    text-align: left;
    margin-left: auto;
    margin-right: 0;
    justify-content: left;
  }

  /* class for styling elements horizontally centered */
  .center {
    display: block;
    text-align: center;
    margin-left: auto;
    margin-right: auto;
    justify-content: center;
  }

  /* class for styling elements horizontally right aligned */
  .right {
    display: block;
    text-align: right;
    margin-left: 0;
    margin-right: auto;
    justify-content: right;
  }

  /* -------------------------------------------------- */
  /* section elements */
  /* -------------------------------------------------- */

  /* horizontal divider line */
  hr {
    border: none;
    height: 1px;
    background: #bdbdbd;
  }

  /* paragraphs, horizontal dividers, figures, tables, code */
  p,
  hr,
  figure,
  table,
  pre {
    /* treat all as "paragraphs", with consistent vertical margins */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* figures */
  /* -------------------------------------------------- */

  /* figure */
  figure {
    max-width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure caption */
  figcaption {
    padding: 0;
    padding-top: 10px;
  }

  /* figure image element */
  figure > img,
  figure > svg {
    max-width: 100%;
    display: block;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure auto-number */
  img + figcaption > span:first-of-type,
  svg + figcaption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* tables */
  /* -------------------------------------------------- */

  /* table */
  table {
    border-collapse: collapse;
    border-spacing: 0;
    width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* table cells */
  th,
  td {
    border: solid 1px #bdbdbd;
    padding: 10px;
    /* squash table if too wide for page by forcing line breaks */
    overflow-wrap: break-word;
    word-break: break-word;
  }

  /* header row and even rows */
  th,
  tr:nth-child(2n) {
    background-color: #fafafa;
  }

  /* odd rows */
  tr:nth-child(2n + 1) {
    background-color: #ffffff;
  }

  /* table caption */
  caption {
    text-align: left;
    padding: 0;
    padding-bottom: 10px;
  }

  /* table auto-number */
  table > caption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* code */
  /* -------------------------------------------------- */

  /* multi-line code block */
  pre {
    padding: 10px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
    break-inside: avoid;
    text-align: left;
  }

  /* inline code, ie code within normal text */
  :not(pre) > code {
    padding: 0 4px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
  }

  /* code text */
  /* apply all children, to reach syntax highlighting sub-elements */
  code,
  code * {
    /* force monospace font */
    font-family: "Source Code Pro", "Courier New", monospace;
  }

  /* -------------------------------------------------- */
  /* quotes */
  /* -------------------------------------------------- */

  /* quoted text */
  blockquote {
    margin: 0;
    padding: 0;
    border-left: 4px solid #bdbdbd;
    padding-left: 16px;
    break-inside: avoid;
  }

  /* -------------------------------------------------- */
  /* banners */
  /* -------------------------------------------------- */

  /* info banners */
  .banner {
    box-sizing: border-box;
    display: block;
    position: relative;
    width: 100%;
    margin-top: 20px;
    margin-bottom: 20px;
    padding: 20px;
    text-align: center;
  }

  /* paragraph in banner */
  .banner > p {
    margin: 0;
  }

  /* -------------------------------------------------- */
  /* highlight colors */
  /* -------------------------------------------------- */

  .white {
    background: #ffffff;
  }
  .lightgrey {
    background: #eeeeee;
  }
  .grey {
    background: #757575;
  }
  .darkgrey {
    background: #424242;
  }
  .black {
    background: #000000;
  }
  .lightred {
    background: #ffcdd2;
  }
  .lightyellow {
    background: #ffecb3;
  }
  .lightgreen {
    background: #dcedc8;
  }
  .lightblue {
    background: #e3f2fd;
  }
  .lightpurple {
    background: #f3e5f5;
  }
  .red {
    background: #f44336;
  }
  .orange {
    background: #ff9800;
  }
  .yellow {
    background: #ffeb3b;
  }
  .green {
    background: #4caf50;
  }
  .blue {
    background: #2196f3;
  }
  .purple {
    background: #9c27b0;
  }
  .white,
  .lightgrey,
  .lightred,
  .lightyellow,
  .lightgreen,
  .lightblue,
  .lightpurple,
  .orange,
  .yellow,
  .white a,
  .lightgrey a,
  .lightred a,
  .lightyellow a,
  .lightgreen a,
  .lightblue a,
  .lightpurple a,
  .orange a,
  .yellow a {
    color: #000000;
  }
  .grey,
  .darkgrey,
  .black,
  .red,
  .green,
  .blue,
  .purple,
  .grey a,
  .darkgrey a,
  .black a,
  .red a,
  .green a,
  .blue a,
  .purple a {
    color: #ffffff;
  }

  /* -------------------------------------------------- */
  /* buttons */
  /* -------------------------------------------------- */

  /* class for styling links like buttons */
  .button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    margin: 5px;
    padding: 10px 20px;
    font-size: 0.75em;
    font-weight: 600;
    text-transform: uppercase;
    text-decoration: none;
    letter-spacing: 1px;
    background: none;
    color: #2196f3;
    border: solid 1px #bdbdbd;
    border-radius: 5px;
  }

  /* buttons when hovered */
  .button:hover:not([disabled]),
  .icon_button:hover:not([disabled]) {
    cursor: pointer;
    background: #f5f5f5;
  }

  /* buttons when disabled */
  .button[disabled],
  .icon_button[disabled] {
    opacity: 0.35;
    pointer-events: none;
  }

  /* class for styling buttons containg only single icon */
  .icon_button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    text-decoration: none;
    margin: 0;
    padding: 0;
    background: none;
    border-radius: 5px;
    border: none;
    width: 20px;
    height: 20px;
    min-width: 20px;
    min-height: 20px;
  }

  /* icon button inner svg image */
  .icon_button > svg {
    height: 16px;
  }

  /* -------------------------------------------------- */
  /* icons */
  /* -------------------------------------------------- */

  /* class for styling icons inline with text */
  .inline_icon {
    height: 1em;
    position: relative;
    top: 0.125em;
  }

  /* -------------------------------------------------- */
  /* references */
  /* -------------------------------------------------- */

  .csl-entry {
    margin-top: 15px;
    margin-bottom: 15px;
  }

  /* -------------------------------------------------- */
  /* print control */
  /* -------------------------------------------------- */

  @media print {
    @page {
      /* suggested printing margin */
      margin: 0.5in;
    }

    /* document and "page" elements */
    html,
    body {
      margin: 0;
      padding: 0;
      width: 100%;
      height: 100%;
    }

    /* "page" element */
    body {
      font-size: 11pt !important;
      line-height: 1.35;
    }

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
      margin: 15px 0;
    }

    /* figures and tables */
    figure,
    table {
      font-size: 0.85em;
    }

    /* table cells */
    th,
    td {
      padding: 5px;
    }

    /* shrink font awesome icons */
    i.fas,
    i.fab,
    i.far,
    i.fal {
      transform: scale(0.85);
    }

    /* decrease banner margins */
    .banner {
      margin-top: 15px;
      margin-bottom: 15px;
      padding: 15px;
    }

    /* class for centering an element vertically on its own page */
    .page_center {
      margin: auto;
      width: 100%;
      height: 100%;
      display: flex;
      align-items: center;
      vertical-align: middle;
      break-before: page;
      break-after: page;
    }

    /* always insert a page break before the element */
    .page_break_before {
      break-before: page;
    }

    /* always insert a page break after the element */
    .page_break_after {
      break-after: page;
    }

    /* avoid page break before the element */
    .page_break_before_avoid {
      break-before: avoid;
    }

    /* avoid page break after the element */
    .page_break_after_avoid {
      break-after: avoid;
    }

    /* avoid page break inside the element */
    .page_break_inside_avoid {
      break-inside: avoid;
    }
  }

  /* -------------------------------------------------- */
  /* override pandoc css quirks */
  /* -------------------------------------------------- */

  .sourceCode {
    /* prevent unsightly overflow in wide code blocks */
    overflow: auto !important;
  }

  div.sourceCode {
    /* prevent background fill on top-most code block  container */
    background: none !important;
  }

  .sourceCode * {
    /* force consistent line spacing */
    line-height: 1.5 !important;
  }

  div.sourceCode {
    /* style code block margins same as <pre> element */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* tablenos */
  /* -------------------------------------------------- */

  /* tablenos wrapper */
  .tablenos {
    width: 100%;
    margin: 20px 0;
  }

  .tablenos > table {
    /* move margins from table to table_wrapper to allow margin collapsing */
    margin: 0;
  }

  @media only screen {
    /* tablenos wrapper */
    .tablenos {
      /* show scrollbar on tables if necessary to prevent overflow */
      overflow-x: auto !important;
    }

    .tablenos th,
    .tablenos td {
      overflow-wrap: unset !important;
      word-break: unset !important;
    }

    /* table in wrapper */
    .tablenos table,
    .tablenos table * {
      /* don't break table words */
      overflow-wrap: normal !important;
    }
  }
</style>
<!-- 
    Plugin Core

    Functions needed for and shared across all first-party plugins.
-->

<script>
  // get element that is target of hash (from link element or url)
  function getHashTarget(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector(`[id="${id}"]`);
    if (!target) return;

    // if figure or table, modify target to get expected element
    if (id.indexOf("fig:") === 0) target = target.querySelector("figure");
    if (id.indexOf("tbl:") === 0) target = target.querySelector("table");

    return target;
  }

  // get position/dimensions of element or viewport
  function getRectInView(element) {
    let rect = {};
    rect.left = 0;
    rect.top = 0;
    rect.right = document.documentElement.clientWidth;
    rect.bottom = document.documentElement.clientHeight;
    let style = {};

    if (element instanceof HTMLElement) {
      rect = element.getBoundingClientRect();
      style = window.getComputedStyle(element);
    }

    const margin = {};
    margin.left = parseFloat(style.marginLeftWidth) || 0;
    margin.top = parseFloat(style.marginTopWidth) || 0;
    margin.right = parseFloat(style.marginRightWidth) || 0;
    margin.bottom = parseFloat(style.marginBottomWidth) || 0;

    const border = {};
    border.left = parseFloat(style.borderLeftWidth) || 0;
    border.top = parseFloat(style.borderTopWidth) || 0;
    border.right = parseFloat(style.borderRightWidth) || 0;
    border.bottom = parseFloat(style.borderBottomWidth) || 0;

    const newRect = {};
    newRect.left = rect.left + margin.left + border.left;
    newRect.top = rect.top + margin.top + border.top;
    newRect.right = rect.right + margin.right + border.right;
    newRect.bottom = rect.bottom + margin.bottom + border.bottom;
    newRect.width = newRect.right - newRect.left;
    newRect.height = newRect.bottom - newRect.top;

    return newRect;
  }

  // get position of element relative to page
  function getRectInPage(element) {
    const rect = getRectInView(element);
    const body = getRectInView(document.body);

    const newRect = {};
    newRect.left = rect.left - body.left;
    newRect.top = rect.top - body.top;
    newRect.right = rect.right - body.left;
    newRect.bottom = rect.bottom - body.top;
    newRect.width = rect.width;
    newRect.height = rect.height;

    return newRect;
  }

  // get closest element before specified element that matches query
  function firstBefore(element, query) {
    while (element && element !== document.body && !element.matches(query))
      element = element.previousElementSibling || element.parentNode;

    return element;
  }

  // check if element is part of collapsed heading
  function isCollapsed(element) {
    while (element && element !== document.body) {
      if (element.dataset.collapsed === "true") return true;
      element = element.parentNode;
    }
    return false;
  }

  // expand any collapsed parent containers of element if necessary
  function expandElement(element) {
    if (isCollapsed(element)) {
      // accordion plugin
      const heading = firstBefore(element, "h2");
      if (heading) heading.click();
      // details/summary HTML element
      const summary = firstBefore(element, "summary");
      if (summary) summary.click();
    }
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);

    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // get list of elements after a start element up to element matching query
  function nextUntil(element, query, exclude) {
    const elements = [];
    while (((element = element.nextElementSibling), element)) {
      if (element.matches(query)) break;
      if (!element.matches(exclude)) elements.push(element);
    }
    return elements;
  }
</script>
<!--
  Accordion Plugin

  Allows sections of content under h2 headings to be collapsible.
-->

<script type="module">
  // whether to always start expanded ('false'), always start collapsed
  // ('true'), or start collapsed when screen small ('auto')
  const startCollapsed = "auto";

  // start script
  function start() {
    // run through each <h2> heading
    const headings = document.querySelectorAll("h2");
    for (const heading of headings) {
      addArrow(heading);

      // start expanded/collapsed based on option
      if (
        startCollapsed === "true" ||
        (startCollapsed === "auto" && isSmallScreen()) ||
        heading.dataset.collapsed === "true"
      )
        collapseHeading(heading);
      else expandElement(heading);
    }

    // attach hash change listener to window
    window.addEventListener("hashchange", onHashChange);
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) goToElement(target);
  }

  // add arrow to heading
  function addArrow(heading) {
    // add arrow button
    const arrow = document.createElement("button");
    arrow.innerHTML = document.querySelector(".icon_angle_down").innerHTML;
    arrow.classList.add("icon_button", "accordion_arrow");
    heading.insertBefore(arrow, heading.firstChild);

    // attach click listener to heading and button
    heading.addEventListener("click", onHeadingClick);
    arrow.addEventListener("click", onArrowClick);
  }

  // determine if on mobile-like device with small screen
  function isSmallScreen() {
    return Math.min(window.innerWidth, window.innerHeight) < 480;
  }

  // when <h2> heading is clicked
  function onHeadingClick(event) {
    // only collapse if <h2> itself is target of click (eg, user did
    // not click on anchor within <h2>)
    if (event.target === this) toggleCollapse(this);
  }

  // when arrow button is clicked
  function onArrowClick() {
    toggleCollapse(this.parentNode);
  }

  // collapse section if expanded, expand if collapsed
  function toggleCollapse(heading) {
    if (heading.dataset.collapsed === "false") collapseHeading(heading);
    else expandElement(heading);
  }

  // elements to exclude from collapse, such as table of contents panel,
  // hypothesis panel, etc
  const exclude = "#toc_panel, div.annotator-frame, #lightbox_overlay";

  // collapse section
  function collapseHeading(heading) {
    heading.setAttribute("data-collapsed", "true");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "true");
  }

  // expand section
  function expandElement(heading) {
    heading.setAttribute("data-collapsed", "false");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "false");
  }

  // get list of elements between this <h2> and next <h2> or <h1>
  // ("children" of the <h2> section)
  function getChildren(heading) {
    return nextUntil(heading, "h2, h1", exclude);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
  <!-- modified from: https://fontawesome.com/icons/angle-down -->
  <svg width="16" height="16" viewBox="0 0 448 512">
    <path
      fill="currentColor"
      d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* accordion arrow button */
    .accordion_arrow {
      margin-right: 10px;
    }

    /* arrow icon when <h2> data-collapsed attribute true */
    h2[data-collapsed="true"] > .accordion_arrow > svg {
      transform: rotate(-90deg);
    }

    /* all elements (except <h2>'s) when data-collapsed attribute true */
    *:not(h2)[data-collapsed="true"] {
      display: none;
    }

    /* accordion arrow button when hovered and <h2>'s when hovered */
    .accordion_arrow:hover,
    h2[data-collapsed="true"]:hover,
    h2[data-collapsed="false"]:hover {
      cursor: pointer;
    }
  }

  /* always hide accordion arrow button on print */
  @media only print {
    .accordion_arrow {
      display: none;
    }
  }
</style>
<!--
  Anchors Plugin

  Adds an anchor next to each of a certain type of element that provides a
  human-readable url to that specific item/position in the document (e.g.
  "manuscript.html#abstract"). It also makes it such that scrolling out of view
  of a target removes its identifier from the url.
-->

<script type="module">
  // which types of elements to add anchors next to, in "document.querySelector"
  // format
  const typesQuery =
    'h1, h2, h3, div[id^="fig:"], div[id^="tbl:"], span[id^="eq:"]';

  // start script
  function start() {
    // add anchor to each element of specified types
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) addAnchor(element);

    // attach scroll listener to window
    window.addEventListener("scroll", onScroll);
  }

  // when window is scrolled
  function onScroll() {
    // if url has hash and user has scrolled out of view of hash
    // target, remove hash from url
    const tolerance = 100;
    const target = getHashTarget();
    if (target) {
      if (
        target.getBoundingClientRect().top > window.innerHeight + tolerance ||
        target.getBoundingClientRect().bottom < 0 - tolerance
      )
        history.pushState(null, null, " ");
    }
  }

  // add anchor to element
  function addAnchor(element) {
    let addTo; // element to add anchor button to

    // if figure or table, modify withId and addTo to get expected
    // elements
    if (element.id.indexOf("fig:") === 0) {
      addTo = element.querySelector("figcaption");
    } else if (element.id.indexOf("tbl:") === 0) {
      addTo = element.querySelector("caption");
    } else if (element.id.indexOf("eq:") === 0) {
      addTo = element.querySelector(".eqnos-number");
    }

    addTo = addTo || element;
    const id = element.id || null;

    // do not add anchor if element doesn't have assigned id.
    // id is generated by pandoc and is assumed to be unique and
    // human-readable
    if (!id) return;

    // create anchor button
    const anchor = document.createElement("a");
    anchor.innerHTML = document.querySelector(".icon_link").innerHTML;
    anchor.title = "Link to this part of the document";
    anchor.classList.add("icon_button", "anchor");
    anchor.dataset.ignore = "true";
    anchor.href = "#" + id;
    addTo.appendChild(anchor);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- link icon -->

<template class="icon_link">
  <!-- modified from: https://fontawesome.com/icons/link -->
  <svg width="16" height="16" viewBox="0 0 512 512">
    <path
      fill="currentColor"
      d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* anchor button */
    .anchor {
      opacity: 0;
      margin-left: 5px;
    }

    /* anchor buttons within <h2>'s */
    h2 .anchor {
      margin-left: 10px;
    }

    /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
    *:hover > .anchor,
    .anchor:hover,
    .anchor:focus {
      opacity: 1;
    }

    /* anchor button when hovered */
    .anchor:hover {
      cursor: pointer;
    }
  }

  /* always show anchor button on devices with no mouse/hover ability */
  @media (hover: none) {
    .anchor {
      opacity: 1;
    }
  }

  /* always hide anchor button on print */
  @media only print {
    .anchor {
      display: none;
    }
  }
</style>
<!-- 
    Attributes Plugin

    Allows arbitrary HTML attributes to be attached to (almost) any element.
    Place an HTML comment inside or next to the desired element with the content:
    $attribute="value"
-->

<script type="module">
  // start script
  function start() {
    // get list of comments in document
    const comments = findComments();

    for (const comment of comments)
      if (comment.parentElement)
        addAttributes(comment.parentElement, comment.nodeValue.trim());
  }

  // add html attributes to specified element based on string of
  // html attributes and values
  function addAttributes(element, text) {
    // regex's for finding attribute/value pairs in the format of
    // attribute="value" or attribute='value
    const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
    const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

    // loop through attribute/value pairs
    let match;
    while ((match = text.match(regex2) || text.match(regex1))) {
      // get attribute and value from regex capture groups
      let attribute = match[1];
      let value = match[2];

      // remove from string
      text = text.substring(match.index + match[0].length);

      if (!attribute || !value) break;

      // set attribute of parent element
      try {
        element.setAttribute(attribute, value);
      } catch (error) {
        console.log(error);
      }

      // special case for colspan
      if (attribute === "colspan") removeTableCells(element, value);
    }
  }

  // get list of comment elements in document
  function findComments() {
    const comments = [];

    // iterate over comment nodes in document
    function acceptNode(node) {
      return NodeFilter.FILTER_ACCEPT;
    }
    const iterator = document.createNodeIterator(
      document.body,
      NodeFilter.SHOW_COMMENT,
      acceptNode
    );
    let node;
    while ((node = iterator.nextNode())) comments.push(node);

    return comments;
  }

  // remove certain number of cells after specified cell
  function removeTableCells(cell, number) {
    number = parseInt(number);
    if (!number) return;

    // remove elements
    for (; number > 1; number--) {
      if (cell.nextElementSibling) cell.nextElementSibling.remove();
    }
  }

  // start script on DOMContentLoaded instead of load to ensure this plugins
  // runs before other plugins
  window.addEventListener("DOMContentLoaded", start);
</script>
<!--
  Jump to First Plugin

  Adds a button next to each reference entry, figure, and table that jumps the
  page to the first occurrence of a link to that item in the manuscript.
-->

<script type="module">
  // whether to add buttons next to reference entries
  const references = "true";
  // whether to add buttons next to figures
  const figures = "true";
  // whether to add buttons next to tables
  const tables = "true";

  // start script
  function start() {
    if (references !== "false")
      makeButtons(`div[id^="ref-"]`, ".csl-left-margin", "reference");
    if (figures !== "false")
      makeButtons(`div[id^="fig:"]`, "figcaption", "figure");
    if (tables !== "false") makeButtons(`div[id^="tbl:"]`, "caption", "table");
  }

  // when jump button clicked
  function onButtonClick() {
    const first = getFirstOccurrence(this.dataset.id);
    if (!first) return;

    // update url hash so navigating "back" in history will return user to button
    window.location.hash = this.dataset.id;
    // scroll to link
    const timeout = function () {
      goToElement(first, window.innerHeight * 0.5);
    };
    window.setTimeout(timeout, 0);
  }

  // get first occurrence of link to item in document
  function getFirstOccurrence(id) {
    let query = "a";
    query += '[href="#' + id + '"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelector(query);
  }

  // add button next to each reference entry, figure, or table
  function makeButtons(query, containerQuery, subject) {
    const elements = document.querySelectorAll(query);
    for (const element of elements) {
      const id = element.id;
      const buttonContainer = element.querySelector(containerQuery);
      const first = getFirstOccurrence(id);

      // if can't find link to reference or place to put button, ignore
      if (!first || !buttonContainer) continue;

      // make jump button
      let button = document.createElement("button");
      button.classList.add("icon_button", "jump_arrow");
      button.title = `Jump to the first occurrence of this ${subject} in the document`;
      const icon = document.querySelector(".icon_angle_double_up");
      button.innerHTML = icon.innerHTML;
      button.dataset.id = id;
      button.dataset.ignore = "true";
      button.addEventListener("click", onButtonClick);
      buttonContainer.prepend(button);
    }
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
  <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
  <svg width="16" height="16" viewBox="0 0 320 512">
    <path
      fill="currentColor"
      d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* jump button */
    .jump_arrow {
      position: relative;
      top: 0.125em;
      margin-right: 5px;
    }
  }

  /* always hide jump button on print */
  @media only print {
    .jump_arrow {
      display: none;
    }
  }
</style>
<!-- 
    Lightbox Plugin

    Makes it such that when a user clicks on an image, the image fills the
    screen and the user can pan/drag/zoom the image and navigate between other
    images in the document.
-->

<script type="module">
  // list of possible zoom/scale factors
  const zooms =
    "0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1, 1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8";
  // whether to fit image to view ('fit'), display at 100% and shrink if
  // necessary ('shrink'), or always display at 100% ('100')
  const defaultZoom = "fit";
  // whether to zoom in/out toward center of view ('true') or mouse ('false')
  const centerZoom = "false";

  // start script
  function start() {
    // run through each <img> element
    const imgs = document.querySelectorAll("figure > img");
    let count = 1;
    for (const img of imgs) {
      img.classList.add("lightbox_document_img");
      img.dataset.number = count;
      img.dataset.total = imgs.length;
      img.addEventListener("click", openLightbox);
      count++;
    }

    // attach mouse and key listeners to window
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("keyup", onKeyUp);
  }

  // when mouse is moved anywhere in window
  function onWindowMouseMove(event) {
    window.mouseX = event.clientX;
    window.mouseY = event.clientY;
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("lightbox_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("lightbox_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeLightbox();
        break;
    }
  }

  // open lightbox
  function openLightbox() {
    const lightbox = makeLightbox(this);
    if (!lightbox) return;

    blurBody(lightbox);
    document.body.appendChild(lightbox);
  }

  // make lightbox
  function makeLightbox(img) {
    // delete lightbox if it exists, start fresh
    closeLightbox();

    // create screen overlay containing lightbox
    const overlay = document.createElement("div");
    overlay.id = "lightbox_overlay";

    // create image info boxes
    const numberInfo = document.createElement("div");
    const zoomInfo = document.createElement("div");
    numberInfo.id = "lightbox_number_info";
    zoomInfo.id = "lightbox_zoom_info";

    // create container for image
    const imageContainer = document.createElement("div");
    imageContainer.id = "lightbox_image_container";
    const lightboxImg = makeLightboxImg(
      img,
      imageContainer,
      numberInfo,
      zoomInfo
    );
    imageContainer.appendChild(lightboxImg);

    // create bottom container for caption and navigation buttons
    const bottomContainer = document.createElement("div");
    bottomContainer.id = "lightbox_bottom_container";
    const caption = makeCaption(img);
    const prevButton = makePrevButton(img);
    const nextButton = makeNextButton(img);
    bottomContainer.appendChild(prevButton);
    bottomContainer.appendChild(caption);
    bottomContainer.appendChild(nextButton);

    // attach top middle and bottom to overlay
    overlay.appendChild(numberInfo);
    overlay.appendChild(zoomInfo);
    overlay.appendChild(imageContainer);
    overlay.appendChild(bottomContainer);

    return overlay;
  }

  // make <img> object that is intuitively draggable and zoomable
  function makeLightboxImg(sourceImg, container, numberInfoBox, zoomInfoBox) {
    // create copy of source <img>
    const img = sourceImg.cloneNode(true);
    img.classList.remove("lightbox_document_img");
    img.removeAttribute("id");
    img.removeAttribute("width");
    img.removeAttribute("height");
    img.style.position = "unset";
    img.style.margin = "0";
    img.style.padding = "0";
    img.style.width = "";
    img.style.height = "";
    img.style.minWidth = "";
    img.style.minHeight = "";
    img.style.maxWidth = "";
    img.style.maxHeight = "";
    img.id = "lightbox_img";

    // build sorted list of zoomSteps
    const zoomSteps = zooms.split(/[^0-9.]/).map((step) => parseFloat(step));
    zoomSteps.sort((a, b) => a - b);

    // <img> object property variables
    let zoom = 1;
    let translateX = 0;
    let translateY = 0;
    let clickMouseX = undefined;
    let clickMouseY = undefined;
    let clickTranslateX = undefined;
    let clickTranslateY = undefined;

    updateNumberInfo();

    // update image numbers displayed in info box
    function updateNumberInfo() {
      numberInfoBox.innerHTML =
        sourceImg.dataset.number + " of " + sourceImg.dataset.total;
    }

    // update zoom displayed in info box
    function updateZoomInfo() {
      let zoomInfo = zoom * 100;
      if (!Number.isInteger(zoomInfo)) zoomInfo = zoomInfo.toFixed(2);
      zoomInfoBox.innerHTML = zoomInfo + "%";
    }

    // move to closest zoom step above current zoom
    const zoomIn = function () {
      for (const zoomStep of zoomSteps) {
        if (zoomStep > zoom) {
          zoom = zoomStep;
          break;
        }
      }
      updateTransform();
    };

    // move to closest zoom step above current zoom
    const zoomOut = function () {
      zoomSteps.reverse();
      for (const zoomStep of zoomSteps) {
        if (zoomStep < zoom) {
          zoom = zoomStep;
          break;
        }
      }
      zoomSteps.reverse();

      updateTransform();
    };

    // update display of <img> based on scale/translate properties
    const updateTransform = function () {
      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      // get new width/height after scale
      const rect = img.getBoundingClientRect();
      // limit translate
      translateX = Math.max(translateX, -rect.width / 2);
      translateX = Math.min(translateX, rect.width / 2);
      translateY = Math.max(translateY, -rect.height / 2);
      translateY = Math.min(translateY, rect.height / 2);

      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      updateZoomInfo();
    };

    // fit <img> to container
    const fit = function () {
      // no x/y offset, 100% zoom by default
      translateX = 0;
      translateY = 0;
      zoom = 1;

      // widths of <img> and container
      const imgWidth = img.naturalWidth;
      const imgHeight = img.naturalHeight;
      const containerWidth = parseFloat(
        window.getComputedStyle(container).width
      );
      const containerHeight = parseFloat(
        window.getComputedStyle(container).height
      );

      // how much zooming is needed to fit <img> to container
      const xRatio = imgWidth / containerWidth;
      const yRatio = imgHeight / containerHeight;
      const maxRatio = Math.max(xRatio, yRatio);
      const newZoom = 1 / maxRatio;

      // fit <img> to container according to option
      if (defaultZoom === "shrink") {
        if (maxRatio > 1) zoom = newZoom;
      } else if (defaultZoom === "fit") zoom = newZoom;

      updateTransform();
    };

    // when mouse wheel is rolled anywhere in container
    const onContainerWheel = function (event) {
      if (!event) return;

      // let ctrl + mouse wheel to zoom behave as normal
      if (event.ctrlKey) return;

      // prevent normal scroll behavior
      event.preventDefault();
      event.stopPropagation();

      // point around which to scale img
      const viewRect = container.getBoundingClientRect();
      const viewX = (viewRect.left + viewRect.right) / 2;
      const viewY = (viewRect.top + viewRect.bottom) / 2;
      const originX = centerZoom === "true" ? viewX : mouseX;
      const originY = centerZoom === "true" ? viewY : mouseY;

      // get point on image under origin
      const oldRect = img.getBoundingClientRect();
      const oldPercentX = (originX - oldRect.left) / oldRect.width;
      const oldPercentY = (originY - oldRect.top) / oldRect.height;

      // increment/decrement zoom
      if (event.deltaY < 0) zoomIn();
      if (event.deltaY > 0) zoomOut();

      // get offset between previous image point and origin
      const newRect = img.getBoundingClientRect();
      const offsetX = originX - (newRect.left + newRect.width * oldPercentX);
      const offsetY = originY - (newRect.top + newRect.height * oldPercentY);

      // translate image to keep image point under origin
      translateX += offsetX;
      translateY += offsetY;

      // perform translate
      updateTransform();
    };

    // when container is clicked
    function onContainerClick(event) {
      // if container itself is target of click, and not other
      // element above it
      if (event.target === this) closeLightbox();
    }

    // when mouse button is pressed on image
    const onImageMouseDown = function (event) {
      // store original mouse position relative to image
      clickMouseX = window.mouseX;
      clickMouseY = window.mouseY;
      clickTranslateX = translateX;
      clickTranslateY = translateY;
      event.stopPropagation();
      event.preventDefault();
    };

    // when mouse button is released anywhere in window
    const onWindowMouseUp = function (event) {
      // reset original mouse position
      clickMouseX = undefined;
      clickMouseY = undefined;
      clickTranslateX = undefined;
      clickTranslateY = undefined;

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mouseup", onWindowMouseUp);
    };

    // when mouse is moved anywhere in window
    const onWindowMouseMove = function (event) {
      if (
        clickMouseX === undefined ||
        clickMouseY === undefined ||
        clickTranslateX === undefined ||
        clickTranslateY === undefined
      )
        return;

      // offset image based on original and current mouse position
      translateX = clickTranslateX + window.mouseX - clickMouseX;
      translateY = clickTranslateY + window.mouseY - clickMouseY;
      updateTransform();
      event.preventDefault();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mousemove", onWindowMouseMove);
    };

    // when window is resized
    const onWindowResize = function (event) {
      fit();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("resize", onWindowResize);
    };

    // attach the necessary event listeners
    img.addEventListener("dblclick", fit);
    img.addEventListener("mousedown", onImageMouseDown);
    container.addEventListener("wheel", onContainerWheel);
    container.addEventListener("mousedown", onContainerClick);
    container.addEventListener("touchstart", onContainerClick);
    window.addEventListener("mouseup", onWindowMouseUp);
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("resize", onWindowResize);

    // run fit() after lightbox atttached to document and <img> Loaded
    // so needed container and img dimensions available
    img.addEventListener("load", fit);

    return img;
  }

  // make caption
  function makeCaption(img) {
    const caption = document.createElement("div");
    caption.id = "lightbox_caption";
    const captionSource = img.nextElementSibling;
    if (captionSource.tagName.toLowerCase() === "figcaption") {
      const captionCopy = makeCopy(captionSource);
      caption.innerHTML = captionCopy.innerHTML;
    }

    caption.addEventListener("touchstart", function (event) {
      event.stopPropagation();
    });

    return caption;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // make button to jump to previous image in document
  function makePrevButton(img) {
    const prevButton = document.createElement("button");
    prevButton.id = "lightbox_prev_button";
    prevButton.title = "Jump to the previous image in the document [←]";
    prevButton.classList.add("icon_button", "lightbox_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;

    // attach click listeners to button
    prevButton.addEventListener("click", function () {
      getPrevImg(img).click();
    });

    return prevButton;
  }

  // make button to jump to next image in document
  function makeNextButton(img) {
    const nextButton = document.createElement("button");
    nextButton.id = "lightbox_next_button";
    nextButton.title = "Jump to the next image in the document [→]";
    nextButton.classList.add("icon_button", "lightbox_button");
    nextButton.innerHTML = document.querySelector(
      ".icon_caret_right"
    ).innerHTML;

    // attach click listeners to button
    nextButton.addEventListener("click", function () {
      getNextImg(img).click();
    });

    return nextButton;
  }

  // get previous image in document
  function getPrevImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if < 1
    if (index - 1 >= 0) index--;
    else index = imgs.length - 1;
    return imgs[index];
  }

  // get next image in document
  function getNextImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if > total
    if (index + 1 <= imgs.length - 1) index++;
    else index = 0;
    return imgs[index];
  }

  // close lightbox
  function closeLightbox() {
    focusBody();

    const lightbox = document.getElementById("lightbox_overlay");
    if (lightbox) lightbox.remove();
  }

  // make all elements behind lightbox non-focusable
  function blurBody(overlay) {
    const all = document.querySelectorAll("*");
    for (const element of all) element.tabIndex = -1;
    document.body.classList.add("body_no_scroll");
  }

  // make all elements focusable again
  function focusBody() {
    const all = document.querySelectorAll("*");
    for (const element of all) element.removeAttribute("tabIndex");
    document.body.classList.remove("body_no_scroll");
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* regular <img> in document when hovered */
    img.lightbox_document_img:hover {
      cursor: pointer;
    }

    .body_no_scroll {
      overflow: hidden !important;
    }

    /* screen overlay */
    #lightbox_overlay {
      display: flex;
      flex-direction: column;
      position: fixed;
      left: 0;
      top: 0;
      right: 0;
      bottom: 0;
      background: rgba(0, 0, 0, 0.75);
      z-index: 3;
    }

    /* middle area containing lightbox image */
    #lightbox_image_container {
      flex-grow: 1;
      display: flex;
      justify-content: center;
      align-items: center;
      overflow: hidden;
      position: relative;
      padding: 20px;
    }

    /* bottom area containing caption */
    #lightbox_bottom_container {
      display: flex;
      justify-content: center;
      align-items: center;
      height: 100px;
      min-height: 100px;
      max-height: 100px;
      background: rgba(0, 0, 0, 0.5);
    }

    /* image number info text box */
    #lightbox_number_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      left: 2px;
      top: 0;
      z-index: 4;
    }

    /* zoom info text box */
    #lightbox_zoom_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      right: 2px;
      top: 0;
      z-index: 4;
    }

    /* copy of image caption */
    #lightbox_caption {
      box-sizing: border-box;
      display: inline-block;
      width: 100%;
      max-height: 100%;
      padding: 10px 0;
      text-align: center;
      overflow-y: auto;
      color: #ffffff;
    }

    /* navigation previous/next button */
    .lightbox_button {
      width: 100px;
      height: 100%;
      min-width: 100px;
      min-height: 100%;
      color: #ffffff;
    }

    /* navigation previous/next button when hovered */
    .lightbox_button:hover {
      background: none !important;
    }

    /* navigation button icon */
    .lightbox_button > svg {
      height: 25px;
    }

    /* figure auto-number */
    #lightbox_caption > span:first-of-type {
      font-weight: bold;
      margin-right: 5px;
    }

    /* lightbox image when hovered */
    #lightbox_img:hover {
      cursor: grab;
    }

    /* lightbox image when grabbed */
    #lightbox_img:active {
      cursor: grabbing;
    }
  }

  /* when on screen < 480px wide */
  @media only screen and (max-width: 480px) {
    /* make navigation buttons skinnier on small screens to make more room for caption text */
    .lightbox_button {
      width: 50px;
      min-width: 50px;
    }
  }

  /* always hide lightbox on print */
  @media only print {
    #lightbox_overlay {
      display: none;
    }
  }
</style>
<!-- 
  Link Highlight Plugin

  Makes it such that when a user hovers or focuses a link, other links that have
  the same target will be highlighted. It also makes it such that when clicking
  a link, the target of the link (eg reference, figure, table) is briefly
  highlighted.
-->

<script type="module">
  // whether to also highlight links that go to external urls
  const externalLinks = "false";
  // whether user must click off to unhighlight instead of just
  // un-hovering
  const clickUnhighlight = "false";
  // whether to also highlight links that are unique
  const highlightUnique = "true";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach mouse and focus listeners to link
      link.addEventListener("mouseenter", onLinkFocus);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("mouseleave", onLinkUnhover);
    }

    // attach click and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("hashchange", onHashChange);

    // run hash change on window load in case user has navigated
    // directly to hash
    onHashChange();
  }

  // when link is focused (tabbed to) or hovered
  function onLinkFocus() {
    highlight(this);
  }

  // when link is unhovered
  function onLinkUnhover() {
    if (clickUnhighlight !== "true") unhighlightAll();
  }

  // when the mouse is clicked anywhere in window
  function onClick(event) {
    unhighlightAll();
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) glowElement(target);
  }

  // start glow sequence on an element
  function glowElement(element) {
    const startGlow = function () {
      onGlowEnd();
      element.dataset.glow = "true";
      element.addEventListener("animationend", onGlowEnd);
    };
    const onGlowEnd = function () {
      element.removeAttribute("data-glow");
      element.removeEventListener("animationend", onGlowEnd);
    };
    startGlow();
  }

  // highlight link and all others with same target
  function highlight(link) {
    // force unhighlight all to start fresh
    unhighlightAll();

    // get links with same target
    if (!link) return;
    const sameLinks = getSameLinks(link);

    // if link unique and option is off, exit and don't highlight
    if (sameLinks.length <= 1 && highlightUnique !== "true") return;

    // highlight all same links, and "select" (special highlight) this
    // one
    for (const sameLink of sameLinks) {
      if (sameLink === link) sameLink.setAttribute("data-selected", "true");
      else sameLink.setAttribute("data-highlighted", "true");
    }
  }

  // unhighlight all links
  function unhighlightAll() {
    const links = getLinks();
    for (const link of links) {
      link.setAttribute("data-selected", "false");
      link.setAttribute("data-highlighted", "false");
    }
  }

  // get links with same target
  function getSameLinks(link) {
    const results = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        results.push(otherLink);
    }
    return results;
  }

  // get all links of types we wish to handle
  function getLinks() {
    let query = "a";
    if (externalLinks !== "true") query += '[href^="#"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelectorAll(query);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<style>
  @media only screen {
    /* anything with data-highlighted attribute true */
    [data-highlighted="true"] {
      background: #ffeb3b;
    }

    /* anything with data-selected attribute true */
    [data-selected="true"] {
      background: #ff8a65 !important;
    }

    /* animation definition for glow */
    @keyframes highlight_glow {
      0% {
        background: none;
      }
      10% {
        background: #bbdefb;
      }
      100% {
        background: none;
      }
    }

    /* anything with data-glow attribute true */
    [data-glow="true"] {
      animation: highlight_glow 2s;
    }
  }
</style>
<!--
  Table of Contents Plugin

  Provides a "table of contents" (toc) panel on the side of the document that
  allows the user to conveniently navigate between sections of the document.
-->

<script type="module">
  // which types of elements to add links for, in "document.querySelector" format
  const typesQuery = "h1, h2, h3";
  // whether toc starts open. use 'true' or 'false', or 'auto' to
  // use 'true' behavior when screen wide enough and 'false' when not
  const startOpen = "false";
  // whether toc closes when clicking on toc link. use 'true' or
  // 'false', or 'auto' to use 'false' behavior when screen wide
  // enough and 'true' when not
  const clickClose = "auto";
  // if list item is more than this many characters, text will be
  // truncated
  const charLimit = "50";
  // whether or not to show bullets next to each toc item
  const bullets = "false";

  // start script
  function start() {
    // make toc panel and populate with entries (links to document
    // sections)
    const panel = makePanel();
    if (!panel) return;
    makeEntries(panel);
    // attach panel to document after making entries, so 'toc' heading
    // in panel isn't included in toc
    document.body.insertBefore(panel, document.body.firstChild);

    // initial panel state
    if (startOpen === "true" || (startOpen === "auto" && !isSmallScreen()))
      openPanel();
    else closePanel();

    // attach click, scroll, and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("scroll", onScroll);
    window.addEventListener("hashchange", onScroll);
    window.addEventListener("keyup", onKeyUp);
    onScroll();

    // add class to push document body down out of way of toc button
    document.body.classList.add("toc_body_nudge");
  }

  // determine if screen wide enough to fit toc panel
  function isSmallScreen() {
    // in default theme:
    // 816px = 8.5in = width of "page" (<body>) element
    // 260px = min width of toc panel (*2 for both sides of <body>)
    return window.innerWidth < 816 + 260 * 2;
  }

  // when mouse is clicked anywhere in window
  function onClick() {
    if (isSmallScreen()) closePanel();
  }

  // when window is scrolled or hash changed
  function onScroll() {
    highlightViewed();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    // close on esc
    if (event.key === "Escape") closePanel();
  }

  // find entry of currently viewed document section in toc and highlight
  function highlightViewed() {
    const firstId = getFirstInView(typesQuery);

    // get toc entries (links), unhighlight all, then highlight viewed
    const list = document.getElementById("toc_list");
    if (!firstId || !list) return;
    const links = list.querySelectorAll("a");
    for (const link of links) link.dataset.viewing = "false";
    const link = list.querySelector('a[href="#' + firstId + '"]');
    if (!link) return;
    link.dataset.viewing = "true";
  }

  // get first or previous toc listed element in top half of view
  function getFirstInView(query) {
    // get all elements matching query and with id
    const elements = document.querySelectorAll(query);
    const elementsWithIds = [];
    for (const element of elements) {
      if (element.id) elementsWithIds.push(element);
    }

    // get first or previous element in top half of view
    for (let i = 0; i < elementsWithIds.length; i++) {
      const element = elementsWithIds[i];
      const prevElement = elementsWithIds[Math.max(0, i - 1)];
      if (element.getBoundingClientRect().top >= 0) {
        if (element.getBoundingClientRect().top < window.innerHeight / 2)
          return element.id;
        else return prevElement.id;
      }
    }
  }

  // make panel
  function makePanel() {
    // create panel
    const panel = document.createElement("div");
    panel.id = "toc_panel";
    if (bullets === "true") panel.dataset.bullets = "true";

    // create header
    const header = document.createElement("div");
    header.id = "toc_header";

    // create toc button
    const button = document.createElement("button");
    button.id = "toc_button";
    button.innerHTML = document.querySelector(".icon_th_list").innerHTML;
    button.title = "Table of Contents";
    button.classList.add("icon_button");

    // create header text
    const text = document.createElement("h4");
    text.innerHTML = "Table of Contents";

    // create container for toc list
    const list = document.createElement("div");
    list.id = "toc_list";

    // attach click listeners
    panel.addEventListener("click", onPanelClick);
    header.addEventListener("click", onHeaderClick);
    button.addEventListener("click", onButtonClick);

    // attach elements
    header.appendChild(button);
    header.appendChild(text);
    panel.appendChild(header);
    panel.appendChild(list);

    return panel;
  }

  // create toc entries (links) to each element of the specified types
  function makeEntries(panel) {
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) {
      // do not add link if element doesn't have assigned id
      if (!element.id) continue;

      // create link/list item
      const link = document.createElement("a");
      link.classList.add("toc_link");
      switch (element.tagName.toLowerCase()) {
        case "h1":
          link.dataset.level = "1";
          break;
        case "h2":
          link.dataset.level = "2";
          break;
        case "h3":
          link.dataset.level = "3";
          break;
        case "h4":
          link.dataset.level = "4";
          break;
      }
      link.title = element.innerText;
      let text = element.innerText;
      if (text.length > charLimit) text = text.slice(0, charLimit) + "...";
      link.innerHTML = text;
      link.href = "#" + element.id;
      link.addEventListener("click", onLinkClick);

      // attach link
      panel.querySelector("#toc_list").appendChild(link);
    }
  }

  // when panel is clicked
  function onPanelClick(event) {
    // stop click from propagating to window/document and closing panel
    event.stopPropagation();
  }

  // when header itself is clicked
  function onHeaderClick(event) {
    togglePanel();
  }

  // when button is clicked
  function onButtonClick(event) {
    togglePanel();
    // stop header underneath button from also being clicked
    event.stopPropagation();
  }

  // when link is clicked
  function onLinkClick(event) {
    if (clickClose === "true" || (clickClose === "auto" && isSmallScreen()))
      closePanel();
    else openPanel();
  }

  // open panel if closed, close if opened
  function togglePanel() {
    const panel = document.getElementById("toc_panel");
    if (!panel) return;

    if (panel.dataset.open === "true") closePanel();
    else openPanel();
  }

  // open panel
  function openPanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "true";
  }

  // close panel
  function closePanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "false";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- th list icon -->

<template class="icon_th_list">
  <!-- modified from: https://fontawesome.com/icons/th-list -->
  <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
    <path
      fill="currentColor"
      d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* toc panel */
    #toc_panel {
      box-sizing: border-box;
      position: fixed;
      top: 0;
      left: 0;
      background: #ffffff;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      z-index: 2;
    }

    /* toc panel when closed */
    #toc_panel[data-open="false"] {
      min-width: 60px;
      width: 60px;
      height: 60px;
      border-right: solid 1px #bdbdbd;
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc panel when open */
    #toc_panel[data-open="true"] {
      min-width: 260px;
      max-width: 480px;
      /* keep panel edge consistent distance away from "page" edge */
      width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
      bottom: 0;
      border-right: solid 1px #bdbdbd;
    }

    /* toc panel header */
    #toc_header {
      box-sizing: border-box;
      display: flex;
      flex-direction: row;
      align-items: center;
      height: 60px;
      margin: 0;
      padding: 20px;
    }

    /* toc panel header when hovered */
    #toc_header:hover {
      cursor: pointer;
    }

    /* toc panel header when panel open */
    #toc_panel[data-open="true"] > #toc_header {
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc open/close header button */
    #toc_button {
      margin-right: 20px;
    }

    /* hide toc list and header text when closed */
    #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
    #toc_panel[data-open="false"] > #toc_list {
      display: none;
    }

    /* toc list of entries */
    #toc_list {
      box-sizing: border-box;
      width: 100%;
      padding: 20px;
      position: absolute;
      top: calc(60px + 1px);
      bottom: 0;
      overflow: auto;
    }

    /* toc entry, link to section in document */
    .toc_link {
      display: block;
      padding: 5px;
      position: relative;
      font-weight: 600;
      text-decoration: none;
    }

    /* toc entry when hovered or when "viewed" */
    .toc_link:hover,
    .toc_link[data-viewing="true"] {
      background: #f5f5f5;
    }

    /* toc entry, level 1 indentation */
    .toc_link[data-level="1"] {
      margin-left: 0;
    }

    /* toc entry, level 2 indentation */
    .toc_link[data-level="2"] {
      margin-left: 20px;
    }

    /* toc entry, level 3 indentation */
    .toc_link[data-level="3"] {
      margin-left: 40px;
    }

    /* toc entry, level 4 indentation */
    .toc_link[data-level="4"] {
      margin-left: 60px;
    }

    /* toc entry bullets */
    #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
      position: absolute;
      left: -15px;
      top: -1px;
      font-size: 1.5em;
    }

    /* toc entry, level 2 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
      content: "\2022";
    }

    /* toc entry, level 3 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
      content: "\25AB";
    }

    /* toc entry, level 4 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
      content: "-";
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* push <body> ("page") element down to make room for toc icon */
    .toc_body_nudge {
      padding-top: 60px;
    }

    /* toc icon when panel closed and not hovered */
    #toc_panel[data-open="false"]:not(:hover) {
      background: rgba(255, 255, 255, 0.75);
    }
  }

  /* always hide toc panel on print */
  @media only print {
    #toc_panel {
      display: none;
    }
  }
</style>
<!-- 
  Tooltips Plugin

  Makes it such that when the user hovers or focuses a link to a citation or
  figure, a tooltip appears with a preview of the reference content, along with
  arrows to navigate between instances of the same reference in the document.
-->

<script type="module">
  // whether user must click off to close tooltip instead of just un-hovering
  const clickClose = "false";
  // delay (in ms) between opening and closing tooltip
  const delay = "100";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach hover and focus listeners to link
      link.addEventListener("mouseover", onLinkHover);
      link.addEventListener("mouseleave", onLinkUnhover);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("touchend", onLinkTouch);
    }

    // attach mouse, key, and resize listeners to window
    window.addEventListener("mousedown", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("keyup", onKeyUp);
    window.addEventListener("resize", onResize);
  }

  // when link is hovered
  function onLinkHover() {
    // function to open tooltip
    const delayOpenTooltip = function () {
      openTooltip(this);
    }.bind(this);

    // run open function after delay
    this.openTooltipTimer = window.setTimeout(delayOpenTooltip, delay);
  }

  // when mouse leaves link
  function onLinkUnhover() {
    // cancel opening tooltip
    window.clearTimeout(this.openTooltipTimer);

    // don't close on unhover if option specifies
    if (clickClose === "true") return;

    // function to close tooltip
    const delayCloseTooltip = function () {
      // if tooltip open and if mouse isn't over tooltip, close
      const tooltip = document.getElementById("tooltip");
      if (tooltip && !tooltip.matches(":hover")) closeTooltip();
    };

    // run close function after delay
    this.closeTooltipTimer = window.setTimeout(delayCloseTooltip, delay);
  }

  // when link is focused (tabbed to)
  function onLinkFocus(event) {
    openTooltip(this);
  }

  // when link is touched on touch screen
  function onLinkTouch(event) {
    // attempt to force hover state on first tap always, and trigger
    // regular link click (and navigation) on second tap
    if (event.target === document.activeElement) event.target.click();
    else {
      document.activeElement.blur();
      event.target.focus();
    }
    if (event.cancelable) event.preventDefault();
    event.stopPropagation();
    return false;
  }

  // when mouse is clicked anywhere in window
  function onClick(event) {
    closeTooltip();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("tooltip_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("tooltip_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeTooltip();
        break;
    }
  }

  // when window is resized or zoomed
  function onResize() {
    closeTooltip();
  }

  // get all links of types we wish to handle
  function getLinks() {
    const queries = [];
    // exclude buttons, anchor links, toc links, etc
    const exclude =
      ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    queries.push('a[href^="#ref-"]' + exclude); // citation links
    queries.push('a[href^="#fig:"]' + exclude); // figure links
    const query = queries.join(", ");
    return document.querySelectorAll(query);
  }

  // get links with same target, get index of link in set, get total
  // same links
  function getSameLinks(link) {
    const sameLinks = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        sameLinks.push(otherLink);
    }

    return {
      elements: sameLinks,
      index: sameLinks.indexOf(link),
      total: sameLinks.length,
    };
  }

  // open tooltip
  function openTooltip(link) {
    // delete tooltip if it exists, start fresh
    closeTooltip();

    // make tooltip element
    const tooltip = makeTooltip(link);

    // if source couldn't be found and tooltip not made, exit
    if (!tooltip) return;

    // make navbar elements
    const navBar = makeNavBar(link);
    if (navBar) tooltip.firstElementChild.appendChild(navBar);

    // attach tooltip to page
    document.body.appendChild(tooltip);

    // position tooltip
    const position = function () {
      positionTooltip(link);
    };
    position();

    // if tooltip contains images, position again after they've loaded
    const imgs = tooltip.querySelectorAll("img");
    for (const img of imgs) img.addEventListener("load", position);
  }

  // close (delete) tooltip
  function closeTooltip() {
    const tooltip = document.getElementById("tooltip");
    if (tooltip) tooltip.remove();
  }

  // make tooltip
  function makeTooltip(link) {
    // get target element that link points to
    const source = getSource(link);

    // if source can't be found, exit
    if (!source) return;

    // create new tooltip
    const tooltip = document.createElement("div");
    tooltip.id = "tooltip";
    const tooltipContent = document.createElement("div");
    tooltipContent.id = "tooltip_content";
    tooltip.appendChild(tooltipContent);

    // make copy of source node and put in tooltip
    const sourceCopy = makeCopy(source);
    tooltipContent.appendChild(sourceCopy);

    // attach mouse event listeners
    tooltip.addEventListener("click", onTooltipClick);
    tooltip.addEventListener("mousedown", onTooltipClick);
    tooltip.addEventListener("touchstart", onTooltipClick);
    tooltip.addEventListener("mouseleave", onTooltipUnhover);

    // (for interaction with lightbox plugin)
    // transfer click on tooltip copied img to original img
    const sourceImg = source.querySelector("img");
    const sourceCopyImg = sourceCopy.querySelector("img");
    if (sourceImg && sourceCopyImg) {
      const clickImg = function () {
        sourceImg.click();
        closeTooltip();
      };
      sourceCopyImg.addEventListener("click", clickImg);
    }

    return tooltip;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
      "class",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // when tooltip is clicked
  function onTooltipClick(event) {
    // when user clicks on tooltip, stop click from transferring
    // outside of tooltip (eg, click off to close tooltip, or eg click
    // off to unhighlight same refs)
    event.stopPropagation();
  }

  // when tooltip is unhovered
  function onTooltipUnhover(event) {
    if (clickClose === "true") return;

    // make sure new mouse/touch/focus no longer over tooltip or any
    // element within it
    const tooltip = document.getElementById("tooltip");
    if (!tooltip) return;
    if (this.contains(event.relatedTarget)) return;

    closeTooltip();
  }

  // make nav bar to go betwen prev/next instances of same reference
  function makeNavBar(link) {
    // find other links to the same source
    const sameLinks = getSameLinks(link);

    // don't show nav bar when singular reference
    if (sameLinks.total <= 1) return;

    // find prev/next links with same target
    const prevLink = getPrevLink(link, sameLinks);
    const nextLink = getNextLink(link, sameLinks);

    // create nav bar
    const navBar = document.createElement("div");
    navBar.id = "tooltip_nav_bar";
    const text = sameLinks.index + 1 + " of " + sameLinks.total;

    // create nav bar prev/next buttons
    const prevButton = document.createElement("button");
    const nextButton = document.createElement("button");
    prevButton.id = "tooltip_prev_button";
    nextButton.id = "tooltip_next_button";
    prevButton.title =
      "Jump to the previous occurence of this item in the document [←]";
    nextButton.title =
      "Jump to the next occurence of this item in the document [→]";
    prevButton.classList.add("icon_button");
    nextButton.classList.add("icon_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;
    nextButton.innerHTML =
      document.querySelector(".icon_caret_right").innerHTML;
    navBar.appendChild(prevButton);
    navBar.appendChild(document.createTextNode(text));
    navBar.appendChild(nextButton);

    // attach click listeners to buttons
    prevButton.addEventListener("click", function () {
      onPrevNextClick(link, prevLink);
    });
    nextButton.addEventListener("click", function () {
      onPrevNextClick(link, nextLink);
    });

    return navBar;
  }

  // get previous link with same target
  function getPrevLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if < 1
    let index;
    if (sameLinks.index - 1 >= 0) index = sameLinks.index - 1;
    else index = sameLinks.total - 1;
    return sameLinks.elements[index];
  }

  // get next link with same target
  function getNextLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if > total
    let index;
    if (sameLinks.index + 1 <= sameLinks.total - 1) index = sameLinks.index + 1;
    else index = 0;
    return sameLinks.elements[index];
  }

  // get element that is target of link or url hash
  function getSource(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector('[id="' + id + '"]');
    if (!target) return;

    // if ref or figure, modify target to get expected element
    if (id.indexOf("ref-") === 0) target = target.querySelector(":nth-child(2)");
    else if (id.indexOf("fig:") === 0) target = target.querySelector("figure");

    return target;
  }

  // when prev/next arrow button is clicked
  function onPrevNextClick(link, prevNextLink) {
    if (link && prevNextLink)
      goToElement(prevNextLink, window.innerHeight * 0.5);
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);
    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // determine position to place tooltip based on link position in
  // viewport and tooltip size
  function positionTooltip(link, left, top) {
    const tooltipElement = document.getElementById("tooltip");
    if (!tooltipElement) return;

    // get convenient vars for position/dimensions of
    // link/tooltip/page/view
    link = getRectInPage(link);
    const tooltip = getRectInPage(tooltipElement);
    const view = getRectInPage();

    // horizontal positioning
    if (left)
      // use explicit value
      left = left;
    else if (link.left + tooltip.width < view.right)
      // fit tooltip to right of link
      left = link.left;
    else if (link.right - tooltip.width > view.left)
      // fit tooltip to left of link
      left = link.right - tooltip.width;
    // center tooltip in view
    else left = (view.right - view.left) / 2 - tooltip.width / 2;

    // vertical positioning
    if (top)
      // use explicit value
      top = top;
    else if (link.top - tooltip.height > view.top)
      // fit tooltip above link
      top = link.top - tooltip.height;
    else if (link.bottom + tooltip.height < view.bottom)
      // fit tooltip below link
      top = link.bottom;
    else {
      // center tooltip in view
      top = view.top + view.height / 2 - tooltip.height / 2;
      // nudge off of link to left/right if possible
      if (link.right + tooltip.width < view.right) left = link.right;
      else if (link.left - tooltip.width > view.left)
        left = link.left - tooltip.width;
    }

    tooltipElement.style.left = left + "px";
    tooltipElement.style.top = top + "px";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* tooltip container */
    #tooltip {
      position: absolute;
      width: 50%;
      min-width: 240px;
      max-width: 75%;
      z-index: 1;
    }

    /* tooltip content */
    #tooltip_content {
      margin-bottom: 5px;
      padding: 20px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
      overflow-wrap: break-word;
    }

    /* tooltip copy of paragraphs and figures */
    #tooltip_content > p,
    #tooltip_content > figure {
      margin: 0;
      max-height: 320px;
      overflow-y: auto;
    }

    /* tooltip copy of <img> */
    #tooltip_content > figure > img,
    #tooltip_content > figure > svg {
      max-height: 260px;
    }

    /* navigation bar */
    #tooltip_nav_bar {
      margin-top: 10px;
      text-align: center;
    }

    /* navigation bar previous/next buton */
    #tooltip_nav_bar > .icon_button {
      position: relative;
      top: 3px;
    }

    /* navigation bar previous button */
    #tooltip_nav_bar > .icon_button:first-of-type {
      margin-right: 5px;
    }

    /* navigation bar next button */
    #tooltip_nav_bar > .icon_button:last-of-type {
      margin-left: 5px;
    }
  }

  /* always hide tooltip on print */
  @media only print {
    #tooltip {
      display: none;
    }
  }
</style>
<!--
  Analytics Plugin (third-party) 
  
  Copy and paste code from Google Analytics or similar service here.
-->
<!-- 
  Annotations Plugin

  Allows public annotation of the  manuscript. See https://web.hypothes.is/.
-->

<script type="module">
  // configuration
  window.hypothesisConfig = function () {
    return {
      branding: {
        accentColor: "#2196f3",
        appBackgroundColor: "#f8f8f8",
        ctaBackgroundColor: "#f8f8f8",
        ctaTextColor: "#000000",
        selectionFontFamily: "Open Sans, Helvetica, sans serif",
        annotationFontFamily: "Open Sans, Helvetica, sans serif",
      },
    };
  };

  // hypothesis client script
  const embed = "https://hypothes.is/embed.js";
  // hypothesis annotation count query url
  const query = "https://api.hypothes.is/api/search?limit=0&url=";

  // start script
  function start() {
    const button = makeButton();
    document.body.insertBefore(button, document.body.firstChild);
    insertCount(button);
  }

  // make button
  function makeButton() {
    // create button
    const button = document.createElement("button");
    button.id = "hypothesis_button";
    button.innerHTML = document.querySelector(".icon_hypothesis").innerHTML;
    button.title = "Hypothesis annotations";
    button.classList.add("icon_button");

    function onClick(event) {
      onButtonClick(event, button);
    }

    // attach click listeners
    button.addEventListener("click", onClick);

    return button;
  }

  // insert annotations count
  async function insertCount(button) {
    // get annotation count from Hypothesis based on url
    let count = "-";
    try {
      const canonical = document.querySelector('link[rel="canonical"]');
      const location = window.location;
      const url = encodeURIComponent((canonical || location).href);
      const response = await fetch(query + url);
      const json = await response.json();
      count = json.total || "-";
    } catch (error) {
      console.log(error);
    }

    // put count into button
    const counter = document.createElement("span");
    counter.id = "hypothesis_count";
    counter.innerHTML = count;
    button.title = "View " + count + " Hypothesis annotations";
    button.append(counter);
  }

  // when button is clicked
  function onButtonClick(event, button) {
    const script = document.createElement("script");
    script.src = embed;
    document.body.append(script);
    button.remove();
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- hypothesis icon -->

<template class="icon_hypothesis">
  <!-- modified from: https://simpleicons.org/icons/hypothesis.svg / https://git.io/Jf1VB -->
  <svg width="16" height="16" viewBox="0 0 24 24" tabindex="-1">
    <path
      fill="currentColor"
      d="M3.43 0C2.5 0 1.72 .768 1.72 1.72V18.86C1.72 19.8 2.5 20.57 3.43 20.57H9.38L12 24L14.62 20.57H20.57C21.5 20.57 22.29 19.8 22.29 18.86V1.72C22.29 .77 21.5 0 20.57 0H3.43M5.14 3.43H7.72V9.43S8.58 7.72 10.28 7.72C12 7.72 13.74 8.57 13.74 11.24V17.14H11.16V12C11.16 10.61 10.28 10.07 9.43 10.29C8.57 10.5 7.72 11.41 7.72 13.29V17.14H5.14V3.43M18 13.72C18.95 13.72 19.72 14.5 19.72 15.42A1.71 1.71 0 0 1 18 17.13A1.71 1.71 0 0 1 16.29 15.42C16.29 14.5 17.05 13.71 18 13.71Z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  /* hypothesis activation button */
  #hypothesis_button {
    box-sizing: border-box;
    position: fixed;
    top: 0;
    right: 0;
    width: 60px;
    height: 60px;
    background: #ffffff;
    border-radius: 0;
    border-left: solid 1px #bdbdbd;
    border-bottom: solid 1px #bdbdbd;
    box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
    z-index: 2;
  }

  /* hypothesis button svg */
  #hypothesis_button > svg {
    position: relative;
    top: -4px;
  }

  /* hypothesis annotation count */
  #hypothesis_count {
    position: absolute;
    left: 0;
    right: 0;
    bottom: 5px;
  }

  /* side panel */
  .annotator-frame {
    width: 280px !important;
  }

  /* match highlight color to rest of theme */
  .annotator-highlights-always-on .annotator-hl {
    background-color: #ffeb3b !important;
  }

  /* match focused color to rest of theme */
  .annotator-hl.annotator-hl-focused {
    background-color: #ff8a65 !important;
  }

  /* match bucket bar color to rest of theme */
  .annotator-bucket-bar {
    background: #f5f5f5 !important;
  }

  /* always hide button, toolbar, and tooltip on print */
  @media only print {
    #hypothesis_button {
      display: none;
    }

    .annotator-frame {
      display: none !important;
    }

    hypothesis-adder {
      display: none !important;
    }
  }
</style>
<!-- 
  Mathjax Plugin (third-party) 

  Allows the proper rendering of math/equations written in LaTeX.
  See https://www.mathjax.org/.
-->

<script type="text/x-mathjax-config">
  // configuration
  MathJax.Hub.Config({
    "CommonHTML": { linebreaks: { automatic: true } },
    "HTML-CSS": { linebreaks: { automatic: true } },
    "SVG": { linebreaks: { automatic: true } },
    "fast-preview": { disabled: true }
  });
</script>

<script
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
  integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A=="
  crossorigin="anonymous"
></script>

<style>
  /* mathjax containers */
  .math.display > span:not(.MathJax_Preview) {
    /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
    display: flex !important;
    overflow-x: auto !important;
    overflow-y: hidden !important;
    justify-content: center;
    align-items: center;
    margin: 0 !important;
  }

  /* right click menu */
  .MathJax_Menu {
    border-radius: 5px !important;
    border: solid 1px #bdbdbd !important;
    box-shadow: none !important;
  }

  /* equation auto-number */
  span[id^="eq:"] > span.math.display + span {
    font-weight: 600;
  }

  /* equation */
  span[id^="eq:"] > span.math.display > span {
    /* nudge to make room for equation auto-number and anchor */
    margin-right: 60px !important;
  }
</style>
</body>
</html>
